
<!DOCTYPE html>
<html class="client-nojs" lang="en" dir="ltr">
<head>
<meta charset="UTF-8"/>
<title>List of RNA structure prediction software - Wikipedia</title>
<script>document.documentElement.className="client-js";RLCONF={"wgBreakFrames":!1,"wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgRequestId":"XqXaZgpAEJsAA01@Y08AAADD","wgCSPNonce":!1,"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":!1,"wgNamespaceNumber":0,"wgPageName":"List_of_RNA_structure_prediction_software","wgTitle":"List of RNA structure prediction software","wgCurRevisionId":952034884,"wgRevisionId":952034884,"wgArticleId":15308316,"wgIsArticle":!0,"wgIsRedirect":!1,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["CS1 maint: multiple names: authors list","CS1: long volume value","Articles with short description","All articles with dead external links","Articles with dead external links from March 2020","Articles with permanently dead external links",
"Articles with dead external links from September 2019","Structural bioinformatics software","Lists of software","RNA"],"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgRelevantPageName":"List_of_RNA_structure_prediction_software","wgRelevantArticleId":15308316,"wgIsProbablyEditable":!0,"wgRelevantPageIsProbablyEditable":!0,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgMediaViewerOnClick":!0,"wgMediaViewerEnabledByDefault":!0,"wgPopupsReferencePreviews":!1,"wgPopupsConflictsWithNavPopupGadget":!1,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","pageVariantFallbacks":"en"},"wgMFDisplayWikibaseDescriptions":{"search":!0,"nearby":!0,"watchlist":!0,"tagline":!1},"wgWMESchemaEditAttemptStepOversample":!1,"wgULSCurrentAutonym":"English","wgNoticeProject":"wikipedia","wgWikibaseItemId":"Q6595358","wgCentralAuthMobileDomain":!1,"wgEditSubmitButtonLabelPublish":!0};RLSTATE={"ext.globalCssJs.user.styles":"ready","site.styles":
"ready","noscript":"ready","user.styles":"ready","ext.globalCssJs.user":"ready","user":"ready","user.options":"loading","ext.cite.styles":"ready","ext.math.styles":"ready","jquery.tablesorter.styles":"ready","mediawiki.toc.styles":"ready","skins.vector.styles.legacy":"ready","wikibase.client.init":"ready","ext.visualEditor.desktopArticleTarget.noscript":"ready","ext.uls.interlanguage":"ready","ext.wikimediaBadges":"ready"};RLPAGEMODULES=["ext.cite.ux-enhancements","ext.math.scripts","site","mediawiki.page.startup","skins.vector.js","mediawiki.page.ready","jquery.tablesorter","mediawiki.toc","ext.gadget.ReferenceTooltips","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.extra-toolbar-buttons","ext.gadget.switcher","ext.centralauth.centralautologin","ext.popups","ext.visualEditor.desktopArticleTarget.init","ext.visualEditor.targetLoader","ext.eventLogging","ext.wikimediaEvents","ext.navigationTiming","ext.uls.compactlinks","ext.uls.interface","ext.cx.eventlogging.campaigns",
"ext.quicksurveys.init","ext.centralNotice.geoIP","ext.centralNotice.startUp"];</script>
<script>(RLQ=window.RLQ||[]).push(function(){mw.loader.implement("user.options@1hzgi",function($,jQuery,require,module){/*@nomin*/mw.user.tokens.set({"patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});
});});</script>
<link rel="stylesheet" href="/w/load.php?lang=en&amp;modules=ext.cite.styles%7Cext.math.styles%7Cext.uls.interlanguage%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cjquery.tablesorter.styles%7Cmediawiki.toc.styles%7Cskins.vector.styles.legacy%7Cwikibase.client.init&amp;only=styles&amp;skin=vector"/>
<script async="" src="/w/load.php?lang=en&amp;modules=startup&amp;only=scripts&amp;raw=1&amp;skin=vector"></script>
<meta name="ResourceLoaderDynamicStyles" content=""/>
<link rel="stylesheet" href="/w/load.php?lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=vector"/>
<meta name="generator" content="MediaWiki 1.35.0-wmf.28"/>
<meta name="referrer" content="origin"/>
<meta name="referrer" content="origin-when-crossorigin"/>
<meta name="referrer" content="origin-when-cross-origin"/>
<link rel="alternate" type="application/x-wiki" title="Edit this page" href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit"/>
<link rel="edit" title="Edit this page" href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit"/>
<link rel="apple-touch-icon" href="/static/apple-touch/wikipedia.png"/>
<link rel="shortcut icon" href="/static/favicon/wikipedia.ico"/>
<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (en)"/>
<link rel="EditURI" type="application/rsd+xml" href="//en.wikipedia.org/w/api.php?action=rsd"/>
<link rel="license" href="//creativecommons.org/licenses/by-sa/3.0/"/>
<link rel="alternate" type="application/atom+xml" title="Wikipedia Atom feed" href="/w/index.php?title=Special:RecentChanges&amp;feed=atom"/>
<link rel="canonical" href="https://en.wikipedia.org/wiki/List_of_RNA_structure_prediction_software"/>
<link rel="dns-prefetch" href="//login.wikimedia.org"/>
<link rel="dns-prefetch" href="//meta.wikimedia.org" />
<!--[if lt IE 9]><script src="/w/resources/lib/html5shiv/html5shiv.js"></script><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject mw-editable page-List_of_RNA_structure_prediction_software rootpage-List_of_RNA_structure_prediction_software skin-vector action-view">
<div id="mw-page-base" class="noprint"></div>
<div id="mw-head-base" class="noprint"></div>
<div id="content" class="mw-body" role="main">
	<a id="top"></a>
		<div id="siteNotice" class="mw-body-content"><!-- CentralNotice --></div>
	<div class="mw-indicators mw-body-content">
</div>

	<h1 id="firstHeading" class="firstHeading" lang="en">List of RNA structure prediction software</h1>
	
	<div id="bodyContent" class="mw-body-content">
		<div id="siteSub" class="noprint">From Wikipedia, the free encyclopedia</div>
		<div id="contentSub"></div>
		
		
		<div id="jump-to-nav"></div>
		<a class="mw-jump-link" href="#mw-head">Jump to navigation</a>
		<a class="mw-jump-link" href="#p-search">Jump to search</a>
		<div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><div class="mw-parser-output"><div class="shortdescription nomobile noexcerpt noprint searchaux" style="display:none">Wikipedia list article</div>
<p>This <b>list of RNA structure prediction software</b> is a compilation of software tools and web portals used for <a href="/wiki/RNA_structure" class="mw-redirect" title="RNA structure">RNA structure</a> prediction.
</p>
<div id="toc" class="toc" role="navigation" aria-labelledby="mw-toc-heading"><input type="checkbox" role="button" id="toctogglecheckbox" class="toctogglecheckbox" style="display:none" /><div class="toctitle" lang="en" dir="ltr"><h2 id="mw-toc-heading">Contents</h2><span class="toctogglespan"><label class="toctogglelabel" for="toctogglecheckbox"></label></span></div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="#Single_sequence_secondary_structure_prediction."><span class="tocnumber">1</span> <span class="toctext">Single sequence secondary structure prediction.</span></a></li>
<li class="toclevel-1 tocsection-2"><a href="#Single_sequence_tertiary_structure_prediction"><span class="tocnumber">2</span> <span class="toctext">Single sequence tertiary structure prediction</span></a></li>
<li class="toclevel-1 tocsection-3"><a href="#Comparative_methods"><span class="tocnumber">3</span> <span class="toctext">Comparative methods</span></a></li>
<li class="toclevel-1 tocsection-4"><a href="#Intermolecular_interactions:_RNA-RNA"><span class="tocnumber">4</span> <span class="toctext">Intermolecular interactions: RNA-RNA</span></a></li>
<li class="toclevel-1 tocsection-5"><a href="#Intermolecular_interactions:_MicroRNA:any_RNA"><span class="tocnumber">5</span> <span class="toctext">Intermolecular interactions: MicroRNA:any RNA</span></a></li>
<li class="toclevel-1 tocsection-6"><a href="#Intermolecular_interactions:_MicroRNA:UTR"><span class="tocnumber">6</span> <span class="toctext">Intermolecular interactions: MicroRNA:UTR</span></a></li>
<li class="toclevel-1 tocsection-7"><a href="#ncRNA_gene_prediction_software"><span class="tocnumber">7</span> <span class="toctext">ncRNA gene prediction software</span></a></li>
<li class="toclevel-1 tocsection-8"><a href="#Family_specific_gene_prediction_software"><span class="tocnumber">8</span> <span class="toctext">Family specific gene prediction software</span></a></li>
<li class="toclevel-1 tocsection-9"><a href="#RNA_homology_search_software"><span class="tocnumber">9</span> <span class="toctext">RNA homology search software</span></a></li>
<li class="toclevel-1 tocsection-10"><a href="#Benchmarks"><span class="tocnumber">10</span> <span class="toctext">Benchmarks</span></a></li>
<li class="toclevel-1 tocsection-11"><a href="#Alignment_viewers,_editors"><span class="tocnumber">11</span> <span class="toctext">Alignment viewers, editors</span></a></li>
<li class="toclevel-1 tocsection-12"><a href="#Inverse_folding,_RNA_design"><span class="tocnumber">12</span> <span class="toctext">Inverse folding, RNA design</span></a></li>
<li class="toclevel-1 tocsection-13"><a href="#Secondary_structure_viewers,_editors"><span class="tocnumber">13</span> <span class="toctext">Secondary structure viewers, editors</span></a></li>
<li class="toclevel-1 tocsection-14"><a href="#See_also"><span class="tocnumber">14</span> <span class="toctext">See also</span></a></li>
<li class="toclevel-1 tocsection-15"><a href="#References"><span class="tocnumber">15</span> <span class="toctext">References</span></a></li>
</ul>
</div>

<h2><span class="mw-headline" id="Single_sequence_secondary_structure_prediction.">Single sequence secondary structure prediction.</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=1" title="Edit section: Single sequence secondary structure prediction.">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description
</th>
<th>Knots<br /><sup id="cite_ref-1" class="reference"><a href="#cite_note-1">&#91;Note 1&#93;</a></sup>
</th>
<th>Links</th>
<th>References
</th></tr>
<tr>
<th><a href="/w/index.php?title=CentroidFold&amp;action=edit&amp;redlink=1" class="new" title="CentroidFold (page does not exist)">CentroidFold</a>
</th>
<td>Secondary structure prediction based on generalized centroid estimator</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="https://github.com/satoken/centroid-rna-package">sourcecode</a> <a rel="nofollow" class="external text" href="http://rtools.cbrc.jp/">webserver</a></td>
<td><sup id="cite_ref-pmid19095700_2-0" class="reference"><a href="#cite_note-pmid19095700-2">&#91;1&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=CentroidHomfold&amp;action=edit&amp;redlink=1" class="new" title="CentroidHomfold (page does not exist)">CentroidHomfold</a>
</th>
<td>Secondary structure prediction by using homologous sequence information</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="https://github.com/satoken/centroid-rna-package">sourcecode</a> <a rel="nofollow" class="external text" href="http://rtools.cbrc.jp/">webserver</a></td>
<td><sup id="cite_ref-pmid19478007_3-0" class="reference"><a href="#cite_note-pmid19478007-3">&#91;2&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Context_Fold&amp;action=edit&amp;redlink=1" class="new" title="Context Fold (page does not exist)">Context Fold</a>
</th>
<td>An RNA secondary structure prediction software based on feature-rich trained scoring models.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.cs.bgu.ac.il/~negevcb/contextfold/ContextFold_1_00.zip">sourcecode</a> <a rel="nofollow" class="external text" href="http://www.cs.bgu.ac.il/~negevcb/contextfold/">webserver</a></td>
<td><sup id="cite_ref-pmid22035327_4-0" class="reference"><a href="#cite_note-pmid22035327-4">&#91;3&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=CONTRAfold&amp;action=edit&amp;redlink=1" class="new" title="CONTRAfold (page does not exist)">CONTRAfold</a>
</th>
<td>Secondary structure prediction method based on conditional log-linear models (CLLMs), a flexible class of probabilistic models which generalize upon <a href="/wiki/Stochastic_context-free_grammar" class="mw-redirect" title="Stochastic context-free grammar">SCFGs</a> by using discriminative training and <a href="/wiki/Feature-rich" class="mw-redirect" title="Feature-rich">feature-rich</a> scoring.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://contra.stanford.edu/contrafold/">sourcecode</a> <a rel="nofollow" class="external text" href="http://contra.stanford.edu/contrafold/server.html">webserver</a></td>
<td><sup id="cite_ref-pmid16873527_5-0" class="reference"><a href="#cite_note-pmid16873527-5">&#91;4&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/Crumple" class="mw-redirect" title="Crumple">Crumple</a>
</th>
<td>Simple, cleanly written software to produce the full set of possible secondary structures for one sequence, given optional constraints.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://adenosine.chem.ou.edu#crumple">sourcecode</a></td>
<td><sup id="cite_ref-pmid21723827_6-0" class="reference"><a href="#cite_note-pmid21723827-6">&#91;5&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=CyloFold&amp;action=edit&amp;redlink=1" class="new" title="CyloFold (page does not exist)">CyloFold</a>
</th>
<td>Secondary structure prediction method based on placement of helices allowing complex pseudoknots.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://cylofold.abcc.ncifcrf.gov/">webserver</a></td>
<td><sup id="cite_ref-pmid20501603_7-0" class="reference"><a href="#cite_note-pmid20501603-7">&#91;6&#93;</a></sup>
</td></tr>
<tr>
<th><b><a href="/w/index.php?title=E2Efold&amp;action=edit&amp;redlink=1" class="new" title="E2Efold (page does not exist)">E2Efold</a></b>
</th>
<td>A deep learning based method for efficiently predicting secondary structure by differentiating through a constrained optimization solver, without using dynamic programming.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes
</td>
<td><a rel="nofollow" class="external text" href="https://github.com/ml4bio/e2efold">sourcecode</a>
</td>
<td><sup id="cite_ref-8" class="reference"><a href="#cite_note-8">&#91;7&#93;</a></sup><sup id="cite_ref-9" class="reference"><a href="#cite_note-9">&#91;8&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=GTFold&amp;action=edit&amp;redlink=1" class="new" title="GTFold (page does not exist)">GTFold</a>
</th>
<td>Fast and scalable multicore code for predicting RNA secondary structure.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://gtfold.sourceforge.net/">link</a> <a rel="nofollow" class="external text" href="https://github.com/gtfold/gtfold/">sourcecode</a></td>
<td><sup id="cite_ref-pmid22747589_10-0" class="reference"><a href="#cite_note-pmid22747589-10">&#91;9&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=IPknot&amp;action=edit&amp;redlink=1" class="new" title="IPknot (page does not exist)">IPknot</a>
</th>
<td>Fast and accurate prediction of RNA secondary structures with pseudoknots using integer programming.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="https://github.com/satoken/ipknot">sourcecode</a> <a rel="nofollow" class="external text" href="http://rtips.dna.bio.keio.ac.jp/ipknot/">webserver</a></td>
<td><sup id="cite_ref-pmid21685106_11-0" class="reference"><a href="#cite_note-pmid21685106-11">&#91;10&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=KineFold&amp;action=edit&amp;redlink=1" class="new" title="KineFold (page does not exist)">KineFold</a>
</th>
<td>Folding kinetics of RNA sequences including pseudoknots by including an implementation of the partition function for knots.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://kinefold.curie.fr/">linuxbinary</a>, <a rel="nofollow" class="external text" href="http://kinefold.curie.fr/cgi-bin/form.pl">webserver</a></td>
<td><sup id="cite_ref-pmid15980546_12-0" class="reference"><a href="#cite_note-pmid15980546-12">&#91;11&#93;</a></sup><sup id="cite_ref-pmid14676318_13-0" class="reference"><a href="#cite_note-pmid14676318-13">&#91;12&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Mfold&amp;action=edit&amp;redlink=1" class="new" title="Mfold (page does not exist)">Mfold</a>
</th>
<td><a href="/wiki/Gibbs_free_energy" title="Gibbs free energy">MFE</a> (Minimum Free Energy) RNA structure prediction algorithm.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.bioinfo.rpi.edu/applications/mfold/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://mfold.rit.albany.edu/?q=mfold">webserver</a></td>
<td><sup id="cite_ref-pmid6163133_14-0" class="reference"><a href="#cite_note-pmid6163133-14">&#91;13&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=PKiss&amp;action=edit&amp;redlink=1" class="new" title="PKiss (page does not exist)">pKiss</a>
</th>
<td>A dynamic programming algorithm for the prediction of a restricted class (H-type and kissing hairpins) of RNA pseudoknots.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://bibiwsserv.cebitec.uni-bielefeld.de/fold-grammars/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://bibiserv2.cebitec.uni-bielefeld.de/pkiss?id=pkiss_view_submission">webserver</a></td>
<td><sup id="cite_ref-pmid_pkiss_15-0" class="reference"><a href="#cite_note-pmid_pkiss-15">&#91;14&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Pknots&amp;action=edit&amp;redlink=1" class="new" title="Pknots (page does not exist)">Pknots</a>
</th>
<td>A dynamic programming algorithm for optimal RNA pseudoknot prediction using the nearest neighbour energy model.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://selab.janelia.org/software.html">sourcecode</a></td>
<td><sup id="cite_ref-pmid9925784_16-0" class="reference"><a href="#cite_note-pmid9925784-16">&#91;15&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=PknotsRG&amp;action=edit&amp;redlink=1" class="new" title="PknotsRG (page does not exist)">PknotsRG</a>
</th>
<td>A dynamic programming algorithm for the prediction of a restricted class (H-type) of RNA pseudoknots.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/pknotsrg/welcome.html">sourcecode</a>, <a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/pknotsrg/submission.html">webserver</a></td>
<td><sup id="cite_ref-pmid17478505_17-0" class="reference"><a href="#cite_note-pmid17478505-17">&#91;16&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNA123&amp;action=edit&amp;redlink=1" class="new" title="RNA123 (page does not exist)">RNA123</a>
</th>
<td>Secondary structure prediction via thermodynamic-based folding algorithms and novel structure-based sequence alignment specific for RNA.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.rna123.com/">webserver</a></td>
<td>
</td></tr>
<tr>
<th><a href="/wiki/ViennaRNA_Package" title="ViennaRNA Package">RNAfold</a>
</th>
<td>MFE RNA structure prediction algorithm. Includes an implementation of the partition function for computing basepair probabilities and circular RNA folding.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/~ivo/RNA/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi">webserver</a></td>
<td>
<p><sup id="cite_ref-pmid6163133_14-1" class="reference"><a href="#cite_note-pmid6163133-14">&#91;13&#93;</a></sup><sup id="cite_ref-RNAinverse_18-0" class="reference"><a href="#cite_note-RNAinverse-18">&#91;17&#93;</a></sup><sup id="cite_ref-pmid1695107_19-0" class="reference"><a href="#cite_note-pmid1695107-19">&#91;18&#93;</a></sup><sup id="cite_ref-pmid16452114_20-0" class="reference"><a href="#cite_note-pmid16452114-20">&#91;19&#93;</a></sup><sup id="cite_ref-pmid17611759_21-0" class="reference"><a href="#cite_note-pmid17611759-21">&#91;20&#93;</a></sup>
</p>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAshapes&amp;action=edit&amp;redlink=1" class="new" title="RNAshapes (page does not exist)">RNAshapes</a>
</th>
<td>MFE RNA structure prediction based on abstract shapes. Shape abstraction retains adjacency and nesting of structural features, but disregards helix lengths, thus reduces the number of suboptimal solutions without losing significant information. Furthermore, shapes represent classes of structures for which probabilities based on Boltzmann-weighted energies can be computed.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/download/tools/rnashapes.html">source &amp; binaries</a>, <a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnashapes/">webserver</a></td>
<td><sup id="cite_ref-abstract_shapes_22-0" class="reference"><a href="#cite_note-abstract_shapes-22">&#91;21&#93;</a></sup><sup id="cite_ref-shape_probabilities_23-0" class="reference"><a href="#cite_note-shape_probabilities-23">&#91;22&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAstructure&amp;action=edit&amp;redlink=1" class="new" title="RNAstructure (page does not exist)">RNAstructure</a>
</th>
<td>A program to predict lowest free energy structures and base pair probabilities for RNA or DNA sequences.  Programs are also available to predict <i>maximum expected accuracy</i> structures and these can include pseudoknots.  Structure prediction can be constrained using experimental data, including SHAPE, enzymatic cleavage, and chemical modification accessibility. Graphical user interfaces are available for Windows, Mac OS X, Linux.  Programs are also available for use with Unix-style text interfaces.  Also, a C++ class library is available.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://rna.urmc.rochester.edu/RNAstructure.html">source &amp; binaries</a>, <a rel="nofollow" class="external text" href="http://rna.urmc.rochester.edu/RNAstructureWeb/">webserver</a></td>
<td>
<p><sup id="cite_ref-RNAstructure_24-0" class="reference"><a href="#cite_note-RNAstructure-24">&#91;23&#93;</a></sup><sup id="cite_ref-RNAstructure-partition_25-0" class="reference"><a href="#cite_note-RNAstructure-partition-25">&#91;24&#93;</a></sup>
</p>
</td></tr>
<tr>
<th><a href="/w/index.php?title=SARNA-Predict&amp;action=edit&amp;redlink=1" class="new" title="SARNA-Predict (page does not exist)">SARNA-Predict</a>
</th>
<td>RNA Secondary structure prediction method based on simulated annealing. It can also predict structure with pseudoknots.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://herberttsang.org">link</a></td>
<td><sup id="cite_ref-26" class="reference"><a href="#cite_note-26">&#91;25&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Sfold&amp;action=edit&amp;redlink=1" class="new" title="Sfold (page does not exist)">Sfold</a>
</th>
<td>Statistical sampling of all possible structures. The sampling is weighted by partition function probabilities.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://sfold.wadsworth.org">webserver</a></td>
<td><sup id="cite_ref-pmid14654704_27-0" class="reference"><a href="#cite_note-pmid14654704-27">&#91;26&#93;</a></sup><sup id="cite_ref-pmid15215366_28-0" class="reference"><a href="#cite_note-pmid15215366-28">&#91;27&#93;</a></sup><sup id="cite_ref-pmid16043502_29-0" class="reference"><a href="#cite_note-pmid16043502-29">&#91;28&#93;</a></sup><sup id="cite_ref-pmid16109749_30-0" class="reference"><a href="#cite_note-pmid16109749-30">&#91;29&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Sliding_Windows_%26_Assembly&amp;action=edit&amp;redlink=1" class="new" title="Sliding Windows &amp; Assembly (page does not exist)">Sliding Windows &amp; Assembly</a>
</th>
<td>Sliding windows and assembly is a tool chain for folding long series of similar hairpins.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://adenosine.chem.ou.edu#sliding">sourcecode</a></td>
<td><sup id="cite_ref-pmid21723827_6-1" class="reference"><a href="#cite_note-pmid21723827-6">&#91;5&#93;</a></sup>
</td></tr>
<tr>
<th><a href="//doi.org/10.1038/s41467-019-13395-9" class="extiw" title="doi:10.1038/s41467-019-13395-9">SPOT-RNA</a>
</th>
<td>SPOT-RNA is first RNA secondary structure predictor which can predict all kind base pairs (canonical, noncanonical, pseudoknots, and base triplets).</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes
</td>
<td><a rel="nofollow" class="external text" href="https://github.com/jaswindersingh2/SPOT-RNA">sourcecode</a>
<p><a rel="nofollow" class="external text" href="http://sparks-lab.org/jaswinder/server/SPOT-RNA/">webserver</a>
</p>
</td>
<td><sup id="cite_ref-31" class="reference"><a href="#cite_note-31">&#91;30&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=SwiSpot&amp;action=edit&amp;redlink=1" class="new" title="SwiSpot (page does not exist)">SwiSpot</a>
</th>
<td>Command-line utility for predicting alternative (secondary) configurations of <a href="/wiki/Riboswitch" title="Riboswitch">riboswitches</a>. It is based on the prediction of the so-called switching sequence, to subsequently constrain the folding of the two functional structures.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.iet.unipi.it/a.bechini/swispot/">sourcecode</a></td>
<td><sup id="cite_ref-pmid27378291_32-0" class="reference"><a href="#cite_note-pmid27378291-32">&#91;31&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=UNAFold&amp;action=edit&amp;redlink=1" class="new" title="UNAFold (page does not exist)">UNAFold</a>
</th>
<td>The UNAFold software package is an integrated collection of programs that simulate folding, hybridization, and melting pathways for one or two single-stranded nucleic acid sequences.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.bioinfo.rpi.edu/applications/hybrid/download.php">sourcecode</a></td>
<td><sup id="cite_ref-pmid18712296_33-0" class="reference"><a href="#cite_note-pmid18712296-33">&#91;32&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Vsfold/vs_subopt&amp;action=edit&amp;redlink=1" class="new" title="Vsfold/vs subopt (page does not exist)">vsfold/vs subopt</a>
</th>
<td>Folds and predicts RNA secondary structure and pseudoknots using an entropy model derived from polymer physics. The program vs_subopt computes suboptimal structures based on the free energy landscape derived from vsfold5.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.rna.it-chiba.ac.jp">webserver</a></td>
<td><sup id="cite_ref-pmid17878940_34-0" class="reference"><a href="#cite_note-pmid17878940-34">&#91;33&#93;</a></sup><sup id="cite_ref-pmidnone_35-0" class="reference"><a href="#cite_note-pmidnone-35">&#91;34&#93;</a></sup>
</td></tr>
<tr>
<td colspan="5">
<dl><dt>Notes</dt>
<dd></dd></dl>
<div class="reflist" style="list-style-type: decimal;">
<div class="mw-references-wrap"><ol class="references">
<li id="cite_note-1"><span class="mw-cite-backlink"><b><a href="#cite_ref-1">^</a></b></span> <span class="reference-text"><b>Knots:</b> <a href="/wiki/Pseudoknot" title="Pseudoknot">Pseudoknot</a> prediction, &lt;yes|no&gt;.</span>
</li>
</ol></div></div>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="Single_sequence_tertiary_structure_prediction">Single sequence tertiary structure prediction</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=2" title="Edit section: Single sequence tertiary structure prediction">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description
</th>
<th>Knots<br /><sup id="cite_ref-36" class="reference"><a href="#cite_note-36">&#91;Note 1&#93;</a></sup>
</th>
<th>Links</th>
<th>References
</th></tr>
<tr>
<th><a href="/w/index.php?title=BARNACLE&amp;action=edit&amp;redlink=1" class="new" title="BARNACLE (page does not exist)">BARNACLE</a>
</th>
<td>A Python library for the probabilistic sampling of RNA structures that are compatible with a given nucleotide sequence and that are RNA-like on a local length scale.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://sourceforge.net/projects/barnacle-rna/">sourcecode</a></td>
<td><sup id="cite_ref-pmid19543381_37-0" class="reference"><a href="#cite_note-pmid19543381-37">&#91;35&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=FARNA&amp;action=edit&amp;redlink=1" class="new" title="FARNA (page does not exist)">FARNA</a>
</th>
<td>Automated de novo prediction of native-like RNA tertiary structures .</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td></td>
<td><sup id="cite_ref-pmid17726102_38-0" class="reference"><a href="#cite_note-pmid17726102-38">&#91;36&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=IFoldRNA&amp;action=edit&amp;redlink=1" class="new" title="IFoldRNA (page does not exist)">iFoldRNA</a>
</th>
<td>three-dimensional RNA structure prediction and folding</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://iFoldRNA.dokhlab.org">webserver</a></td>
<td><sup id="cite_ref-pmid18579566_39-0" class="reference"><a href="#cite_note-pmid18579566-39">&#91;37&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MC-Fold_MC-Sym_Pipeline&amp;action=edit&amp;redlink=1" class="new" title="MC-Fold MC-Sym Pipeline (page does not exist)">MC-Fold MC-Sym Pipeline</a>
</th>
<td>Thermodynamics and Nucleotide cyclic motifs for RNA structure prediction algorithm. 2D and 3D structures.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.major.iric.ca/MajorLabEn/MC-Tools.html">sourcecode</a>, <a rel="nofollow" class="external text" href="http://www.major.iric.ca/MC-Pipeline/">webserver</a></td>
<td><sup id="cite_ref-pmid18322526_40-0" class="reference"><a href="#cite_note-pmid18322526-40">&#91;38&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=NAST_(software)&amp;action=edit&amp;redlink=1" class="new" title="NAST (software) (page does not exist)">NAST</a>
</th>
<td>Coarse-grained modeling of large RNA molecules with knowledge-based potentials and structural filters</td>
<td style="background: #ececec; color: #2C2C2C; font-size: smaller; vertical-align: middle; text-align: center;" class="unknown table-unknown">Unknown</td>
<td><a rel="nofollow" class="external text" href="https://simtk.org/home/nast">executables</a></td>
<td><sup id="cite_ref-pmid19144906_41-0" class="reference"><a href="#cite_note-pmid19144906-41">&#91;39&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MMB_(software)&amp;action=edit&amp;redlink=1" class="new" title="MMB (software) (page does not exist)">MMB</a>
</th>
<td>Turning limited experimental information into 3D models of RNA</td>
<td style="background: #ececec; color: #2C2C2C; font-size: smaller; vertical-align: middle; text-align: center;" class="unknown table-unknown">Unknown</td>
<td><a rel="nofollow" class="external text" href="https://simtk.org/home/rnatoolbox">sourcecode</a></td>
<td><sup id="cite_ref-pmid20651028_42-0" class="reference"><a href="#cite_note-pmid20651028-42">&#91;40&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNA123&amp;action=edit&amp;redlink=1" class="new" title="RNA123 (page does not exist)">RNA123</a>
</th>
<td>Integrated platform for de novo and homology modeling of RNA 3D structures, where coordinate file input, sequence editing, sequence alignment, structure prediction and analysis features are all accessed from one intuitive graphical user interface.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td></td>
<td>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAComposer&amp;action=edit&amp;redlink=1" class="new" title="RNAComposer (page does not exist)">RNAComposer</a>
</th>
<td>Fully automated prediction of large RNA 3D structures.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://rnacomposer.cs.put.poznan.pl/">webserver</a> <a rel="nofollow" class="external text" href="http://rnacomposer.ibch.poznan.pl/">webserver</a></td>
<td><sup id="cite_ref-pmid22539264_43-0" class="reference"><a href="#cite_note-pmid22539264-43">&#91;41&#93;</a></sup>
</td></tr>
<tr>
<td colspan="5">
<dl><dt>Notes</dt>
<dd></dd></dl>
<div class="reflist" style="list-style-type: decimal;">
<div class="mw-references-wrap"><ol class="references">
<li id="cite_note-36"><span class="mw-cite-backlink"><b><a href="#cite_ref-36">^</a></b></span> <span class="reference-text"><b>Knots:</b> <a href="/wiki/Pseudoknot" title="Pseudoknot">Pseudoknot</a> prediction, &lt;yes|no&gt;.</span>
</li>
</ol></div></div>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="Comparative_methods">Comparative methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=3" title="Edit section: Comparative methods">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>The single sequence methods mentioned above have a difficult job detecting a small sample of reasonable secondary structures from a large space of possible structures. A good way to reduce the size of the space is to use evolutionary approaches. Structures that have been conserved by evolution are far more likely to be the functional form. The methods below use this approach.
</p>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description
</th>
<th>Number of sequences<br /><sup id="cite_ref-44" class="reference"><a href="#cite_note-44">&#91;Note 1&#93;</a></sup>
</th>
<th>Alignment<br /><sup id="cite_ref-45" class="reference"><a href="#cite_note-45">&#91;Note 2&#93;</a></sup>
</th>
<th>Structure<br /><sup id="cite_ref-46" class="reference"><a href="#cite_note-46">&#91;Note 3&#93;</a></sup>
</th>
<th>Knots<br /><sup id="cite_ref-47" class="reference"><a href="#cite_note-47">&#91;Note 4&#93;</a></sup>
</th>
<th>Link</th>
<th>References
</th></tr>
<tr>
<th><a href="/w/index.php?title=Carnac_(software)&amp;action=edit&amp;redlink=1" class="new" title="Carnac (software) (page does not exist)">Carnac</a>
</th>
<td>Comparative analysis combined with MFE folding.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://bioinfo.lifl.fr/RNA/carnac/index.php">sourcecode</a>, <a rel="nofollow" class="external text" href="http://bioinfo.lifl.fr/RNA/carnac/carnac.php">webserver</a></td>
<td><sup id="cite_ref-pmid12499300_48-0" class="reference"><a href="#cite_note-pmid12499300-48">&#91;42&#93;</a></sup><sup id="cite_ref-pmid15215367_49-0" class="reference"><a href="#cite_note-pmid15215367-49">&#91;43&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=CentroidAlifold&amp;action=edit&amp;redlink=1" class="new" title="CentroidAlifold (page does not exist)">CentroidAlifold</a>
</th>
<td>Common secondary structure prediction based on generalized centroid estimator</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="https://github.com/satoken/centroid-rna-package">sourcecode</a></td>
<td><sup id="cite_ref-pmid20843778_50-0" class="reference"><a href="#cite_note-pmid20843778-50">&#91;44&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=CentroidAlign&amp;action=edit&amp;redlink=1" class="new" title="CentroidAlign (page does not exist)">CentroidAlign</a>
</th>
<td>Fast and accurate multiple aligner for RNA sequences</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.ncrna.org/software/centroidalign">sourcecode</a></td>
<td><sup id="cite_ref-pmid19808876_51-0" class="reference"><a href="#cite_note-pmid19808876-51">&#91;45&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=CMfinder&amp;action=edit&amp;redlink=1" class="new" title="CMfinder (page does not exist)">CMfinder</a>
</th>
<td>an expectation maximization algorithm using covariance models for motif description. Uses heuristics for effective motif search, and a Bayesian framework for structure prediction combining folding energy and sequence covariation.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial"><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle 3\leq seqs\leq 60}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mn>3</mn>
        <mo>&#x2264;<!-- ≤ --></mo>
        <mi>s</mi>
        <mi>e</mi>
        <mi>q</mi>
        <mi>s</mi>
        <mo>&#x2264;<!-- ≤ --></mo>
        <mn>60</mn>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle 3\leq seqs\leq 60}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/5af313df3bc5a03239192e8abc3431829723385f" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:14.018ex; height:2.509ex;" alt="3\leq seqs\leq 60"/></span></td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://bio.cs.washington.edu/yzizhen/CMfinder/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://wingless.cs.washington.edu/htbin-post/unrestricted/CMfinderWeb/CMfinderInput.pl">webserver</a>, <a rel="nofollow" class="external text" href="http://bio.cs.washington.edu/yzizhen/CMfinder/">website</a></td>
<td><sup id="cite_ref-pmid16357030_52-0" class="reference"><a href="#cite_note-pmid16357030-52">&#91;46&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=CONSAN&amp;action=edit&amp;redlink=1" class="new" title="CONSAN (page does not exist)">CONSAN</a>
</th>
<td>implements a pinned Sankoff algorithm for simultaneous pairwise RNA alignment and consensus structure prediction.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">2</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://selab.janelia.org/software.html">sourcecode</a></td>
<td><sup id="cite_ref-pmid16952317_53-0" class="reference"><a href="#cite_note-pmid16952317-53">&#91;47&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=DAFS_(software)&amp;action=edit&amp;redlink=1" class="new" title="DAFS (software) (page does not exist)">DAFS</a>
</th>
<td>Simultaneous aligning and folding of RNA sequences via dual decomposition.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="https://github.com/satoken/dafs">sourcecode</a></td>
<td><sup id="cite_ref-pmid23060618_54-0" class="reference"><a href="#cite_note-pmid23060618-54">&#91;48&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Dynalign&amp;action=edit&amp;redlink=1" class="new" title="Dynalign (page does not exist)">Dynalign</a>
</th>
<td>an algorithm that improves the accuracy of structure prediction by combining free energy minimization and comparative sequence analysis to find a low free energy structure common to two sequences without requiring any sequence identity.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">2</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://rna.urmc.rochester.edu/dynalign.html">sourcecode</a></td>
<td><sup id="cite_ref-pmid11902836_55-0" class="reference"><a href="#cite_note-pmid11902836-55">&#91;49&#93;</a></sup><sup id="cite_ref-pmid15731207_56-0" class="reference"><a href="#cite_note-pmid15731207-56">&#91;50&#93;</a></sup><sup id="cite_ref-pmid17445273_57-0" class="reference"><a href="#cite_note-pmid17445273-57">&#91;51&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=FoldalignM&amp;action=edit&amp;redlink=1" class="new" title="FoldalignM (page does not exist)">FoldalignM</a>
</th>
<td>A multiple RNA structural RNA alignment method, to a large extent based on the PMcomp program.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://foldalign.ku.dk/software/">sourcecode</a></td>
<td><sup id="cite_ref-pmid17324941_58-0" class="reference"><a href="#cite_note-pmid17324941-58">&#91;52&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=FRUUT&amp;action=edit&amp;redlink=1" class="new" title="FRUUT (page does not exist)">FRUUT</a>
</th>
<td>A pairwise RNA structural alignment tool based on the comparison of RNA trees. Considers alignments in which the compared trees can be rooted differently (with respect to the standard “external loop” corresponding roots), and/or permuted with respect to branching order.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.cs.bgu.ac.il/~negevcb/FRUUT/code/FRUUT-2.40.jar">sourcecode</a>, <a rel="nofollow" class="external text" href="http://www.cs.bgu.ac.il/~negevcb/FRUUT/">webserver</a></td>
<td><sup id="cite_ref-978-3-642-33122-0_11_59-0" class="reference"><a href="#cite_note-978-3-642-33122-0_11-59">&#91;53&#93;</a></sup><sup id="cite_ref-pmid23590940_60-0" class="reference"><a href="#cite_note-pmid23590940-60">&#91;54&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=GraphClust&amp;action=edit&amp;redlink=1" class="new" title="GraphClust (page does not exist)">GraphClust</a>
</th>
<td>Fast RNA structural clustering method of local RNA secondary structures. Predicted clusters are refined using LocARNA and CMsearch. Due to the linear time complexity for clustering it is possible to analyse large RNA datasets.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.bioinf.uni-freiburg.de/Software/GraphClust/">sourcecode</a></td>
<td><sup id="cite_ref-pmid22689765_61-0" class="reference"><a href="#cite_note-pmid22689765-61">&#91;55&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=KNetFold&amp;action=edit&amp;redlink=1" class="new" title="KNetFold (page does not exist)">KNetFold</a>
</th>
<td>Computes a consensus RNA secondary structure from an RNA sequence alignment based on machine learning.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www-lmmb.ncifcrf.gov/~bshapiro/downloader_v1/register.php">linuxbinary</a>, <a rel="nofollow" class="external text" href="http://knetfold.abcc.ncifcrf.gov/">webserver</a></td>
<td><sup id="cite_ref-pmid16495232_62-0" class="reference"><a href="#cite_note-pmid16495232-62">&#91;56&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=LARA_(software)&amp;action=edit&amp;redlink=1" class="new" title="LARA (software) (page does not exist)">LARA</a>
</th>
<td>Produce a global fold and alignment of ncRNA families using integer linear programming and Lagrangian relaxation.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="https://www.mi.fu-berlin.de/w/LiSA/">sourcecode</a></td>
<td><sup id="cite_ref-pmid17662141_63-0" class="reference"><a href="#cite_note-pmid17662141-63">&#91;57&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=LocaRNA&amp;action=edit&amp;redlink=1" class="new" title="LocaRNA (page does not exist)">LocaRNA</a>
</th>
<td>LocaRNA is the successor of PMcomp with an improved time complexity. It is a variant of Sankoff's algorithm for simultaneous folding and alignment, which takes as input pre-computed base pair probability matrices from McCaskill's algorithm as produced by RNAfold -p. Thus the method can also be viewed as way to compare base pair probability matrices.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.bioinf.uni-freiburg.de/Software/LocARNA/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://rna.informatik.uni-freiburg.de:8080/LocARNA/">webserver</a></td>
<td><sup id="cite_ref-pmid17432929_64-0" class="reference"><a href="#cite_note-pmid17432929-64">&#91;58&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MASTR&amp;action=edit&amp;redlink=1" class="new" title="MASTR (page does not exist)">MASTR</a>
</th>
<td>A sampling approach using <a href="/wiki/Markov_chain_Monte_Carlo" title="Markov chain Monte Carlo">Markov chain Monte Carlo</a> in a <a href="/wiki/Simulated_annealing" title="Simulated annealing">simulated annealing</a> framework, where both structure and alignment is optimized by making small local changes. The score combines the log-likelihood of the alignment, a covariation term and the basepair probabilities.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://mastr.binf.ku.dk/">sourcecode</a></td>
<td><sup id="cite_ref-pmid17038338_65-0" class="reference"><a href="#cite_note-pmid17038338-65">&#91;59&#93;</a></sup><sup id="cite_ref-pmid18006551_66-0" class="reference"><a href="#cite_note-pmid18006551-66">&#91;60&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Multilign&amp;action=edit&amp;redlink=1" class="new" title="Multilign (page does not exist)">Multilign</a>
</th>
<td>This method uses multiple Dynalign calculations to find a low free energy structure common to any number of sequences. It does not require any sequence identity.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://rna.urmc.rochester.edu/RNAstructure.html">sourcecode</a></td>
<td><sup id="cite_ref-pmid21193521_67-0" class="reference"><a href="#cite_note-pmid21193521-67">&#91;61&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Murlet&amp;action=edit&amp;redlink=1" class="new" title="Murlet (page does not exist)">Murlet</a>
</th>
<td>a multiple alignment tool for RNA sequences using iterative alignment based on Sankoff's algorithm with sharply reduced computational time and memory.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://murlet.ncrna.org/murlet/murlet.html">webserver</a></td>
<td><sup id="cite_ref-pmid17459961_68-0" class="reference"><a href="#cite_note-pmid17459961-68">&#91;62&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MXSCARNA&amp;action=edit&amp;redlink=1" class="new" title="MXSCARNA (page does not exist)">MXSCARNA</a>
</th>
<td>a multiple alignment tool for RNA sequences using progressive alignment based on pairwise structural alignment algorithm of SCARNA.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://mxscarna.ncrna.org/mxscarna/mxscarna.html">webserver</a> <a rel="nofollow" class="external text" href="http://www.ncrna.org/software/mxscarna/download/">sourcecode</a></td>
<td><sup id="cite_ref-MXSCARNA_69-0" class="reference"><a href="#cite_note-MXSCARNA-69">&#91;63&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=PAliKiss&amp;action=edit&amp;redlink=1" class="new" title="PAliKiss (page does not exist)">pAliKiss</a>
</th>
<td>pAliKiss predicts RNA secondary structures for fixed RNA multiple sequence alignments, with special attention for pseudoknotted structures. This program is an offspring of the hybridization of RNAalishapes and pKiss.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://bibiserv2.cebitec.uni-bielefeld.de/palikiss">webserver</a> <a rel="nofollow" class="external text" href="http://bibiwsserv.cebitec.uni-bielefeld.de/fold-grammars/">sourcecode</a></td>
<td><sup id="cite_ref-pmid_pkiss_15-1" class="reference"><a href="#cite_note-pmid_pkiss-15">&#91;14&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=PARTS&amp;action=edit&amp;redlink=1" class="new" title="PARTS (page does not exist)">PARTS</a>
</th>
<td>A method for joint prediction of alignment and common secondary structures of two RNA sequences using a probabilistic model based on pseudo free energies obtained from precomputed base pairing and alignment probabilities.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">2</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://rna.urmc.rochester.edu">sourcecode</a></td>
<td><sup id="cite_ref-pmid18304945_70-0" class="reference"><a href="#cite_note-pmid18304945-70">&#91;64&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Pfold&amp;action=edit&amp;redlink=1" class="new" title="Pfold (page does not exist)">Pfold</a>
</th>
<td>Folds alignments using a SCFG trained on rRNA alignments.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial"><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle \leq 40}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mo>&#x2264;<!-- ≤ --></mo>
        <mn>40</mn>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle \leq 40}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/2f5b453dd1104e81703bd92e06377e959271af84" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.505ex; width:4.778ex; height:2.343ex;" alt="\leq 40"/></span></td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="https://web.archive.org/web/20120510145232/http://www.daimi.au.dk/~compbio/rnafold/">webserver</a></td>
<td><sup id="cite_ref-pmid10383470_71-0" class="reference"><a href="#cite_note-pmid10383470-71">&#91;65&#93;</a></sup><sup id="cite_ref-pmid12824339_72-0" class="reference"><a href="#cite_note-pmid12824339-72">&#91;66&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=PETfold&amp;action=edit&amp;redlink=1" class="new" title="PETfold (page does not exist)">PETfold</a>
</th>
<td>Formally integrates both the energy-based and evolution-based approaches in one model to predict the folding of multiple aligned RNA sequences by a maximum expected accuracy scoring. The structural probabilities are calculated by RNAfold and Pfold.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://genome.ku.dk/resources/petfold/">sourcecode</a></td>
<td><sup id="cite_ref-pmid18836192_73-0" class="reference"><a href="#cite_note-pmid18836192-73">&#91;67&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=PhyloQFold&amp;action=edit&amp;redlink=1" class="new" title="PhyloQFold (page does not exist)">PhyloQFold</a>
</th>
<td>Method that takes advantage of the evolutionary history of a group of aligned RNA sequences for sampling consensus secondary structures, including pseudoknots, according to their approximate posterior probability.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://evol.bio.lmu.de/_statgen/software/phyloqfold/">sourcecode</a></td>
<td><sup id="cite_ref-pmid22796961_74-0" class="reference"><a href="#cite_note-pmid22796961-74">&#91;68&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=PMcomp/PMmulti&amp;action=edit&amp;redlink=1" class="new" title="PMcomp/PMmulti (page does not exist)">PMcomp/PMmulti</a>
</th>
<td>PMcomp is a variant of Sankoff's algorithm for simultaneous folding and alignment, which takes as input pre-computed base pair probability matrices from McCaskill's algorithm as produced by RNAfold -p. Thus the method can also be viewed as way to compare base pair probability matrices. PMmulti is a wrapper program that does progressive multiple alignments by repeatedly calling pmcomp</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial"><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle 2\leq seqs\leq 6}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mn>2</mn>
        <mo>&#x2264;<!-- ≤ --></mo>
        <mi>s</mi>
        <mi>e</mi>
        <mi>q</mi>
        <mi>s</mi>
        <mo>&#x2264;<!-- ≤ --></mo>
        <mn>6</mn>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle 2\leq seqs\leq 6}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/58a2363e45f91a9603b3fe5b18382cd524f9426b" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:12.856ex; height:2.509ex;" alt="2\leq seqs\leq 6"/></span></td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/~ivo/RNA/PMcomp/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://rna.tbi.univie.ac.at/cgi-bin/pmcgi.pl">webserver</a></td>
<td><sup id="cite_ref-pmid15073017_75-0" class="reference"><a href="#cite_note-pmid15073017-75">&#91;69&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAG&amp;action=edit&amp;redlink=1" class="new" title="RNAG (page does not exist)">RNAG</a>
</th>
<td>A Gibbs sampling method to determine a conserved structure and the structural alignment.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://ccmbweb.ccv.brown.edu/rnag.html">sourcecode</a></td>
<td><sup id="cite_ref-pmid21788211_76-0" class="reference"><a href="#cite_note-pmid21788211-76">&#91;70&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/T-Coffee" title="T-Coffee">R-COFFEE</a>
</th>
<td>uses RNAlpfold to compute the secondary structure of the provided sequences. A modified version of <a href="/wiki/T-Coffee" title="T-Coffee">T-Coffee</a> is then used to compute the multiple sequence alignment having the best agreement with the sequences and the structures. R-Coffee can be combined with any existing sequence alignment method.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.tcoffee.org/Projects_home_page/r_coffee_home_page.html">sourcecode</a>, <a rel="nofollow" class="external text" href="http://www.tcoffee.org/">webserver</a></td>
<td><sup id="cite_ref-pmid18420654_77-0" class="reference"><a href="#cite_note-pmid18420654-77">&#91;71&#93;</a></sup><sup id="cite_ref-pmid18483080_78-0" class="reference"><a href="#cite_note-pmid18483080-78">&#91;72&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=TurboFold&amp;action=edit&amp;redlink=1" class="new" title="TurboFold (page does not exist)">TurboFold</a>
</th>
<td>This algorithm predicts conserved structures in any number of sequences.  It uses probabilistic alignment and partition functions to map conserved pairs between sequences, and then iterates the partition functions to improve structure prediction accuracy</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://rna.urmc.rochester.edu/RNAstructure.html">sourcecode</a></td>
<td><sup id="cite_ref-pmid21507242_79-0" class="reference"><a href="#cite_note-pmid21507242-79">&#91;73&#93;</a></sup><sup id="cite_ref-pmid22285566_80-0" class="reference"><a href="#cite_note-pmid22285566-80">&#91;74&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=R-scape&amp;action=edit&amp;redlink=1" class="new" title="R-scape (page does not exist)">R-scape</a>
</th>
<td>Verify conserved secondary structure by measuring covarying basepairs and their statistical significance compared to pure phylogeney. Will propose a most conserved ("optimized") one if no secondary structure is given.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://eddylab.org/R-scape/">home page</a></td>
<td><sup id="cite_ref-81" class="reference"><a href="#cite_note-81">&#91;75&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNA123&amp;action=edit&amp;redlink=1" class="new" title="RNA123 (page does not exist)">RNA123</a>
</th>
<td>Included structure based sequence alignment (SBSA) algorithm uses a novel suboptimal version of the Needleman-Wunsch global sequence alignment method that fully accounts for secondary structure in the template and query. It also uses two separate substitution matrices optimized for RNA helices and single stranded regions. The SBSA algorithm provides &gt;90% accurate sequence alignments even for structures as large as bacterial 23S rRNA: ~2,800 nts.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.rna123.com/">webserver</a></td>
<td>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAalifold&amp;action=edit&amp;redlink=1" class="new" title="RNAalifold (page does not exist)">RNAalifold</a>
</th>
<td>Folds precomputed alignments using mix of free-energy and covariation measures. Ships with the <a href="/wiki/ViennaRNA_Package" title="ViennaRNA Package">ViennaRNA Package</a>.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/~ivo/RNA/">homepage</a></td>
<td><sup id="cite_ref-RNAinverse_18-1" class="reference"><a href="#cite_note-RNAinverse-18">&#91;17&#93;</a></sup><sup id="cite_ref-pmid12079347_82-0" class="reference"><a href="#cite_note-pmid12079347-82">&#91;76&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAalishapes&amp;action=edit&amp;redlink=1" class="new" title="RNAalishapes (page does not exist)">RNAalishapes</a>
</th>
<td>Tool for secondary structure prediction for precomputed alignments using a mix of free-energy and a covariation measures. Output can be sifted by the abstract shapes concept to focus on major difference in suboptimal results.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://bibiwsserv.cebitec.uni-bielefeld.de/fold-grammars/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://bibiserv2.cebitec.uni-bielefeld.de/rnaalishapes">webserver</a></td>
<td><sup id="cite_ref-pmid17020924_83-0" class="reference"><a href="#cite_note-pmid17020924-83">&#91;77&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAcast&amp;action=edit&amp;redlink=1" class="new" title="RNAcast (page does not exist)">RNAcast</a>
</th>
<td>enumerates the near-optimal abstract shape space, and predicts as the consensus an abstract shape common to all sequences, and for each sequence, the thermodynamically best structure which has this abstract shape.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnacast/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnashapes/submission.html">webserver</a></td>
<td><sup id="cite_ref-pmid16020472_84-0" class="reference"><a href="#cite_note-pmid16020472-84">&#91;78&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAforester&amp;action=edit&amp;redlink=1" class="new" title="RNAforester (page does not exist)">RNAforester</a>
</th>
<td>Compare and align RNA secondary structures via a "forest alignment" approach.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnaforester/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnaforester/submission.html">webserver</a></td>
<td><sup id="cite_ref-pmid16452790_85-0" class="reference"><a href="#cite_note-pmid16452790-85">&#91;79&#93;</a></sup><sup id="cite_ref-pmid17048408_86-0" class="reference"><a href="#cite_note-pmid17048408-86">&#91;80&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAmine&amp;action=edit&amp;redlink=1" class="new" title="RNAmine (page does not exist)">RNAmine</a>
</th>
<td>Frequent stem pattern miner from unaligned RNA sequences is a software tool to extract the structural motifs from a set of RNA sequences.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://rnamine.ncrna.org/RNAMINE/">webserver</a></td>
<td><sup id="cite_ref-pmid16908501_87-0" class="reference"><a href="#cite_note-pmid16908501-87">&#91;81&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNASampler&amp;action=edit&amp;redlink=1" class="new" title="RNASampler (page does not exist)">RNASampler</a>
</th>
<td>A probabilistic sampling approach that combines intrasequence base pairing probabilities with intersequence base alignment probabilities. This is used to sample possible stems for each sequence and compare these stems between all pairs of sequences to predict a consensus structure for two sequences. The method is extended to predict the common structure conserved among multiple sequences by using a consistency-based score that incorporates information from all the pairwise structural alignments.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="https://web.archive.org/web/20100609233409/http://ural.wustl.edu/~xingxu/RNASampler/">sourcecode</a></td>
<td><sup id="cite_ref-pmid17537756_88-0" class="reference"><a href="#cite_note-pmid17537756-88">&#91;82&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=SCARNA&amp;action=edit&amp;redlink=1" class="new" title="SCARNA (page does not exist)">SCARNA</a>
</th>
<td>Stem Candidate Aligner for RNA (Scarna) is a fast, convenient tool for structural alignment of a pair of RNA sequences. It aligns two RNA sequences and calculates the similarities of them, based on the estimated common secondary structures. It works even for pseudoknotted secondary structures.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">2</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.scarna.org/scarna/">webserver</a></td>
<td><sup id="cite_ref-pmid16690634_89-0" class="reference"><a href="#cite_note-pmid16690634-89">&#91;83&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=SimulFold&amp;action=edit&amp;redlink=1" class="new" title="SimulFold (page does not exist)">SimulFold</a>
</th>
<td>simultaneously inferring RNA structures including pseudoknots, alignments, and trees using a Bayesian MCMC framework.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.cs.ubc.ca/~irmtraud/simulfold/">sourcecode</a></td>
<td><sup id="cite_ref-pmid17696604_90-0" class="reference"><a href="#cite_note-pmid17696604-90">&#91;84&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/Stemloc" title="Stemloc">Stemloc</a>
</th>
<td>a program for pairwise RNA structural alignment based on probabilistic models of RNA structure known as Pair <a href="/wiki/Stochastic_context-free_grammar" class="mw-redirect" title="Stochastic context-free grammar">stochastic context-free grammars</a>.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://biowiki.org/StemLoc">sourcecode</a></td>
<td><sup id="cite_ref-pmid15790387_91-0" class="reference"><a href="#cite_note-pmid15790387-91">&#91;85&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=StrAl&amp;action=edit&amp;redlink=1" class="new" title="StrAl (page does not exist)">StrAl</a>
</th>
<td>an alignment tool designed to provide multiple alignments of non-coding RNAs following a fast progressive strategy. It combines the thermodynamic base pairing information derived from RNAfold calculations in the form of base pairing probability vectors with the information of the primary sequence.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial"><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle \leq 50}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mo>&#x2264;<!-- ≤ --></mo>
        <mn>50</mn>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle \leq 50}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/56d04b76fce7ce1935a0d8bd014080424a308da3" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.505ex; width:4.778ex; height:2.343ex;" alt="\leq 50"/></span></td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.biophys.uni-duesseldorf.de/stral/about.php">sourcecode</a>, <a rel="nofollow" class="external text" href="http://www.biophys.uni-duesseldorf.de/stral/advancedForm.php">webserver</a></td>
<td><sup id="cite_ref-pmid16613908_92-0" class="reference"><a href="#cite_note-pmid16613908-92">&#91;86&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=TFold&amp;action=edit&amp;redlink=1" class="new" title="TFold (page does not exist)">TFold</a>
</th>
<td>A tool for predicting non-coding RNA secondary structures including pseudoknots. It takes in input an alignment of RNA sequences and returns the predicted secondary structure(s). It combines criteria of stability, conservation and covariation in order to search for stems and pseudoknots. Users can change different parameters values, set (or not) some known stems (if there are) which are taken into account by the system, choose to get several possible structures or only one, search for pseudoknots or not, etc.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://tfold.ibisc.univ-evry.fr:8080/TFold/">webserver</a></td>
<td><sup id="cite_ref-&#123;&#123;&#80;MID&#124;20047957&#125;&#125;_93-0" class="reference"><a href="#cite_note-{{PMID|20047957}}-93">&#91;87&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=WAR_(software)&amp;action=edit&amp;redlink=1" class="new" title="WAR (software) (page does not exist)">WAR</a>
</th>
<td>a webserver that makes it possible to simultaneously use a number of state of the art methods for performing multiple alignment and secondary structure prediction for noncoding RNA sequences.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial"><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle 2\leq seqs\leq 50}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mn>2</mn>
        <mo>&#x2264;<!-- ≤ --></mo>
        <mi>s</mi>
        <mi>e</mi>
        <mi>q</mi>
        <mi>s</mi>
        <mo>&#x2264;<!-- ≤ --></mo>
        <mn>50</mn>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle 2\leq seqs\leq 50}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/39d7a84b4ada6b7747eda67c7187fff0e07bafad" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:14.018ex; height:2.509ex;" alt="2\leq seqs\leq 50"/></span></td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://genome.ku.dk/resources/war/">webserver</a></td>
<td><sup id="cite_ref-pmid18492721_94-0" class="reference"><a href="#cite_note-pmid18492721-94">&#91;88&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/Xrate" title="Xrate">Xrate</a>
</th>
<td>a program for analysis of multiple sequence alignments using phylogenetic <a href="/wiki/Stochastic_context-free_grammar" class="mw-redirect" title="Stochastic context-free grammar">grammars</a>, that may be viewed as a flexible generalization of the "Pfold" program.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://biowiki.org/XrateSoftware">sourcecode</a></td>
<td><sup id="cite_ref-pmid17018148_95-0" class="reference"><a href="#cite_note-pmid17018148-95">&#91;89&#93;</a></sup>
</td></tr>
<tr>
<td colspan="8">
<dl><dt>Notes</dt>
<dd></dd></dl>
<div class="reflist" style="list-style-type: decimal;">
<div class="mw-references-wrap"><ol class="references">
<li id="cite_note-44"><span class="mw-cite-backlink"><b><a href="#cite_ref-44">^</a></b></span> <span class="reference-text"><b>Number of sequences:</b> &lt;any|num&gt;.</span>
</li>
<li id="cite_note-45"><span class="mw-cite-backlink"><b><a href="#cite_ref-45">^</a></b></span> <span class="reference-text"><b>Alignment:</b> predicts an <a href="/wiki/Sequence_alignment" title="Sequence alignment">alignment</a>, &lt;input|yes|no&gt;.</span>
</li>
<li id="cite_note-46"><span class="mw-cite-backlink"><b><a href="#cite_ref-46">^</a></b></span> <span class="reference-text"><b>Structure:</b> predicts <a href="/wiki/RNA_structure" class="mw-redirect" title="RNA structure">structure</a>, &lt;input|yes|no&gt;.</span>
</li>
<li id="cite_note-47"><span class="mw-cite-backlink"><b><a href="#cite_ref-47">^</a></b></span> <span class="reference-text"><b>Knots:</b> <a href="/wiki/Pseudoknot" title="Pseudoknot">Pseudoknot</a> prediction, &lt;yes|no&gt;.</span>
</li>
</ol></div></div>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="Intermolecular_interactions:_RNA-RNA">Intermolecular interactions: RNA-RNA</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=4" title="Edit section: Intermolecular interactions: RNA-RNA">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Many <a href="/wiki/NcRNA" class="mw-redirect" title="NcRNA">ncRNAs</a> function by binding to other <a href="/wiki/RNA" title="RNA">RNAs</a>. For example, <a href="/wiki/MiRNA" class="mw-redirect" title="MiRNA">miRNAs</a> regulate protein coding gene expression by binding to <a href="/wiki/Three_prime_untranslated_region" title="Three prime untranslated region">3' UTRs</a>, <a href="/wiki/SnoRNA" class="mw-redirect" title="SnoRNA">small nucleolar RNAs</a> guide post-transcriptional modifications by binding to <a href="/wiki/RRNA" class="mw-redirect" title="RRNA">rRNA</a>, <a href="/wiki/U4_spliceosomal_RNA" title="U4 spliceosomal RNA">U4 spliceosomal RNA</a> and <a href="/wiki/U6_spliceosomal_RNA" title="U6 spliceosomal RNA">U6 spliceosomal RNA</a> bind to each other forming part of the <a href="/wiki/Spliceosome" title="Spliceosome">spliceosome</a> and many small bacterial RNAs regulate gene expression by antisense interactions E.g. <a href="/wiki/GcvB_RNA" title="GcvB RNA">GcvB</a>, <a href="/wiki/OxyS_RNA" title="OxyS RNA">OxyS</a> and <a href="/wiki/RyhB_RNA" class="mw-redirect" title="RyhB RNA">RyhB</a>.
</p>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description</th>
<th>Intra-molecular structure</th>
<th>Comparative</th>
<th>Link</th>
<th>References
</th></tr>
<tr>
<th>RNApredator
</th>
<td>RNApredator uses a dynamic programming approach to compute RNA-RNA interaction sites.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://rna.tbi.univie.ac.at/RNApredator2">webserver</a></td>
<td><sup id="cite_ref-PMC3125805_96-0" class="reference"><a href="#cite_note-PMC3125805-96">&#91;90&#93;</a></sup>
</td></tr>
<tr>
<th>GUUGle
</th>
<td>A utility for fast determination of RNA-RNA matches with perfect hybridization via A-U, C-G, and G-U base pairing.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/guugle/">webserver</a></td>
<td><sup id="cite_ref-pmid16403789_97-0" class="reference"><a href="#cite_note-pmid16403789-97">&#91;91&#93;</a></sup>
</td></tr>
<tr>
<th>IntaRNA
</th>
<td>Efficient target prediction incorporating the accessibility of target sites.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.bioinf.uni-freiburg.de/Software/#IntaRNA-download">sourcecode</a> <a rel="nofollow" class="external text" href="http://rna.informatik.uni-freiburg.de/IntaRNA/">webserver</a></td>
<td><sup id="cite_ref-pmid28472523_98-0" class="reference"><a href="#cite_note-pmid28472523-98">&#91;92&#93;</a></sup><sup id="cite_ref-pmid24838564_99-0" class="reference"><a href="#cite_note-pmid24838564-99">&#91;93&#93;</a></sup><sup id="cite_ref-pmid18940824_100-0" class="reference"><a href="#cite_note-pmid18940824-100">&#91;94&#93;</a></sup><sup id="cite_ref-pmid19850757_101-0" class="reference"><a href="#cite_note-pmid19850757-101">&#91;95&#93;</a></sup><sup id="cite_ref-pmid20444875_102-0" class="reference"><a href="#cite_note-pmid20444875-102">&#91;96&#93;</a></sup>
</td></tr>
<tr>
<th>CopraRNA
</th>
<td>Tool for sRNA target prediction. It computes whole genome predictions by mix of distinct whole genome IntaRNA predictions.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.bioinf.uni-freiburg.de/Software/#CopraRNA-download">sourcecode</a> <a rel="nofollow" class="external text" href="http://rna.informatik.uni-freiburg.de/CopraRNA/">webserver</a></td>
<td><sup id="cite_ref-pmid23980183_103-0" class="reference"><a href="#cite_note-pmid23980183-103">&#91;97&#93;</a></sup><sup id="cite_ref-pmid24838564_99-1" class="reference"><a href="#cite_note-pmid24838564-99">&#91;93&#93;</a></sup>
</td></tr>
<tr>
<th>MINT
</th>
<td>Automatic tool to analyze three-dimensional structures of RNA and DNA molecules, their full-atom molecular dynamics trajectories or other conformation sets (e.g. X-ray or NMR-derived structures). For each RNA or DNA conformation MINT determines the hydrogen bonding network resolving the base pairing patterns, identifies secondary structure motifs (helices, junctions, loops, etc.) and pseudoknots. Also estimates the energy of stacking and phosphate anion-base interactions.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="https://github.com/gvalchca/MINT">sourcecode</a> <a rel="nofollow" class="external text" href="http://mint.cent.uw.edu.pl/index.php?strona=MintRun">webserver</a></td>
<td><sup id="cite_ref-MINT-RNA-NAR_104-0" class="reference"><a href="#cite_note-MINT-RNA-NAR-104">&#91;98&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/NUPACK" title="NUPACK">NUPACK</a>
</th>
<td>Computes the full unpseudoknotted partition function of interacting strands in dilute solution. Calculates the concentrations, mfes, and base-pairing probabilities of the ordered complexes below a certain complexity. Also computes the partition function and basepairing of single strands including a class of pseudoknotted structures. Also enables design of ordered complexes.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://nupack.org/">NUPACK</a></td>
<td><sup id="cite_ref-NUPACK_105-0" class="reference"><a href="#cite_note-NUPACK-105">&#91;99&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=OligoWalk/RNAstructure&amp;action=edit&amp;redlink=1" class="new" title="OligoWalk/RNAstructure (page does not exist)">OligoWalk/RNAstructure</a>
</th>
<td>Predicts bimolecular secondary structures with and without intramolecular structure.  Also predicts the hybridization affinity of a short nucleic acid to an RNA target.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external autonumber" href="http://rna.urmc.rochester.edu">[1]</a></td>
<td><sup id="cite_ref-oligowalk_106-0" class="reference"><a href="#cite_note-oligowalk-106">&#91;100&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/PiRNA_(software)" class="mw-redirect" title="PiRNA (software)">piRNA</a>
</th>
<td>Calculates the partition function and thermodynamics of RNA-RNA interactions. It considers all possible joint secondary structure of two interacting nucleic acids that do not contain pseudoknots, interaction pseudoknots, or zigzags.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://compbio.cs.sfu.ca/taverna/pirna/">linuxbinary</a></td>
<td><sup id="cite_ref-piRNA_107-0" class="reference"><a href="#cite_note-piRNA-107">&#91;101&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAripalign_(software)&amp;action=edit&amp;redlink=1" class="new" title="RNAripalign (software) (page does not exist)">RNAripalign</a>
</th>
<td>Calculates the partition function and thermodynamics of RNA-RNA interactions based on structural alignments. Also supports RNA-RNA interaction prediction for single sequences. It outputs suboptimal structures based on Boltzmann distribution. It considers all possible joint secondary structure of two interacting nucleic acids that do not contain pseudoknots, interaction pseudoknots, or zigzags.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external autonumber" href="http://www.bioinf.uni-leipzig.de/~qin/resources/ripalign.tar.gz">[2]</a></td>
<td><sup id="cite_ref-RNAripalign_108-0" class="reference"><a href="#cite_note-RNAripalign-108">&#91;102&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RactIP&amp;action=edit&amp;redlink=1" class="new" title="RactIP (page does not exist)">RactIP</a>
</th>
<td>Fast and accurate prediction of RNA-RNA interaction using integer programming.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="https://github.com/satoken/ractip">sourcecode</a> <a rel="nofollow" class="external text" href="http://rtips.dna.bio.keio.ac.jp/ractip/">webserver</a></td>
<td><sup id="cite_ref-pmid20823308_109-0" class="reference"><a href="#cite_note-pmid20823308-109">&#91;103&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAaliduplex&amp;action=edit&amp;redlink=1" class="new" title="RNAaliduplex (page does not exist)">RNAaliduplex</a>
</th>
<td>Based on RNAduplex with bonuses for covarying sites</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/~ivo/RNA/">sourcecode</a></td>
<td><sup id="cite_ref-RNAinverse_18-2" class="reference"><a href="#cite_note-RNAinverse-18">&#91;17&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAcofold&amp;action=edit&amp;redlink=1" class="new" title="RNAcofold (page does not exist)">RNAcofold</a>
</th>
<td>Works much like RNAfold, but allows specifying two RNA sequences which are then allowed to form a dimer structure.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/~ivo/RNA/">sourcecode</a></td>
<td><sup id="cite_ref-RNAinverse_18-3" class="reference"><a href="#cite_note-RNAinverse-18">&#91;17&#93;</a></sup><sup id="cite_ref-pmid16722605_110-0" class="reference"><a href="#cite_note-pmid16722605-110">&#91;104&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAduplex&amp;action=edit&amp;redlink=1" class="new" title="RNAduplex (page does not exist)">RNAduplex</a>
</th>
<td>Computes optimal and suboptimal secondary structures for hybridization. The calculation is simplified by allowing only inter-molecular base pairs.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/~ivo/RNA/">sourcecode</a></td>
<td><sup id="cite_ref-RNAinverse_18-4" class="reference"><a href="#cite_note-RNAinverse-18">&#91;17&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAhybrid&amp;action=edit&amp;redlink=1" class="new" title="RNAhybrid (page does not exist)">RNAhybrid</a>
</th>
<td>Tool to find the minimum free energy hybridisation of a long and a short RNA.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnahybrid/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnahybrid/submission.html">webserver</a></td>
<td><sup id="cite_ref-pmid15383676_111-0" class="reference"><a href="#cite_note-pmid15383676-111">&#91;105&#93;</a></sup><sup id="cite_ref-pmid16845047_112-0" class="reference"><a href="#cite_note-pmid16845047-112">&#91;106&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAup&amp;action=edit&amp;redlink=1" class="new" title="RNAup (page does not exist)">RNAup</a>
</th>
<td>Calculates the thermodynamics of RNA-RNA interactions. RNA-RNA binding is decomposed into two stages. (1) First the probability that a sequence interval (e.g. a binding site) remains unpaired is computed. (2) Then the binding energy given that the binding site is unpaired is calculated as the optimum over all possible types of bindings.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/~ivo/RNA/">sourcecode</a></td>
<td><sup id="cite_ref-RNAinverse_18-5" class="reference"><a href="#cite_note-RNAinverse-18">&#91;17&#93;</a></sup><sup id="cite_ref-pmid16446276_113-0" class="reference"><a href="#cite_note-pmid16446276-113">&#91;107&#93;</a></sup>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="Intermolecular_interactions:_MicroRNA:any_RNA">Intermolecular interactions: MicroRNA:any RNA</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=5" title="Edit section: Intermolecular interactions: MicroRNA:any RNA">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>The below table includes interactions that are not limited to UTRs.  
</p>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description</th>
<th>Cross-species</th>
<th>Intra-molecular structure</th>
<th>Comparative</th>
<th>Link</th>
<th>References
</th></tr>
<tr>
<th><a href="/w/index.php?title=ComTAR&amp;action=edit&amp;redlink=1" class="new" title="ComTAR (page does not exist)">comTAR</a>
</th>
<td>A a web tool for the prediction of miRNA targets that is mainly based on the conservation of the potential regulation in plant species.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://rnabiology.ibr-conicet.gov.ar/comtar/">Web tool</a></td>
<td><sup id="cite_ref-114" class="reference"><a href="#cite_note-114">&#91;108&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/RNA22" title="RNA22">RNA22</a>
</th>
<td>The first link (precomputed predictions) provides RNA22 predictions for all protein coding transcripts in human, mouse, roundworm, and fruit fly. It allows visualizing the predictions within a cDNA map and also find transcripts where multiple miR's of interest target. The second web-site link (interactive/custom sequences) first finds putative microRNA binding sites in the sequence of interest, then identifies the targeted microRNA. Both tools are provided by the <a rel="nofollow" class="external text" href="http://cm.jefferson.edu/">Computational Medicine Center</a> at <a rel="nofollow" class="external text" href="http://www.jefferson.edu">Thomas Jefferson University</a>.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://cm.jefferson.edu/rna22v2.0/">precomputed predictions</a> <a rel="nofollow" class="external text" href="http://cm.jefferson.edu/rna22v2/">interactive/custom sequences</a></td>
<td><sup id="cite_ref-pmid16990141_115-0" class="reference"><a href="#cite_note-pmid16990141-115">&#91;109&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAhybrid&amp;action=edit&amp;redlink=1" class="new" title="RNAhybrid (page does not exist)">RNAhybrid</a>
</th>
<td>Tool to find the minimum free energy hybridisation of a long and a short RNA.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnahybrid/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnahybrid/submission.html">webserver</a></td>
<td><sup id="cite_ref-pmid15383676_111-1" class="reference"><a href="#cite_note-pmid15383676-111">&#91;105&#93;</a></sup><sup id="cite_ref-pmid16845047_112-1" class="reference"><a href="#cite_note-pmid16845047-112">&#91;106&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MiRBooking&amp;action=edit&amp;redlink=1" class="new" title="MiRBooking (page does not exist)">miRBooking</a>
</th>
<td>Simulates the stochiometric mode of action of microRNAs using a derivative of the <a href="/wiki/Gale-Shapley_algorithm" class="mw-redirect" title="Gale-Shapley algorithm">Gale-Shapley algorithm</a> for finding a stable set of duplexes. It uses quantifications for traversing the set of mRNA and microRNA pairs and seed complementarity for ranking and assigning sites.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="https://major.iric.ca/mirbooking/">sourcecode</a>, <a rel="nofollow" class="external text" href="https://major.iric.ca/~poirigui/mirbooking-scan/">webserver</a>
</td>
<td><sup id="cite_ref-116" class="reference"><a href="#cite_note-116">&#91;110&#93;</a></sup>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="Intermolecular_interactions:_MicroRNA:UTR">Intermolecular interactions: MicroRNA:UTR</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=6" title="Edit section: Intermolecular interactions: MicroRNA:UTR">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p><a href="/wiki/MiRNA" class="mw-redirect" title="MiRNA">MicroRNAs</a> regulate protein coding gene expression by binding to <a href="/wiki/Three_prime_untranslated_region" title="Three prime untranslated region">3' UTRs</a>, there are tools specifically designed for predicting these interactions. For an evaluation of target prediction methods on high-throughput experimental data see (Baek <i>et al.</i>, Nature 2008),<sup id="cite_ref-pmid18668037_117-0" class="reference"><a href="#cite_note-pmid18668037-117">&#91;111&#93;</a></sup> (Alexiou <i>et al.</i>, Bioinformatics 2009),<sup id="cite_ref-pmid19789267_118-0" class="reference"><a href="#cite_note-pmid19789267-118">&#91;112&#93;</a></sup> or (Ritchie et al., Nature Methods 2009)<sup id="cite_ref-pmid19478799_119-0" class="reference"><a href="#cite_note-pmid19478799-119">&#91;113&#93;</a></sup>
</p>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description</th>
<th>Cross-species</th>
<th>Intra-molecular structure</th>
<th>Comparative</th>
<th>Link</th>
<th>References
</th></tr>
<tr>
<th>Cupid
</th>
<td>Method for <i>simultaneous prediction of miRNA-target interactions and their mediated competing endogenous RNA (ceRNA) interactions</i>. It is an integrative approach significantly improves on miRNA-target prediction accuracy as assessed by both mRNA and protein level measurements in breast cancer cell lines. Cupid is implemented in 3 steps: Step 1: re-evaluate candidate miRNA binding sites in 3’ UTRs. Step2: interactions are predicted by integrating information about selected sites and the statistical dependency between the expression profiles of miRNA and putative targets. Step 3: Cupid assesses whether inferred targets compete for predicted miRNA regulators.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">human</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://cupidtool.sourceforge.net">software (MATLAB)</a></td>
<td><sup id="cite_ref-Chiu_et._al,_Genome_Res_2015_120-0" class="reference"><a href="#cite_note-Chiu_et._al,_Genome_Res_2015-120">&#91;114&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Diana-microT&amp;action=edit&amp;redlink=1" class="new" title="Diana-microT (page does not exist)">Diana-microT</a>
</th>
<td>Version 3.0 is an algorithm based on several parameters calculated individually for each microRNA and it combines conserved and non-conserved microRNA recognition elements into a final prediction score.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">human, mouse</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="https://web.archive.org/web/20101208180159/http://diana.cslab.ece.ntua.gr/microT/">webserver</a></td>
<td><sup id="cite_ref-pmid19765283_121-0" class="reference"><a href="#cite_note-pmid19765283-121">&#91;115&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MicroTar&amp;action=edit&amp;redlink=1" class="new" title="MicroTar (page does not exist)">MicroTar</a>
</th>
<td>An animal miRNA target prediction tool based on miRNA-target complementarity and thermodynamic data.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://tiger.dbs.nus.edu.sg/microtar/">sourcecode</a></td>
<td><sup id="cite_ref-pmid17254305_122-0" class="reference"><a href="#cite_note-pmid17254305-122">&#91;116&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MiTarget&amp;action=edit&amp;redlink=1" class="new" title="MiTarget (page does not exist)">miTarget</a>
</th>
<td>microRNA target gene prediction using a support vector machine.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://cbit.snu.ac.kr/~miTarget/">webserver</a></td>
<td><sup id="cite_ref-pmid16978421_123-0" class="reference"><a href="#cite_note-pmid16978421-123">&#91;117&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MiRror&amp;action=edit&amp;redlink=1" class="new" title="MiRror (page does not exist)">miRror</a>
</th>
<td>Based on the notion of a combinatorial regulation by an ensemble of miRNAs or genes. miRror integrates predictions from a dozen of miRNA resources that are based on complementary algorithms into a unified statistical framework</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.proto.cs.huji.ac.il/mirror/index.php">webserver</a></td>
<td><sup id="cite_ref-124" class="reference"><a href="#cite_note-124">&#91;118&#93;</a></sup><sup id="cite_ref-125" class="reference"><a href="#cite_note-125">&#91;119&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=PicTar&amp;action=edit&amp;redlink=1" class="new" title="PicTar (page does not exist)">PicTar</a>
</th>
<td>Combinatorial microRNA target predictions.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">8 vertebrates</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="https://web.archive.org/web/20080724163022/http://pictar.bio.nyu.edu/">predictions</a></td>
<td><sup id="cite_ref-pmid15806104_126-0" class="reference"><a href="#cite_note-pmid15806104-126">&#91;120&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=PITA_(software)&amp;action=edit&amp;redlink=1" class="new" title="PITA (software) (page does not exist)">PITA</a>
</th>
<td>Incorporates the role of target-site accessibility, as determined by base-pairing interactions within the mRNA, in microRNA target recognition.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://genie.weizmann.ac.il/pubs/mir07/mir07_exe.html">executable</a>, <a rel="nofollow" class="external text" href="http://genie.weizmann.ac.il/pubs/mir07/mir07_prediction.html">webserver</a>, <a rel="nofollow" class="external text" href="http://genie.weizmann.ac.il/pubs/mir07/mir07_data.html">predictions</a></td>
<td><sup id="cite_ref-pmid17893677_127-0" class="reference"><a href="#cite_note-pmid17893677-127">&#91;121&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/RNA22" title="RNA22">RNA22</a>
</th>
<td>The first link (precomputed predictions) provides RNA22 predictions for all protein coding transcripts in human, mouse, roundworm, and fruit fly. It allows visualizing the predictions within a cDNA map and also find transcripts where multiple miR's of interest target. The second web-site link (interactive/custom sequences) first finds putative microRNA binding sites in the sequence of interest, then identifies the targeted microRNA. Both tools are provided by the <a rel="nofollow" class="external text" href="http://cm.jefferson.edu/">Computational Medicine Center</a> at <a rel="nofollow" class="external text" href="http://www.jefferson.edu">Thomas Jefferson University</a>.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://cm.jefferson.edu/rna22v2.0/">precomputed predictions</a> <a rel="nofollow" class="external text" href="http://cm.jefferson.edu/rna22v2/">interactive/custom sequences</a></td>
<td><sup id="cite_ref-pmid16990141_115-1" class="reference"><a href="#cite_note-pmid16990141-115">&#91;109&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAhybrid&amp;action=edit&amp;redlink=1" class="new" title="RNAhybrid (page does not exist)">RNAhybrid</a>
</th>
<td>Tool to find the minimum free energy hybridisation of a long and a short RNA.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnahybrid/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnahybrid/submission.html">webserver</a></td>
<td><sup id="cite_ref-pmid15383676_111-2" class="reference"><a href="#cite_note-pmid15383676-111">&#91;105&#93;</a></sup><sup id="cite_ref-pmid16845047_112-2" class="reference"><a href="#cite_note-pmid16845047-112">&#91;106&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Sylamer&amp;action=edit&amp;redlink=1" class="new" title="Sylamer (page does not exist)">Sylamer</a>
</th>
<td>Method to find significantly over or under-represented words in sequences according to a sorted gene list. Usually used to find significant enrichment or depletion of microRNA or siRNA seed sequences from microarray expression data.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/enright/sylamer/">sourcecode</a> <a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/enright/sylarray/">webserver</a></td>
<td><sup id="cite_ref-pmid18978784_128-0" class="reference"><a href="#cite_note-pmid18978784-128">&#91;122&#93;</a></sup><sup id="cite_ref-pmid20871108_129-0" class="reference"><a href="#cite_note-pmid20871108-129">&#91;123&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=TAREF&amp;action=edit&amp;redlink=1" class="new" title="TAREF (page does not exist)">TAREF</a>
</th>
<td>TARget REFiner (TAREF) predicts microRNA targets on the basis of multiple feature information derived from the flanking regions of the predicted target sites where traditional structure prediction approach may not be successful to assess the openness. It also provides an option to use encoded pattern to refine filtering.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://scbb.ihbt.res.in/TAREF/programchoice.html">server/sourcecode</a></td>
<td><sup id="cite_ref-pmid=20413915_130-0" class="reference"><a href="#cite_note-pmid=20413915-130">&#91;124&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=P-TAREF&amp;action=edit&amp;redlink=1" class="new" title="P-TAREF (page does not exist)">p-TAREF</a>
</th>
<td>plant TARget REFiner (p-TAREF) identifies plant microRNA targets on the basis of multiple feature information derived from the flanking regions of the predicted target sites where traditional structure prediction approach may not be successful to assess the openness. It also provides an option to use encoded pattern to refine filtering.  It first time employed power of machine learning approach with scoring scheme through <i>support vector regression</i> (SVR) while considering structural and alignment aspects of targeting in plants with plant specific models. p-TAREF has been implemented in concurrent architecture in server and standalone form, making it one of the very few available target identification tools able to run concurrently on simple desktops while performing huge transcriptome level analysis accurately and fast. Also provides option to experimentally validate the predicted targets, on the spot, using expression data, which has been integrated in its back-end, to draw confidence on prediction along with SVR score.p-TAREF performance benchmarking has been done extensively through different tests and compared with other plant miRNA target identification tools. p-TAREF was found to perform better.</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://scbb.ihbt.res.in/SCBB_dept/Software.php">server/standalone</a></td>
<td>
</td></tr>
<tr>
<th><a href="/wiki/TargetScan" title="TargetScan">TargetScan</a>
</th>
<td>Predicts biological targets of miRNAs by searching for the presence of sites that match the seed region of each miRNA. In flies and nematodes, predictions are ranked based on the probability of their evolutionary conservation.  In zebrafish, predictions are ranked based on site number, site type, and site context, which includes factors that influence target-site accessibility.  In mammals, the user can choose whether the predictions should be ranked based on the probability of their conservation or on site number, type, and context.  In mammals and nematodes, the user can choose to extend predictions beyond conserved sites and consider all sites.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">vertebrates, flies, nematodes</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">evaluated indirectly</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.targetscan.org/cgi-bin/targetscan/data_download.cgi?db=vert_60">sourcecode</a>, <a rel="nofollow" class="external text" href="http://www.targetscan.org/">webserver</a></td>
<td><sup id="cite_ref-pmid14697198_131-0" class="reference"><a href="#cite_note-pmid14697198-131">&#91;125&#93;</a></sup><sup id="cite_ref-pmid15652477_132-0" class="reference"><a href="#cite_note-pmid15652477-132">&#91;126&#93;</a></sup><sup id="cite_ref-pmid17612493_133-0" class="reference"><a href="#cite_note-pmid17612493-133">&#91;127&#93;</a></sup><sup id="cite_ref-pmid21909094_134-0" class="reference"><a href="#cite_note-pmid21909094-134">&#91;128&#93;</a></sup><sup id="cite_ref-135" class="reference"><a href="#cite_note-135">&#91;129&#93;</a></sup><sup id="cite_ref-136" class="reference"><a href="#cite_note-136">&#91;130&#93;</a></sup>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="ncRNA_gene_prediction_software">ncRNA gene prediction software</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=7" title="Edit section: ncRNA gene prediction software">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description
</th>
<th>Number of sequences<br /><sup id="cite_ref-137" class="reference"><a href="#cite_note-137">&#91;Note 1&#93;</a></sup>
</th>
<th>Alignment<br /><sup id="cite_ref-138" class="reference"><a href="#cite_note-138">&#91;Note 2&#93;</a></sup>
</th>
<th>Structure<br /><sup id="cite_ref-139" class="reference"><a href="#cite_note-139">&#91;Note 3&#93;</a></sup>
</th>
<th>Link</th>
<th>References
</th></tr>
<tr>
<th><a href="/w/index.php?title=Alifoldz&amp;action=edit&amp;redlink=1" class="new" title="Alifoldz (page does not exist)">Alifoldz</a>
</th>
<td>Assessing a multiple sequence alignment for the existence of an unusual stable and conserved RNA secondary structure.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/papers/SUPPLEMENTS/Alifoldz/">sourcecode</a></td>
<td><sup id="cite_ref-pmid15313604_140-0" class="reference"><a href="#cite_note-pmid15313604-140">&#91;131&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=EvoFold&amp;action=edit&amp;redlink=1" class="new" title="EvoFold (page does not exist)">EvoFold</a>
</th>
<td>a comparative method for identifying functional RNA structures in multiple-sequence alignments. It is based on a probabilistic model-construction called a phylo-SCFG and exploits the characteristic differences of the substitution process in stem-pairing and unpaired regions to make its predictions.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.cbse.ucsc.edu/~jsp/EvoFold/">linuxbinary</a></td>
<td><sup id="cite_ref-pmid16628248_141-0" class="reference"><a href="#cite_note-pmid16628248-141">&#91;132&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=GraphClust&amp;action=edit&amp;redlink=1" class="new" title="GraphClust (page does not exist)">GraphClust</a>
</th>
<td>Fast RNA structural clustering method to identify common (local) RNA secondary structures. Predicted structural clusters are presented as alignment. Due to the linear time complexity for clustering it is possible to analyse large RNA datasets.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.bioinf.uni-freiburg.de/Software/GraphClust/">sourcecode</a></td>
<td><sup id="cite_ref-pmid22689765_61-1" class="reference"><a href="#cite_note-pmid22689765-61">&#91;55&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MSARi&amp;action=edit&amp;redlink=1" class="new" title="MSARi (page does not exist)">MSARi</a>
</th>
<td>heuristic search for statistically significant conservation of RNA secondary structure in deep multiple sequence alignments.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://theory.csail.mit.edu/MSARi">sourcecode</a></td>
<td><sup id="cite_ref-142" class="reference"><a href="#cite_note-142">&#91;133&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=QRNA&amp;action=edit&amp;redlink=1" class="new" title="QRNA (page does not exist)">QRNA</a>
</th>
<td>This is the code from Elena Rivas that accompanies a submitted manuscript "<i>Noncoding RNA gene detection using comparative sequence analysis</i>". QRNA uses comparative genome sequence analysis to detect conserved RNA secondary structures, including both ncRNA genes and cis-regulatory RNA structures.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">2</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://selab.janelia.org/software.html">sourcecode</a></td>
<td><sup id="cite_ref-pmid11801179_143-0" class="reference"><a href="#cite_note-pmid11801179-143">&#91;134&#93;</a></sup><sup id="cite_ref-pmid11553332_144-0" class="reference"><a href="#cite_note-pmid11553332-144">&#91;135&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAz&amp;action=edit&amp;redlink=1" class="new" title="RNAz (page does not exist)">RNAz</a>
</th>
<td>program for predicting structurally conserved and thermodynamic stable RNA secondary structures in multiple sequence alignments. It can be used in genome wide screens to detect functional RNA structures, as found in noncoding RNAs and cis-acting regulatory elements of mRNAs.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">input</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/~wash/RNAz/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://rna.tbi.univie.ac.at/cgi-bin/RNAz.cgi">webserver</a> <a rel="nofollow" class="external text" href="http://psb.stanford.edu/psb-online/proceedings/psb10/abstracts/2010_p69.html">RNAz 2</a></td>
<td><sup id="cite_ref-pmid15665081_145-0" class="reference"><a href="#cite_note-pmid15665081-145">&#91;136&#93;</a></sup><sup id="cite_ref-pmid17452347_146-0" class="reference"><a href="#cite_note-pmid17452347-146">&#91;137&#93;</a></sup><sup id="cite_ref-pmid17993695_147-0" class="reference"><a href="#cite_note-pmid17993695-147">&#91;138&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=ScanFold&amp;action=edit&amp;redlink=1" class="new" title="ScanFold (page does not exist)">ScanFold</a>
</th>
<td>A program for predicting unique local RNA structures in large sequences with unusually stable folding.</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">1</td>
<td style="background: #FFD; color: black; vertical-align: middle; text-align: center;" class="partial table-partial">None</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="https://github.com/moss-lab/ScanFold">sourcecode</a> <a rel="nofollow" class="external text" href="https://mosslabtools.bb.iastate.edu/scanfold">webserver</a></td>
<td><sup id="cite_ref-148" class="reference"><a href="#cite_note-148">&#91;139&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/Xrate" title="Xrate">Xrate</a>
</th>
<td>a program for analysis of multiple sequence alignments using phylogenetic <a href="/wiki/Stochastic_context-free_grammar" class="mw-redirect" title="Stochastic context-free grammar">grammars</a>, that may be viewed as a flexible generalization of the "Evofold" program.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">any</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://biowiki.org/XrateSoftware">sourcecode</a></td>
<td><sup id="cite_ref-pmid17018148_95-1" class="reference"><a href="#cite_note-pmid17018148-95">&#91;89&#93;</a></sup>
</td></tr>
<tr>
<td colspan="7">
<dl><dt>Notes</dt>
<dd></dd></dl>
<div class="reflist" style="list-style-type: decimal;">
<div class="mw-references-wrap"><ol class="references">
<li id="cite_note-137"><span class="mw-cite-backlink"><b><a href="#cite_ref-137">^</a></b></span> <span class="reference-text"><b>Number of sequences:</b> &lt;any|num&gt;.</span>
</li>
<li id="cite_note-138"><span class="mw-cite-backlink"><b><a href="#cite_ref-138">^</a></b></span> <span class="reference-text"><b>Alignment:</b> predicts an <a href="/wiki/Sequence_alignment" title="Sequence alignment">alignment</a>, &lt;input|yes|no&gt;.</span>
</li>
<li id="cite_note-139"><span class="mw-cite-backlink"><b><a href="#cite_ref-139">^</a></b></span> <span class="reference-text"><b>Structure:</b> predicts <a href="/wiki/RNA_structure" class="mw-redirect" title="RNA structure">structure</a>, &lt;input|yes|no&gt;.</span>
</li>
</ol></div></div>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="Family_specific_gene_prediction_software">Family specific gene prediction software</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=8" title="Edit section: Family specific gene prediction software">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description</th>
<th>Family</th>
<th>Link</th>
<th>References
</th></tr>
<tr>
<th>ARAGORN
</th>
<td>ARAGORN detects tRNA and tmRNA in nucleotide sequences.</td>
<td><a href="/wiki/TRNA" class="mw-redirect" title="TRNA">tRNA</a> <a href="/wiki/TmRNA" class="mw-redirect" title="TmRNA">tmRNA</a></td>
<td><a rel="nofollow" class="external text" href="http://130.235.46.10/ARAGORN/">webserver</a> <a rel="nofollow" class="external text" href="http://130.235.46.10/ARAGORN/aragorn1.2.28.c">source</a></td>
<td><sup id="cite_ref-pmid14704338_149-0" class="reference"><a href="#cite_note-pmid14704338-149">&#91;140&#93;</a></sup>
</td></tr>
<tr>
<th>miReader
</th>
<td>miReader is a first of its type to detect mature miRNAs with no dependence on genomic or reference sequences. So far, discovering miRNAs was possible only with species for which genomic or reference sequences would be available as most of the miRNA discovery tools relied on drawing pre-miRNA candidates. Due to this, miRNA biology became limited to model organisms, mostly. miReader allows directly discerning mature miRNAs from small RNA sequencing data, with no need of genomic-reference sequences. It has been developed for many Phyla and species, from vertebrate to plant models. Its accuracy has been found to be consistently &gt;90% in heavy validatory testing.</td>
<td><a href="/w/index.php?title=Mature_miRNA&amp;action=edit&amp;redlink=1" class="new" title="Mature miRNA (page does not exist)">mature miRNA</a></td>
<td><a rel="nofollow" class="external text" href="http://scbb.ihbt.res.in/2810-12/miReader.php">webserver/source</a> <a rel="nofollow" class="external text" href="http://sourceforge.net/projects/mireader/">webserver/source</a></td>
<td><sup id="cite_ref-pmid23805282_150-0" class="reference"><a href="#cite_note-pmid23805282-150">&#91;141&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MiRNAminer&amp;action=edit&amp;redlink=1" class="new" title="MiRNAminer (page does not exist)">miRNAminer</a>
</th>
<td>Given a search query, candidate homologs are identified using BLAST search and then tested for their known miRNA properties, such as secondary structure, energy, alignment and conservation, in order to assess their fidelity.</td>
<td><a href="/wiki/MicroRNA" title="MicroRNA">MicroRNA</a></td>
<td><a rel="nofollow" class="external text" href="http://groups.csail.mit.edu/pag/mirnaminer/">webserver</a></td>
<td><sup id="cite_ref-pmid18215311_151-0" class="reference"><a href="#cite_note-pmid18215311-151">&#91;142&#93;</a></sup>
</td></tr>
<tr>
<th>RISCbinder
</th>
<td>Prediction of guide strand of microRNAs.</td>
<td><a href="/wiki/MicroRNA" title="MicroRNA">Mature miRNA</a></td>
<td><a rel="nofollow" class="external text" href="http://crdd.osdd.net:8081/RISCbinder/">webserver</a></td>
<td><sup id="cite_ref-riscbinder_152-0" class="reference"><a href="#cite_note-riscbinder-152">&#91;143&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAmicro&amp;action=edit&amp;redlink=1" class="new" title="RNAmicro (page does not exist)">RNAmicro</a>
</th>
<td>A SVM-based approach that, in conjunction with a non-stringent filter for consensus secondary structures, is capable of recognizing microRNA precursors in multiple sequence alignments.</td>
<td><a href="/wiki/MicroRNA" title="MicroRNA">MicroRNA</a></td>
<td><a rel="nofollow" class="external text" href="http://www.bioinf.uni-leipzig.de/~jana/software/RNAmicro.html">homepage</a></td>
<td><sup id="cite_ref-pmid16873472_153-0" class="reference"><a href="#cite_note-pmid16873472-153">&#91;144&#93;</a></sup>
</td></tr>
<tr>
<th>RNAmmer
</th>
<td>RNAmmer uses <a href="/wiki/HMMER" title="HMMER">HMMER</a> to annotate <a href="/wiki/RRNA" class="mw-redirect" title="RRNA">rRNA</a> genes in genome sequences. Profiles were built using alignments from the European ribosomal RNA database<sup id="cite_ref-pmid14681368_154-0" class="reference"><a href="#cite_note-pmid14681368-154">&#91;145&#93;</a></sup> and the 5S Ribosomal RNA Database.<sup id="cite_ref-pmid11752286_155-0" class="reference"><a href="#cite_note-pmid11752286-155">&#91;146&#93;</a></sup></td>
<td><a href="/wiki/RRNA" class="mw-redirect" title="RRNA">rRNA</a></td>
<td><a rel="nofollow" class="external text" href="http://www.cbs.dtu.dk/services/RNAmmer/">webserver</a> <a rel="nofollow" class="external text" href="http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?rnammer">source</a></td>
<td><sup id="cite_ref-pmid17452365_156-0" class="reference"><a href="#cite_note-pmid17452365-156">&#91;147&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=SnoReport&amp;action=edit&amp;redlink=1" class="new" title="SnoReport (page does not exist)">SnoReport</a>
</th>
<td>Uses a mix of RNA secondary structure prediction and machine learning that is designed to recognize the two major classes of snoRNAs, box C/D and box H/ACA snoRNAs, among ncRNA candidate sequences.</td>
<td><a href="/wiki/SnoRNA" class="mw-redirect" title="SnoRNA">snoRNA</a></td>
<td><a rel="nofollow" class="external text" href="http://www.bioinf.uni-leipzig.de/~jana/software/SnoReport.html">sourcecode</a></td>
<td><sup id="cite_ref-pmid17895272_157-0" class="reference"><a href="#cite_note-pmid17895272-157">&#91;148&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=SnoScan&amp;action=edit&amp;redlink=1" class="new" title="SnoScan (page does not exist)">SnoScan</a>
</th>
<td>Search for C/D box methylation guide snoRNA genes in a genomic sequence.</td>
<td><a href="/wiki/SnoRNA" class="mw-redirect" title="SnoRNA">C/D box snoRNA</a></td>
<td><a rel="nofollow" class="external text" href="http://lowelab.ucsc.edu/snoscan/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://lowelab.ucsc.edu/snoscan/">webserver</a></td>
<td><sup id="cite_ref-pmid10024243_158-0" class="reference"><a href="#cite_note-pmid10024243-158">&#91;149&#93;</a></sup><sup id="cite_ref-pmid15980563_159-0" class="reference"><a href="#cite_note-pmid15980563-159">&#91;150&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=TRNAscan-SE&amp;action=edit&amp;redlink=1" class="new" title="TRNAscan-SE (page does not exist)">tRNAscan-SE</a>
</th>
<td>a program for the detection of transfer RNA genes in genomic sequence.</td>
<td><a href="/wiki/TRNA" class="mw-redirect" title="TRNA">tRNA</a></td>
<td><a rel="nofollow" class="external text" href="http://lowelab.ucsc.edu/tRNAscan-SE/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://lowelab.ucsc.edu/tRNAscan-SE/">webserver</a></td>
<td><sup id="cite_ref-pmid15980563_159-1" class="reference"><a href="#cite_note-pmid15980563-159">&#91;150&#93;</a></sup><sup id="cite_ref-pmid9023104_160-0" class="reference"><a href="#cite_note-pmid9023104-160">&#91;151&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MiRNAFold&amp;action=edit&amp;redlink=1" class="new" title="MiRNAFold (page does not exist)">miRNAFold</a>
</th>
<td>A fast ab initio software for searching for microRNA precursors in genomes.</td>
<td><a href="/wiki/MicroRNA" title="MicroRNA">microRNA</a></td>
<td><a rel="nofollow" class="external text" href="http://EvryRNA.ibisc.univ-evry.fr/">webserver</a></td>
<td><sup id="cite_ref-pmid22362754_161-0" class="reference"><a href="#cite_note-pmid22362754-161">&#91;152&#93;</a></sup>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="RNA_homology_search_software">RNA homology search software</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=9" title="Edit section: RNA homology search software">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description</th>
<th>Link</th>
<th>References
</th></tr>
<tr>
<th><a href="/w/index.php?title=ERPIN&amp;action=edit&amp;redlink=1" class="new" title="ERPIN (page does not exist)">ERPIN</a>
</th>
<td>"Easy RNA Profile IdentificatioN" is an RNA motif search program reads a sequence alignment and secondary structure, and automatically infers a statistical "secondary structure profile" (SSP). An original Dynamic Programming algorithm then matches this SSP onto any target database, finding solutions and their associated scores.</td>
<td><a rel="nofollow" class="external text" href="http://rna.igmors.u-psud.fr/erpin/">sourcecode</a> <a rel="nofollow" class="external text" href="http://tagc.univ-mrs.fr/erpin/">webserver</a></td>
<td><sup id="cite_ref-pmid11700055_162-0" class="reference"><a href="#cite_note-pmid11700055-162">&#91;153&#93;</a></sup><sup id="cite_ref-pmid15215371_163-0" class="reference"><a href="#cite_note-pmid15215371-163">&#91;154&#93;</a></sup><sup id="cite_ref-pmid15892887_164-0" class="reference"><a href="#cite_note-pmid15892887-164">&#91;155&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Infernal_(software)&amp;action=edit&amp;redlink=1" class="new" title="Infernal (software) (page does not exist)">Infernal</a>
</th>
<td>"INFERence of RNA ALignment" is for searching DNA sequence databases for RNA structure and sequence similarities. It is an implementation of a special case of profile stochastic context-free grammars called covariance models (CMs).</td>
<td><a rel="nofollow" class="external text" href="http://infernal.janelia.org/">sourcecode</a></td>
<td><sup id="cite_ref-pmid17397253_165-0" class="reference"><a href="#cite_note-pmid17397253-165">&#91;156&#93;</a></sup><sup id="cite_ref-pmid12095421_166-0" class="reference"><a href="#cite_note-pmid12095421-166">&#91;157&#93;</a></sup><sup id="cite_ref-pmid8029015_167-0" class="reference"><a href="#cite_note-pmid8029015-167">&#91;158&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=GraphClust&amp;action=edit&amp;redlink=1" class="new" title="GraphClust (page does not exist)">GraphClust</a>
</th>
<td>Fast RNA structural clustering method to identify common (local) RNA secondary structures. Predicted structural clusters are presented as alignment. Due to the linear time complexity for clustering it is possible to analyse large RNA datasets.</td>
<td><a rel="nofollow" class="external text" href="http://www.bioinf.uni-freiburg.de/Software/GraphClust/">sourcecode</a></td>
<td><sup id="cite_ref-pmid22689765_61-2" class="reference"><a href="#cite_note-pmid22689765-61">&#91;55&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=PHMMTS&amp;action=edit&amp;redlink=1" class="new" title="PHMMTS (page does not exist)">PHMMTS</a>
</th>
<td>"pair hidden Markov models on tree structures" is an extension of pair hidden Markov models defined on alignments of trees.</td>
<td><a rel="nofollow" class="external text" href="http://phmmts.dna.bio.keio.ac.jp/">sourcecode</a>, <a rel="nofollow" class="external text" href="http://phmmts.dna.bio.keio.ac.jp/">webserver</a></td>
<td><sup id="cite_ref-pmid16204111_168-0" class="reference"><a href="#cite_note-pmid16204111-168">&#91;159&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RaveNnA&amp;action=edit&amp;redlink=1" class="new" title="RaveNnA (page does not exist)">RaveNnA</a>
</th>
<td>A slow and rigorous or fast and heuristic sequence-based filter for covariance models.</td>
<td><a rel="nofollow" class="external text" href="http://bliss.biology.yale.edu/~zasha/ravenna/">sourcecode</a></td>
<td><sup id="cite_ref-pmid15262817_169-0" class="reference"><a href="#cite_note-pmid15262817-169">&#91;160&#93;</a></sup><sup id="cite_ref-pmid16267089_170-0" class="reference"><a href="#cite_note-pmid16267089-170">&#91;161&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RSEARCH&amp;action=edit&amp;redlink=1" class="new" title="RSEARCH (page does not exist)">RSEARCH</a>
</th>
<td>Takes one RNA sequence with its secondary structure and uses a local alignment algorithm to search a database for homologous RNAs.</td>
<td><a rel="nofollow" class="external text" href="ftp://selab.janelia.org/pub/software/rsearch/">sourcecode</a></td>
<td><sup id="cite_ref-pmid14499004_171-0" class="reference"><a href="#cite_note-pmid14499004-171">&#91;162&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Structator&amp;action=edit&amp;redlink=1" class="new" title="Structator (page does not exist)">Structator</a>
</th>
<td>Ultra fast software for searching for RNA structural motifs employing an innovative index-based bidirectional matching algorithm combined with a new fast fragment chaining strategy.</td>
<td><a rel="nofollow" class="external text" href="https://github.com/fernandomeyer/Structator">sourcecode</a></td>
<td><sup id="cite_ref-pmid21619640_172-0" class="reference"><a href="#cite_note-pmid21619640-172">&#91;163&#93;</a></sup>
</td></tr>
<tr>
<th>RaligNAtor
</th>
<td>Fast online and index-based algorithms for approximate search of RNA sequence-structure patterns
</td>
<td><a rel="nofollow" class="external text" href="https://github.com/fernandomeyer/RaligNAtor">sourcecode</a>
</td>
<td><sup id="cite_ref-173" class="reference"><a href="#cite_note-173">&#91;164&#93;</a></sup>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="Benchmarks">Benchmarks</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=10" title="Edit section: Benchmarks">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description
</th>
<th>Structure<sup id="cite_ref-174" class="reference"><a href="#cite_note-174">&#91;Note 1&#93;</a></sup>
</th>
<th>Alignment<sup id="cite_ref-175" class="reference"><a href="#cite_note-175">&#91;Note 2&#93;</a></sup>
</th>
<th>Phylogeny</th>
<th>Links</th>
<th>References
</th></tr>
<tr>
<th><a href="/w/index.php?title=BRalibase&amp;action=edit&amp;redlink=1" class="new" title="BRalibase (page does not exist)">BRalibase</a> I
</th>
<td>A comprehensive comparison of comparative RNA structure prediction approaches</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://projects.binf.ku.dk/pgardner/bralibase/bralibase1.html">data</a></td>
<td><sup id="cite_ref-pmid15458580_176-0" class="reference"><a href="#cite_note-pmid15458580-176">&#91;165&#93;</a></sup>
</td></tr>
<tr>
<th>BRalibase II
</th>
<td>A benchmark of multiple sequence alignment programs upon structural RNAs</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://projects.binf.ku.dk/pgardner/bralibase/bralibase2.html">data</a></td>
<td><sup id="cite_ref-pmid15860779_177-0" class="reference"><a href="#cite_note-pmid15860779-177">&#91;166&#93;</a></sup>
</td></tr>
<tr>
<th>BRalibase 2.1
</th>
<td>A benchmark of multiple sequence alignment programs upon structural RNAs</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.biophys.uni-duesseldorf.de/bralibase/">data</a></td>
<td><sup id="cite_ref-pmid17062125_178-0" class="reference"><a href="#cite_note-pmid17062125-178">&#91;167&#93;</a></sup>
</td></tr>
<tr>
<th>BRalibase III
</th>
<td>A critical assessment of the performance of homology search methods on noncoding RNA</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://projects.binf.ku.dk/pgardner/bralibase/bralibase3">data</a></td>
<td><sup id="cite_ref-pmid17151342_179-0" class="reference"><a href="#cite_note-pmid17151342-179">&#91;168&#93;</a></sup>
</td></tr>
<tr>
<th>CompaRNA
</th>
<td>An independent comparison of single-sequence and comparative methods for RNA secondary structure prediction</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://comparna.amu.edu.pl">AMU mirror</a> or <a rel="nofollow" class="external text" href="http://iimcb.genesilico.pl/comparna/">IIMCB mirror</a></td>
<td><sup id="cite_ref-pmid23435231_180-0" class="reference"><a href="#cite_note-pmid23435231-180">&#91;169&#93;</a></sup>
</td></tr>
<tr>
<th>RNAconTest
</th>
<td>A test of RNA multiple sequence alignments based entirely on known three dimensional RNA structures</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://DECIPHER.codes/Downloads.html">data</a></td>
<td><sup id="cite_ref-pmid32005745_181-0" class="reference"><a href="#cite_note-pmid32005745-181">&#91;170&#93;</a></sup>
</td></tr>
<tr>
<td colspan="7">
<dl><dt>Notes</dt>
<dd></dd></dl>
<div class="reflist" style="list-style-type: decimal;">
<div class="mw-references-wrap"><ol class="references">
<li id="cite_note-174"><span class="mw-cite-backlink"><b><a href="#cite_ref-174">^</a></b></span> <span class="reference-text"><b>Structure:</b> benchmarks <a href="/wiki/RNA_structure" class="mw-redirect" title="RNA structure">structure</a> prediction tools &lt;yes|no&gt;.</span>
</li>
<li id="cite_note-175"><span class="mw-cite-backlink"><b><a href="#cite_ref-175">^</a></b></span> <span class="reference-text"><b>Alignment:</b>  benchmarks <a href="/wiki/Sequence_alignment" title="Sequence alignment">alignment</a> tools &lt;yes|no&gt;.</span>
</li>
</ol></div></div>
</td></tr></tbody></table>
<h2><span id="Alignment_viewers.2C_editors"></span><span class="mw-headline" id="Alignment_viewers,_editors">Alignment viewers, editors</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=11" title="Edit section: Alignment viewers, editors">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description
</th>
<th>Alignment<sup id="cite_ref-182" class="reference"><a href="#cite_note-182">&#91;Note 1&#93;</a></sup>
</th>
<th>Structure<sup id="cite_ref-183" class="reference"><a href="#cite_note-183">&#91;Note 2&#93;</a></sup>
</th>
<th>Link</th>
<th>References
</th></tr>
<tr>
<th><a href="/w/index.php?title=4sale&amp;action=edit&amp;redlink=1" class="new" title="4sale (page does not exist)">4sale</a>
</th>
<td>A tool for Synchronous RNA Sequence and Secondary Structure Alignment and Editing</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://4sale.bioapps.biozentrum.uni-wuerzburg.de/">sourcecode</a></td>
<td><sup id="cite_ref-pmid17101042_184-0" class="reference"><a href="#cite_note-pmid17101042-184">&#91;171&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Colorstock&amp;action=edit&amp;redlink=1" class="new" title="Colorstock (page does not exist)">Colorstock</a>, <a href="/w/index.php?title=SScolor&amp;action=edit&amp;redlink=1" class="new" title="SScolor (page does not exist)">SScolor</a>, <a href="/w/index.php?title=Raton_(software)&amp;action=edit&amp;redlink=1" class="new" title="Raton (software) (page does not exist)">Raton</a>
</th>
<td>Colorstock, a command-line script using ANSI terminal color; SScolor, a <a href="/wiki/Perl" title="Perl">Perl</a> script that generates static HTML pages; and Raton, an <a href="/wiki/Ajax_(programming)" title="Ajax (programming)">Ajax</a> web application generating dynamic HTML. Each tool can be used to color RNA alignments by secondary structure and to visually highlight compensatory mutations in stems.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://biowiki.org/RNAAlignmentViewers">sourcecode</a></td>
<td><sup id="cite_ref-colorstock_185-0" class="reference"><a href="#cite_note-colorstock-185">&#91;172&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/Integrated_Genome_Browser" title="Integrated Genome Browser">Integrated Genome Browser</a> (IGB)
</th>
<td>Multiple alignment viewer written in <a href="/wiki/Java_(programming_language)" title="Java (programming language)">Java</a>.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://genoviz.sourceforge.net/">sourcecode</a></td>
<td><sup id="cite_ref-binf_igb_paper_186-0" class="reference"><a href="#cite_note-binf_igb_paper-186">&#91;173&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/wiki/Jalview" title="Jalview">Jalview</a>
</th>
<td>Multiple alignment editor written in <a href="/wiki/Java_(programming_language)" title="Java (programming language)">Java</a>.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#F99;vertical-align:middle;text-align:center;" class="table-no">No</td>
<td><a rel="nofollow" class="external text" href="http://www.jalview.org/">sourcecode</a></td>
<td><sup id="cite_ref-pmid19151095_187-0" class="reference"><a href="#cite_note-pmid19151095-187">&#91;174&#93;</a></sup><sup id="cite_ref-pmid14960472_188-0" class="reference"><a href="#cite_note-pmid14960472-188">&#91;175&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RALEE&amp;action=edit&amp;redlink=1" class="new" title="RALEE (page does not exist)">RALEE</a>
</th>
<td>a major mode for the <a href="/wiki/Emacs" title="Emacs">Emacs</a> text editor. It provides functionality to aid the viewing and editing of multiple sequence alignments of structured RNAs.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://personalpages.manchester.ac.uk/staff/sam.griffiths-jones/software/ralee/">sourcecode</a></td>
<td><sup id="cite_ref-pmid15377506_189-0" class="reference"><a href="#cite_note-pmid15377506-189">&#91;176&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=SARSE&amp;action=edit&amp;redlink=1" class="new" title="SARSE (page does not exist)">SARSE</a>
</th>
<td>A graphical sequence editor for working with structural alignments of RNA.</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td style="background:#9F9;vertical-align:middle;text-align:center;" class="table-yes">Yes</td>
<td><a rel="nofollow" class="external text" href="http://sarse.kvl.dk/">sourcecode</a></td>
<td><sup id="cite_ref-pmid17804647_190-0" class="reference"><a href="#cite_note-pmid17804647-190">&#91;177&#93;</a></sup>
</td></tr>
<tr>
<td colspan="6">
<dl><dt>Notes</dt>
<dd></dd></dl>
<div class="reflist" style="list-style-type: decimal;">
<div class="mw-references-wrap"><ol class="references">
<li id="cite_note-182"><span class="mw-cite-backlink"><b><a href="#cite_ref-182">^</a></b></span> <span class="reference-text"><b>Alignment:</b> view and edit an <a href="/wiki/Sequence_alignment" title="Sequence alignment">alignment</a>, &lt;yes|no&gt;.</span>
</li>
<li id="cite_note-183"><span class="mw-cite-backlink"><b><a href="#cite_ref-183">^</a></b></span> <span class="reference-text"><b>Structure:</b> view and edit <a href="/wiki/RNA_structure" class="mw-redirect" title="RNA structure">structure</a>, &lt;yes|no&gt;.</span>
</li>
</ol></div></div>
</td></tr></tbody></table>
<h2><span id="Inverse_folding.2C_RNA_design"></span><span class="mw-headline" id="Inverse_folding,_RNA_design">Inverse folding, RNA design</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=12" title="Edit section: Inverse folding, RNA design">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description</th>
<th>Link</th>
<th>References
</th></tr>
<tr>
<th></th>
<th>Single state design</th>
<th></th>
<th>
</th></tr>
<tr>
<th><a href="/wiki/EteRNA" title="EteRNA">EteRNA</a>/<a href="/w/index.php?title=EteRNABot&amp;action=edit&amp;redlink=1" class="new" title="EteRNABot (page does not exist)">EteRNABot</a>
</th>
<td>An RNA folding game that challenges players to make sequences that fold into a target RNA structure. The best sequences for a given puzzle are synthesized and their structures are probed through chemical mapping. The sequences are then scored by the data's agreement to the target structure and feedback is provided to the players. EteRNABot is a software implementation based on design rules submitted by EteRNA players.</td>
<td><a rel="nofollow" class="external text" href="http://eterna.cmu.edu/web/">EteRNA Game</a> <a rel="nofollow" class="external text" href="http://eternabot.org">EteRNABot web server</a></td>
<td><sup id="cite_ref-EteRNA_191-0" class="reference"><a href="#cite_note-EteRNA-191">&#91;178&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAinverse&amp;action=edit&amp;redlink=1" class="new" title="RNAinverse (page does not exist)">RNAinverse</a>
</th>
<td>The <a href="/wiki/ViennaRNA_Package" title="ViennaRNA Package">ViennaRNA Package</a> provides RNAinverse, an algorithm for designing sequences with desired structure.</td>
<td><a rel="nofollow" class="external text" href="http://www.viennarna.at/cgi-bin/RNAinverse.cgi">Web Server</a></td>
<td><sup id="cite_ref-RNAinverse_18-6" class="reference"><a href="#cite_note-RNAinverse-18">&#91;17&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAiFold&amp;action=edit&amp;redlink=1" class="new" title="RNAiFold (page does not exist)">RNAiFold</a>
</th>
<td>A complete RNA inverse folding approach based on <a href="/wiki/Constraint_programming" title="Constraint programming">constraint programming</a> and implemented using <a href="/wiki/Google_Developers#OR_Tools" title="Google Developers">OR Tools</a> which allows for the specification of a wide range of design constraints. The RNAiFold software provides two algorithms to solve the inverse folding problem: i) RNA-CPdesign explores the complete search space and ii) RNA-LNSdesign based on the <a href="/wiki/Very_large-scale_neighborhood_search" title="Very large-scale neighborhood search">large neighborhood search</a> <a href="/wiki/Metaheuristic" title="Metaheuristic">metaheuristic</a> is suitable to design large structures. The software can also design interacting RNA molecules using RNAcofold of the <a href="/wiki/ViennaRNA_Package" title="ViennaRNA Package">ViennaRNA Package</a>. A fully functional, <a rel="nofollow" class="external text" href="http://bioinformatics.bc.edu/clotelab/RNAiFold/">earlier implementation</a> using COMET is available.</td>
<td><a rel="nofollow" class="external text" href="http://bioinformatics.bc.edu/clotelab/RNAiFold2.0/">Web Server</a> <a rel="nofollow" class="external text" href="http://bioinformatics.bc.edu/clotelab/RNAiFold2.0/index.php?tab=downloads">Source Code</a></td>
<td><sup id="cite_ref-RNAiFold_192-0" class="reference"><a href="#cite_note-RNAiFold-192">&#91;179&#93;</a></sup><sup id="cite_ref-RNAiFoldWeb_193-0" class="reference"><a href="#cite_note-RNAiFoldWeb-193">&#91;180&#93;</a></sup><sup id="cite_ref-RNAiFold2Web_194-0" class="reference"><a href="#cite_note-RNAiFold2Web-194">&#91;181&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNA-SSD&amp;action=edit&amp;redlink=1" class="new" title="RNA-SSD (page does not exist)">RNA-SSD</a>/<a href="/w/index.php?title=RNA_Designer&amp;action=edit&amp;redlink=1" class="new" title="RNA Designer (page does not exist)">RNA Designer</a>
</th>
<td>The RNA-SSD (RNA Secondary Structure Designer) approach first assigns bases probabilistically to each position based probabilistic models. Subsequently, a stochastic local search is used to optimize this sequence. RNA-SSD is publicly available under the name of RNA Designer at the RNASoft web page</td>
<td><a rel="nofollow" class="external text" href="http://www.rnasoft.ca/rnadesign_about.html">Web Server</a></td>
<td><sup id="cite_ref-RNA-SSD_195-0" class="reference"><a href="#cite_note-RNA-SSD-195">&#91;182&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=INFO-RNA&amp;action=edit&amp;redlink=1" class="new" title="INFO-RNA (page does not exist)">INFO-RNA</a>
</th>
<td>INFO-RNA uses a <a href="/wiki/Dynamic_programming" title="Dynamic programming">dynamic programming</a> approach to generate an energy optimized starting sequence that is subsequently further improved by a stochastic local search that uses an effective neighbor selection method.</td>
<td><a rel="nofollow" class="external text" href="http://www.bioinf.uni-freiburg.de/Software/INFO-RNA/">Web Server</a> <a rel="nofollow" class="external text" href="http://www.bioinf.uni-freiburg.de//Software/index.html?en#INFORNA-download">Source Code</a></td>
<td><sup id="cite_ref-INFO-RNA_196-0" class="reference"><a href="#cite_note-INFO-RNA-196">&#91;183&#93;</a></sup><sup id="cite_ref-INFO-RNA-WebServer_197-0" class="reference"><a href="#cite_note-INFO-RNA-WebServer-197">&#91;184&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAexinv&amp;action=edit&amp;redlink=1" class="new" title="RNAexinv (page does not exist)">RNAexinv</a>
</th>
<td>RNAexinv is an extension of RNAinverse to generate sequences that not only fold into a desired structure, but they should also exhibit selected attributes such as thermodynamic stability and mutational robustness. This approach does not necessarily outputs a sequence that perfectly fits the input structure, but a shape abstraction, i.e. it keeps the adjacency and nesting of structural elements, but disregards helix lengths and the exact number unpaired positions, of it.</td>
<td><a rel="nofollow" class="external text" href="http://www.cs.bgu.ac.il/~RNAexinv/">Source Code</a></td>
<td><sup id="cite_ref-RNAexinv_198-0" class="reference"><a href="#cite_note-RNAexinv-198">&#91;185&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNA-ensign&amp;action=edit&amp;redlink=1" class="new" title="RNA-ensign (page does not exist)">RNA-ensign</a>
</th>
<td>This approach applies an efficient <a href="/w/index.php?title=Global_sampling&amp;action=edit&amp;redlink=1" class="new" title="Global sampling (page does not exist)">global sampling</a> algorithm to examine the mutational landscape under structural and thermodynamical constraints. The authors show that the global sampling approach is more robust, succeeds more often and generates more thermodynamically stable sequences than local approaches do.</td>
<td><a rel="nofollow" class="external text" href="http://www.mybiosoftware.com/rna-analysis/12897">Source Code</a></td>
<td><sup id="cite_ref-RNA-ensign_199-0" class="reference"><a href="#cite_note-RNA-ensign-199">&#91;186&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=IncaRNAtion&amp;action=edit&amp;redlink=1" class="new" title="IncaRNAtion (page does not exist)">IncaRNAtion</a>
</th>
<td>Successor of RNA-ensign that can specifically design sequences with a specified GC content using a GC-weighted Boltzmann ensemble and stochastic <a href="/wiki/Backtracking" title="Backtracking">backtracking</a></td>
<td><a rel="nofollow" class="external text" href="https://github.com/McGill-CSB/IncaRNAtion">Source Code</a></td>
<td><sup id="cite_ref-IncaRNAtion_200-0" class="reference"><a href="#cite_note-IncaRNAtion-200">&#91;187&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=DSS-Opt&amp;action=edit&amp;redlink=1" class="new" title="DSS-Opt (page does not exist)">DSS-Opt</a>
</th>
<td>Dynamics in Sequence Space Optimization (DSS-Opt) uses <a href="/wiki/Newtonian_dynamics" title="Newtonian dynamics">Newtonian dynamics</a> in the sequence space, with a negative design term and <a href="/wiki/Simulated_annealing" title="Simulated annealing">simulated annealing</a> to optimize a sequence such that it folds into the desired secondary structure.</td>
<td><a rel="nofollow" class="external text" href="https://github.com/marcom/dss-opt">Source Code</a></td>
<td><sup id="cite_ref-DSS-Opt_201-0" class="reference"><a href="#cite_note-DSS-Opt-201">&#91;188&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=MODENA&amp;action=edit&amp;redlink=1" class="new" title="MODENA (page does not exist)">MODENA</a>
</th>
<td>This approach interprets RNA inverse folding as a multi-objective optimization problem and solves it using a genetic algorithm. In its extended version MODENA is able to design pseudoknotted RNA structures with the aid of IPknot.</td>
<td><a rel="nofollow" class="external text" href="http://rna.eit.hirosaki-u.ac.jp/modena/">Source Code</a></td>
<td><sup id="cite_ref-MODENA_202-0" class="reference"><a href="#cite_note-MODENA-202">&#91;189&#93;</a></sup><sup id="cite_ref-MODENA_E_203-0" class="reference"><a href="#cite_note-MODENA_E-203">&#91;190&#93;</a></sup>
</td></tr>
<tr>
<th>ERD
</th>
<td>Evolutionary RNA Design (<a rel="nofollow" class="external text" href="http://bioinformatics.oxfordjournals.org/content/early/2014/01/08/bioinformatics.btu001.abstract">ERD</a>) can be used to design RNA sequences that fold into a given target structure. Any RNA secondary structure contains different structural components, each having a different length. Therefore, in the first step, the RNA subsequences (pools) corresponding to different components with different lengths are reconstructed. Using these pools, ERD reconstructs an initial RNA sequence which is compatible with the given target structure. Then ERD uses an evolutionary algorithm to improve the quality of the subsequences corresponding to the components. The major contributions of ERD are using the natural RNA sequences, a different method to evaluate the sequences in each population, and a different hierarchical decomposition of the target structure into smaller substructures.</td>
<td><a rel="nofollow" class="external text" href="http://mostafa.ut.ac.ir/corna/erd-cons/">Web Server</a> <a rel="nofollow" class="external text" href="http://mostafa.ut.ac.ir/corna/erd-cons/Download.php">Source Code</a></td>
<td><sup id="cite_ref-ERD_204-0" class="reference"><a href="#cite_note-ERD-204">&#91;191&#93;</a></sup>
</td></tr>
<tr>
<th>antaRNA
</th>
<td>Uses an underlying ant colony foraging heuristic terrain modeling to solve the inverse folding problem. The designed RNA sequences show high compliance to input structural and sequence constraints. Most prominently, also the GC value of the designed sequence can be regulated with high precision. GC value distribution sampling of solution sets is possible and sequence domain specific definition of multiple GC values within one entity. Due to the flexible evaluation of the intermediate sequences using underlying programs such as RNAfold, pKiss, or also HotKnots and IPKnot, RNA secondary nested structures and also pseudoknot structures of H- and K-type are feasible to solve with this approach.</td>
<td><a rel="nofollow" class="external text" href="http://rna.informatik.uni-freiburg.de/AntaRNA/">Web Server</a> <a rel="nofollow" class="external text" href="http://www.bioinf.uni-freiburg.de/Software/antaRNA/">Source Code</a></td>
<td><sup id="cite_ref-antaRNA_basis_205-0" class="reference"><a href="#cite_note-antaRNA_basis-205">&#91;192&#93;</a></sup><sup id="cite_ref-antaRNA_pk_206-0" class="reference"><a href="#cite_note-antaRNA_pk-206">&#91;193&#93;</a></sup>
</td></tr>
<tr>
<th></th>
<th>Dual state design</th>
<th></th>
<th>
</th></tr>
<tr>
<th><a href="/w/index.php?title=Switch.pl&amp;action=edit&amp;redlink=1" class="new" title="Switch.pl (page does not exist)">switch.pl</a>
</th>
<td>The <a href="/wiki/ViennaRNA_Package" title="ViennaRNA Package">ViennaRNA Package</a> provides a <a href="/wiki/Perl" title="Perl">Perl</a> script to design RNA sequences that can adopt two states. For instance <a href="/wiki/RNA_thermometer" title="RNA thermometer">RNA thermometer</a>, which change their structural state depending on the environmental temperature, have been successfully designed using this program.</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/RNA/switch.html">Man Page</a> <a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/RNA/">Source Code</a></td>
<td><sup id="cite_ref-switch.pl_207-0" class="reference"><a href="#cite_note-switch.pl-207">&#91;194&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RiboMaker&amp;action=edit&amp;redlink=1" class="new" title="RiboMaker (page does not exist)">RiboMaker</a>
</th>
<td>Intended to design <a href="/wiki/Small_RNA" title="Small RNA">small RNAs</a> (sRNA) and their target mRNA's 5'UTR. The sRNA is designed to activate or repress protein expression of the mRNA. It is also possible to design just one of the two RNA components provided the other sequence is fixed.</td>
<td><a rel="nofollow" class="external text" href="http://ribomaker.jaramillolab.org/">Web Server</a> <a rel="nofollow" class="external text" href="http://sourceforge.net/projects/ribomaker/">Source Code</a></td>
<td><sup id="cite_ref-RiboMaker_208-0" class="reference"><a href="#cite_note-RiboMaker-208">&#91;195&#93;</a></sup>
</td></tr>
<tr>
<th></th>
<th>Multi state design</th>
<th></th>
<th>
</th></tr>
<tr>
<th><a href="/w/index.php?title=RNAblueprint&amp;action=edit&amp;redlink=1" class="new" title="RNAblueprint (page does not exist)">RNAblueprint</a>
</th>
<td>This C++ library is based on the <a href="/w/index.php?title=RNAdesign&amp;action=edit&amp;redlink=1" class="new" title="RNAdesign (page does not exist)">RNAdesign</a> multiple target sampling algorithm. It brings a <a href="/wiki/SWIG" title="SWIG">SWIG</a> interface for <a href="/wiki/Perl" title="Perl">Perl</a> and <a href="/wiki/Python_(programming_language)" title="Python (programming language)">Python</a> which allows for an effortless integration into various tools. Therefore, multiple target sequence sampling can be combined with many optimization techniques and objective functions.</td>
<td><a rel="nofollow" class="external text" href="https://github.com/ViennaRNA/RNAblueprint">Source Code</a></td>
<td><sup id="cite_ref-RNAblueprint_209-0" class="reference"><a href="#cite_note-RNAblueprint-209">&#91;196&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=RNAdesign&amp;action=edit&amp;redlink=1" class="new" title="RNAdesign (page does not exist)">RNAdesign</a>
</th>
<td>The underlying algorithm is based on a mix of graph coloring and heuristic local optimization to find sequences can adapt multiple prescribed conformations. The software can also use of RNAcofold to design interacting RNA sequence pairs.</td>
<td><a rel="nofollow" class="external text" href="http://www.tbi.univie.ac.at/~choener/rnadesign/">Source Code</a><sup class="noprint Inline-Template"><span style="white-space: nowrap;">&#91;<i><a href="/wiki/Wikipedia:Link_rot" title="Wikipedia:Link rot"><span title="&#160;Dead link since March 2020">permanent dead link</span></a></i>&#93;</span></sup></td>
<td><sup id="cite_ref-RNADesign_210-0" class="reference"><a href="#cite_note-RNADesign-210">&#91;197&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=Frnakenstein&amp;action=edit&amp;redlink=1" class="new" title="Frnakenstein (page does not exist)">Frnakenstein</a>
</th>
<td>Frnakenstein applies a genetic algorithm to solve the inverse RNA folding problem.</td>
<td><a rel="nofollow" class="external text" href="http://www.stats.ox.ac.uk/research/genome/software/frnakenstein">Source Code</a></td>
<td><sup id="cite_ref-Frnakenstein_211-0" class="reference"><a href="#cite_note-Frnakenstein-211">&#91;198&#93;</a></sup>
</td></tr>
<tr>
<th><a href="/w/index.php?title=ARDesigner&amp;action=edit&amp;redlink=1" class="new" title="ARDesigner (page does not exist)">ARDesigner</a>
</th>
<td>The Allosteric RNA Designer (ARDesigner) is a web-based tool that solves the inverse folding problem by incorporating mutational robustness. Beside a local search the software has been equipped with a <a href="/wiki/Simulated_annealing" title="Simulated annealing">simulated annealing</a> approach to effectively search for good solutions. The tool has been used to design <a href="/wiki/RNA_thermometer" title="RNA thermometer">RNA thermometer</a>.</td>
<td><a rel="nofollow" class="external autonumber" href="http://biotech.bmi.ac.cn/ARDesigner">[3]</a><sup class="noprint Inline-Template"><span style="white-space: nowrap;">&#91;<i><a href="/wiki/Wikipedia:Link_rot" title="Wikipedia:Link rot"><span title="&#160;Dead link since September 2019">dead link</span></a></i>&#93;</span></sup></td>
<td><sup id="cite_ref-ARDesigner_212-0" class="reference"><a href="#cite_note-ARDesigner-212">&#91;199&#93;</a></sup>
</td></tr></tbody></table>
<dl><dt>Notes</dt>
<dd></dd></dl>
<div class="reflist" style="list-style-type: decimal;">
</div>
<h2><span id="Secondary_structure_viewers.2C_editors"></span><span class="mw-headline" id="Secondary_structure_viewers,_editors">Secondary structure viewers, editors</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=13" title="Edit section: Secondary structure viewers, editors">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="wikitable sortable">

<tbody><tr>
<th>Name
</th>
<th>Description</th>
<th>Link</th>
<th>References
</th></tr>
<tr>
<th>PseudoViewer
</th>
<td>Automatically visualizing RNA pseudoknot structures as planar graphs.</td>
<td><a rel="nofollow" class="external text" href="http://wilab.inha.ac.kr/pseudoviewer/">webapp/binary</a></td>
<td><sup id="cite_ref-pmid19369500_213-0" class="reference"><a href="#cite_note-pmid19369500-213">&#91;200&#93;</a></sup><sup id="cite_ref-pmid16845039_214-0" class="reference"><a href="#cite_note-pmid16845039-214">&#91;201&#93;</a></sup><sup id="cite_ref-pmid12824341_215-0" class="reference"><a href="#cite_note-pmid12824341-215">&#91;202&#93;</a></sup><sup id="cite_ref-pmid12169562_216-0" class="reference"><a href="#cite_note-pmid12169562-216">&#91;203&#93;</a></sup>
</td></tr>
<tr>
<th>RNA Movies
</th>
<td>browse sequential paths through RNA secondary structure landscapes</td>
<td><a rel="nofollow" class="external text" href="http://bibiserv.techfak.uni-bielefeld.de/rnamovies">sourcecode</a></td>
<td><sup id="cite_ref-pmid17567618_217-0" class="reference"><a href="#cite_note-pmid17567618-217">&#91;204&#93;</a></sup><sup id="cite_ref-pmid10068690_218-0" class="reference"><a href="#cite_note-pmid10068690-218">&#91;205&#93;</a></sup>
</td></tr>
<tr>
<th>RNA-DV
</th>
<td>RNA-DV aims at providing an easy-to-use GUI for visualizing and designing RNA secondary structures. It allows users to interact directly with the RNA structure and perform operations such as changing primary sequence content and connect/disconnect nucleotide bonds. It also integrates thermodynamic energy calculations including four major energy models. RNA-DV recognizes three input formats including CT, RNAML and dot bracket (dp).</td>
<td><a rel="nofollow" class="external text" href="http://rna-dv.sourceforge.net/">sourcecode</a></td>
<td><sup id="cite_ref-219" class="reference"><a href="#cite_note-219">&#91;206&#93;</a></sup>
</td></tr>
<tr>
<th>RNA2D3D
</th>
<td>Program to generate, view, and compare 3-dimensional models of RNA</td>
<td><a rel="nofollow" class="external text" href="http://www-lmmb.ncifcrf.gov/~bshapiro/software.html">binary</a></td>
<td><sup id="cite_ref-pmid18399701_220-0" class="reference"><a href="#cite_note-pmid18399701-220">&#91;207&#93;</a></sup>
</td></tr>
<tr>
<th>RNAstructure
</th>
<td>RNAstructure has a viewer for structures in ct files.  It can also compare predicted structures using the circleplot program.  Structures can be output as postscript files.</td>
<td><a rel="nofollow" class="external text" href="http://rna.urmc.rochester.edu/RNAstructure.html">sourcecode</a></td>
<td><sup id="cite_ref-pmid20230624_221-0" class="reference"><a href="#cite_note-pmid20230624-221">&#91;208&#93;</a></sup>
</td></tr>
<tr>
<th>RNAView/RnamlView
</th>
<td>Use RNAView to automatically identify and classify the types of base pairs that are formed in nucleic acid structures. Use RnamlView to arrange RNA structures.</td>
<td><a rel="nofollow" class="external text" href="http://ndbserver.rutgers.edu/services/">sourcecode</a></td>
<td><sup id="cite_ref-pmid12824344_222-0" class="reference"><a href="#cite_note-pmid12824344-222">&#91;209&#93;</a></sup>
</td></tr>
<tr>
<th>RILogo
</th>
<td>Visualizes the intra-/intermolecular base pairing of two interacting RNAs with sequence logos in a planar graph.</td>
<td><a rel="nofollow" class="external text" href="http://rth.dk/resources/rilogo">web server / sourcecode</a></td>
<td><sup id="cite_ref-pmid22826541_223-0" class="reference"><a href="#cite_note-pmid22826541-223">&#91;210&#93;</a></sup>
</td></tr>
<tr>
<th>VARNA
</th>
<td>A tool for the automated drawing, visualization and annotation of the secondary structure of RNA, initially designed as a companion software for web servers and databases</td>
<td><a rel="nofollow" class="external text" href="http://varna.lri.fr">webapp/sourcecode</a></td>
<td><sup id="cite_ref-pmid19398448_224-0" class="reference"><a href="#cite_note-pmid19398448-224">&#91;211&#93;</a></sup>
</td></tr>
<tr>
<th>forna
</th>
<td>A web based viewer for displaying RNA secondary structures using the force-directed graph layout provided by the <a href="/wiki/D3.js" title="D3.js">d3.js</a> visualization library. It is based on <a href="/w/index.php?title=Fornac&amp;action=edit&amp;redlink=1" class="new" title="Fornac (page does not exist)">fornac</a>, a javascript container for simply drawing a secondary structure on a web page.</td>
<td><a rel="nofollow" class="external text" href="http://rna.tbi.univie.ac.at/forna/">webapp</a><a rel="nofollow" class="external text" href="https://github.com/pkerpedjiev/fornac">fornac source</a><a rel="nofollow" class="external text" href="https://github.com/pkerpedjiev/forna">forna source</a></td>
<td><sup id="cite_ref-pmid26099263_225-0" class="reference"><a href="#cite_note-pmid26099263-225">&#91;212&#93;</a></sup>
</td></tr>
<tr>
<th>R2R
</th>
<td>Program for drawing aesthetic RNA consensus diagrams with automated pair covariance recognition. <a href="/wiki/Rfam" title="Rfam">Rfam</a> uses this program both for drawing the human-annotated SS and the R-scape covariance-optimized structure.</td>
<td><a rel="nofollow" class="external text" href="https://sourceforge.net/projects/weinberg-r2r/">source</a></td>
<td><sup id="cite_ref-226" class="reference"><a href="#cite_note-226">&#91;213&#93;</a></sup>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="See_also">See also</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=14" title="Edit section: See also">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul><li><a href="/wiki/RNA" title="RNA">RNA</a></li>
<li><a href="/wiki/Non-coding_RNA" title="Non-coding RNA">Non-coding RNA</a></li>
<li><a href="/wiki/RNA_structure" class="mw-redirect" title="RNA structure">RNA structure</a></li>
<li><a href="/wiki/Comparison_of_nucleic_acid_simulation_software" title="Comparison of nucleic acid simulation software">Comparison of nucleic acid simulation software</a></li>
<li><a href="/wiki/Comparison_of_software_for_molecular_mechanics_modeling" title="Comparison of software for molecular mechanics modeling">Comparison of software for molecular mechanics modeling</a></li></ul>
<h2><span class="mw-headline" id="References">References</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit&amp;section=15" title="Edit section: References">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="reflist columns references-column-width" style="-moz-column-width: 30em; -webkit-column-width: 30em; column-width: 30em; list-style-type: decimal;">
<ol class="references">
<li id="cite_note-pmid19095700-2"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19095700_2-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMichiaki_HamadaHisanori_KiryuKengo_SatoToutai_Mituyama2009" class="citation journal">Michiaki Hamada; Hisanori Kiryu; Kengo Sato; Toutai Mituyama; Kiyoshi Asai (2009). "Predictions of RNA secondary structure using generalized centroid estimators". <i>Bioinformatics</i>. <b>25</b> (4): 465–473. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtn601">10.1093/bioinformatics/btn601</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19095700">19095700</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Predictions+of+RNA+secondary+structure+using+generalized+centroid+estimators&amp;rft.volume=25&amp;rft.issue=4&amp;rft.pages=465-473&amp;rft.date=2009&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtn601&amp;rft_id=info%3Apmid%2F19095700&amp;rft.au=Michiaki+Hamada&amp;rft.au=Hisanori+Kiryu&amp;rft.au=Kengo+Sato&amp;rft.au=Toutai+Mituyama&amp;rft.au=Kiyoshi+Asai&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><style data-mw-deduplicate="TemplateStyles:r951705291">.mw-parser-output cite.citation{font-style:inherit}.mw-parser-output .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .id-lock-free a,.mw-parser-output .citation .cs1-lock-free a{background-image:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png");background-image:linear-gradient(transparent,transparent),url("//upload.wikimedia.org/wikipedia/commons/6/65/Lock-green.svg");background-repeat:no-repeat;background-size:9px;background-position:right .1em center}.mw-parser-output .id-lock-limited a,.mw-parser-output .id-lock-registration a,.mw-parser-output .citation .cs1-lock-limited a,.mw-parser-output .citation .cs1-lock-registration a{background-image:url("//upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/9px-Lock-gray-alt-2.svg.png");background-image:linear-gradient(transparent,transparent),url("//upload.wikimedia.org/wikipedia/commons/d/d6/Lock-gray-alt-2.svg");background-repeat:no-repeat;background-size:9px;background-position:right .1em center}.mw-parser-output .id-lock-subscription a,.mw-parser-output .citation .cs1-lock-subscription a{background-image:url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-2.svg/9px-Lock-red-alt-2.svg.png");background-image:linear-gradient(transparent,transparent),url("//upload.wikimedia.org/wikipedia/commons/a/aa/Lock-red-alt-2.svg");background-repeat:no-repeat;background-size:9px;background-position:right .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-parser-output .cs1-registration span{border-bottom:1px dotted;cursor:help}.mw-parser-output .cs1-ws-icon a{background-image:url("//upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/12px-Wikisource-logo.svg.png");background-image:linear-gradient(transparent,transparent),url("//upload.wikimedia.org/wikipedia/commons/4/4c/Wikisource-logo.svg");background-repeat:no-repeat;background-size:12px;background-position:right .1em center}.mw-parser-output code.cs1-code{color:inherit;background:inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error{font-size:100%}.mw-parser-output .cs1-maint{display:none;color:#33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-wl-right{padding-right:0.2em}.mw-parser-output .citation .mw-selflink{font-weight:inherit}</style></span>
</li>
<li id="cite_note-pmid19478007-3"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19478007_3-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMichiaki_HamadaHisanori_KiryuKengo_SatoToutai_Mituyama2009" class="citation journal">Michiaki Hamada; Hisanori Kiryu; Kengo Sato; Toutai Mituyama; Kiyoshi Asai (2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2687982">"Predictions of RNA secondary structure by combining homologous sequence information"</a>. <i>Bioinformatics</i>. <b>25</b> (12): i330–8. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp228">10.1093/bioinformatics/btp228</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2687982">2687982</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19478007">19478007</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Predictions+of+RNA+secondary+structure+by+combining+homologous+sequence+information&amp;rft.volume=25&amp;rft.issue=12&amp;rft.pages=i330-8&amp;rft.date=2009&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687982&amp;rft_id=info%3Apmid%2F19478007&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtp228&amp;rft.au=Michiaki+Hamada&amp;rft.au=Hisanori+Kiryu&amp;rft.au=Kengo+Sato&amp;rft.au=Toutai+Mituyama&amp;rft.au=Kiyoshi+Asai&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687982&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid22035327-4"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid22035327_4-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFShay_ZakovYoav_GoldbergMichael_ElhadadMichal_Ziv-Ukelson2011" class="citation journal">Shay Zakov; Yoav Goldberg; Michael Elhadad; Michal Ziv-Ukelson (2011). "Rich parameterization improves RNA structure prediction". <i>Journal of Computational Biology</i>. <b>18</b> (11): 1525–1542. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2011LNCS.6577..546Z">2011LNCS.6577..546Z</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1089%2Fcmb.2011.0184">10.1089/cmb.2011.0184</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22035327">22035327</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Computational+Biology&amp;rft.atitle=Rich+parameterization+improves+RNA+structure+prediction&amp;rft.volume=18&amp;rft.issue=11&amp;rft.pages=1525-1542&amp;rft.date=2011&amp;rft_id=info%3Apmid%2F22035327&amp;rft_id=info%3Adoi%2F10.1089%2Fcmb.2011.0184&amp;rft_id=info%3Abibcode%2F2011LNCS.6577..546Z&amp;rft.au=Shay+Zakov&amp;rft.au=Yoav+Goldberg&amp;rft.au=Michael+Elhadad&amp;rft.au=Michal+Ziv-Ukelson&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16873527-5"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16873527_5-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDoWoodsBatzoglou2006" class="citation journal">Do CB, Woods DA, Batzoglou S (2006). "CONTRAfold: RNA secondary structure prediction without physics-based models". <i>Bioinformatics</i>. <b>22</b> (14): e90–8. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl246">10.1093/bioinformatics/btl246</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16873527">16873527</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=CONTRAfold%3A+RNA+secondary+structure+prediction+without+physics-based+models&amp;rft.volume=22&amp;rft.issue=14&amp;rft.pages=e90-8&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtl246&amp;rft_id=info%3Apmid%2F16873527&amp;rft.aulast=Do&amp;rft.aufirst=CB&amp;rft.au=Woods%2C+DA&amp;rft.au=Batzoglou%2C+S&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid21723827-6"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid21723827_6-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid21723827_6-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFSchroederBleckleyStone2011" class="citation journal">Schroeder S, Bleckley S, Stone JW (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3127170">"Ensemble of secondary structures for encapsidated satellite tobacco mosaic virus RNA consistent with chemical probing and crystallography constraints"</a>. <i>Biophysical Journal</i>. <b>101</b> (1): 167–175. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2011BpJ...101..167S">2011BpJ...101..167S</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.bpj.2011.05.053">10.1016/j.bpj.2011.05.053</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3127170">3127170</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21723827">21723827</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biophysical+Journal&amp;rft.atitle=Ensemble+of+secondary+structures+for+encapsidated+satellite+tobacco+mosaic+virus+RNA+consistent+with+chemical+probing+and+crystallography+constraints.&amp;rft.volume=101&amp;rft.issue=1&amp;rft.pages=167-175&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3127170&amp;rft_id=info%3Apmid%2F21723827&amp;rft_id=info%3Adoi%2F10.1016%2Fj.bpj.2011.05.053&amp;rft_id=info%3Abibcode%2F2011BpJ...101..167S&amp;rft.aulast=Schroeder&amp;rft.aufirst=S&amp;rft.au=Bleckley%2C+S&amp;rft.au=Stone%2C+JW&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3127170&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid20501603-7"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid20501603_7-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFBindewaldKluthShapiro2010" class="citation journal">Bindewald E, Kluth T, Shapiro BA (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2896150">"CyloFold: secondary structure prediction including pseudoknots"</a>. <i>Nucleic Acids Research</i>. <b>38</b> (Web Server issue): 368–72. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkq432">10.1093/nar/gkq432</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2896150">2896150</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20501603">20501603</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=CyloFold%3A+secondary+structure+prediction+including+pseudoknots&amp;rft.volume=38&amp;rft.issue=Web+Server+issue&amp;rft.pages=368-72&amp;rft.date=2010&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2896150&amp;rft_id=info%3Apmid%2F20501603&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkq432&amp;rft.aulast=Bindewald&amp;rft.aufirst=E&amp;rft.au=Kluth%2C+T&amp;rft.au=Shapiro%2C+BA&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2896150&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-8"><span class="mw-cite-backlink"><b><a href="#cite_ref-8">^</a></b></span> <span class="reference-text"><cite id="CITEREFChenLiUmarovGao2020" class="citation arxiv">Chen, Xinshi; Li, Yu; Umarov, Ramzan; Gao, Xin; Song, Le (2020-02-13). "RNA Secondary Structure Prediction By Learning Unrolled Algorithms". <a href="/wiki/ArXiv_(identifier)" class="mw-redirect" title="ArXiv (identifier)">arXiv</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//arxiv.org/abs/2002.05810">2002.05810</a></span> [<a rel="nofollow" class="external text" href="//arxiv.org/archive/cs.LG">cs.LG</a>].</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=preprint&amp;rft.jtitle=arXiv&amp;rft.atitle=RNA+Secondary+Structure+Prediction+By+Learning+Unrolled+Algorithms&amp;rft.date=2020-02-13&amp;rft_id=info%3Aarxiv%2F2002.05810&amp;rft.aulast=Chen&amp;rft.aufirst=Xinshi&amp;rft.au=Li%2C+Yu&amp;rft.au=Umarov%2C+Ramzan&amp;rft.au=Gao%2C+Xin&amp;rft.au=Song%2C+Le&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-9"><span class="mw-cite-backlink"><b><a href="#cite_ref-9">^</a></b></span> <span class="reference-text">Chen, X., Li, Y., Umarov, R., Gao, X., and Song, L. RNAsecondary structure prediction by learning unrolled algorithms. In International Conference on Learning Representations, 2020. URL <a rel="nofollow" class="external free" href="https://openreview.net/forum?id=S1eALyrYDH">https://openreview.net/forum?id=S1eALyrYDH</a>.</span>
</li>
<li id="cite_note-pmid22747589-10"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid22747589_10-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSwensonAndersonAshGaurav2012" class="citation journal">Swenson MS, Anderson J, Ash A, Gaurav P, Sükösd Z, Bader DA, Harvey SC, <a href="/wiki/Christine_Heitsch" title="Christine Heitsch">Heitsch CE</a> (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3748833">"GTfold: enabling parallel RNA secondary structure prediction on multi-core desktops"</a>. <i>BMC Res Notes</i>. <b>5</b>: 341. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1756-0500-5-341">10.1186/1756-0500-5-341</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3748833">3748833</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22747589">22747589</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Res+Notes&amp;rft.atitle=GTfold%3A+enabling+parallel+RNA+secondary+structure+prediction+on+multi-core+desktops.&amp;rft.volume=5&amp;rft.pages=341&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3748833&amp;rft_id=info%3Apmid%2F22747589&amp;rft_id=info%3Adoi%2F10.1186%2F1756-0500-5-341&amp;rft.aulast=Swenson&amp;rft.aufirst=MS&amp;rft.au=Anderson%2C+J&amp;rft.au=Ash%2C+A&amp;rft.au=Gaurav%2C+P&amp;rft.au=S%C3%BCk%C3%B6sd%2C+Z&amp;rft.au=Bader%2C+DA&amp;rft.au=Harvey%2C+SC&amp;rft.au=Heitsch%2C+CE&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3748833&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid21685106-11"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid21685106_11-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSatoKatoHamadaAkutsu2011" class="citation journal">Sato K, Kato Y, Hamada M, Akutsu T, Asai K (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3117384">"IPknot: fast and accurate prediction of RNA secondary structures with pseudoknots using integer programming"</a>. <i>Bioinformatics</i>. <b>27</b> (13): i85-93. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtr215">10.1093/bioinformatics/btr215</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3117384">3117384</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21685106">21685106</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=IPknot%3A+fast+and+accurate+prediction+of+RNA+secondary+structures+with+pseudoknots+using+integer+programming&amp;rft.volume=27&amp;rft.issue=13&amp;rft.pages=i85-93&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3117384&amp;rft_id=info%3Apmid%2F21685106&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtr215&amp;rft.aulast=Sato&amp;rft.aufirst=K&amp;rft.au=Kato%2C+Y&amp;rft.au=Hamada%2C+M&amp;rft.au=Akutsu%2C+T&amp;rft.au=Asai%2C+K&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3117384&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15980546-12"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15980546_12-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFXayaphoummineBucherIsambert2005" class="citation journal">Xayaphoummine A, Bucher T, Isambert H (2005). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1160208">"Kinefold web server for RNA/DNA folding path and structure prediction including pseudoknots and knots"</a>. <i>Nucleic Acids Res</i>. <b>33</b> (Web Server issue): W605–10. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgki447">10.1093/nar/gki447</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1160208">1160208</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15980546">15980546</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=Kinefold+web+server+for+RNA%2FDNA+folding+path+and+structure+prediction+including+pseudoknots+and+knots&amp;rft.volume=33&amp;rft.issue=Web+Server+issue&amp;rft.pages=W605-10&amp;rft.date=2005&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1160208&amp;rft_id=info%3Apmid%2F15980546&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgki447&amp;rft.aulast=Xayaphoummine&amp;rft.aufirst=A&amp;rft.au=Bucher%2C+T&amp;rft.au=Isambert%2C+H&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1160208&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid14676318-13"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid14676318_13-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFXayaphoummineBucherThalmannIsambert2003" class="citation journal">Xayaphoummine A, Bucher T, Thalmann F, Isambert H (2003). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC307563">"Prediction and statistics of pseudoknots in RNA structures using exactly clustered stochastic simulations"</a>. <i>Proc. Natl. Acad. Sci. U.S.A</i>. <b>100</b> (26): 15310–5. <a href="/wiki/ArXiv_(identifier)" class="mw-redirect" title="ArXiv (identifier)">arXiv</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//arxiv.org/abs/physics/0309117">physics/0309117</a></span>. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2003PNAS..10015310X">2003PNAS..10015310X</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.2536430100">10.1073/pnas.2536430100</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC307563">307563</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/14676318">14676318</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;rft.atitle=Prediction+and+statistics+of+pseudoknots+in+RNA+structures+using+exactly+clustered+stochastic+simulations&amp;rft.volume=100&amp;rft.issue=26&amp;rft.pages=15310-5&amp;rft.date=2003&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC307563&amp;rft_id=info%3Abibcode%2F2003PNAS..10015310X&amp;rft_id=info%3Aarxiv%2Fphysics%2F0309117&amp;rft_id=info%3Apmid%2F14676318&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.2536430100&amp;rft.aulast=Xayaphoummine&amp;rft.aufirst=A&amp;rft.au=Bucher%2C+T&amp;rft.au=Thalmann%2C+F&amp;rft.au=Isambert%2C+H&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC307563&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid6163133-14"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid6163133_14-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid6163133_14-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFZukerStiegler1981" class="citation journal">Zuker M, Stiegler P (1981). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC326673">"Optimal computer folding of large RNA sequences using thermodynamics and auxiliary information"</a>. <i>Nucleic Acids Res</i>. <b>9</b> (1): 133–48. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F9.1.133">10.1093/nar/9.1.133</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC326673">326673</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/6163133">6163133</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=Optimal+computer+folding+of+large+RNA+sequences+using+thermodynamics+and+auxiliary+information&amp;rft.volume=9&amp;rft.issue=1&amp;rft.pages=133-48&amp;rft.date=1981&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC326673&amp;rft_id=info%3Apmid%2F6163133&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F9.1.133&amp;rft.aulast=Zuker&amp;rft.aufirst=M&amp;rft.au=Stiegler%2C+P&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC326673&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid_pkiss-15"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid_pkiss_15-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid_pkiss_15-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFTheis,_Corinna_and_Janssen,_Stefan_and_Giegerich,_Robert2010" class="citation conference">Theis, Corinna and Janssen, Stefan and Giegerich, Robert (2010). "Prediction of RNA Secondary Structure Including Kissing Hairpin Motifs".  In Moulton, Vincent and Singh, Mona (ed.). <i>Algorithms in Bioinformatics</i>. <b>6293</b> (Lecture Notes in Computer Science ed.). Springer Berlin Heidelberg. pp.&#160;52–64. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2F978-3-642-15294-8_5">10.1007/978-3-642-15294-8_5</a>. <a href="/wiki/ISBN_(identifier)" class="mw-redirect" title="ISBN (identifier)">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-3-642-15293-1" title="Special:BookSources/978-3-642-15293-1"><bdi>978-3-642-15293-1</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=conference&amp;rft.atitle=Prediction+of+RNA+Secondary+Structure+Including+Kissing+Hairpin+Motifs&amp;rft.btitle=Algorithms+in+Bioinformatics&amp;rft.pages=52-64&amp;rft.edition=Lecture+Notes+in+Computer+Science&amp;rft.pub=Springer+Berlin+Heidelberg&amp;rft.date=2010&amp;rft_id=info%3Adoi%2F10.1007%2F978-3-642-15294-8_5&amp;rft.isbn=978-3-642-15293-1&amp;rft.au=Theis%2C+Corinna+and+Janssen%2C+Stefan+and+Giegerich%2C+Robert&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><span class="cs1-maint citation-comment">CS1 maint: multiple names: authors list (<a href="/wiki/Category:CS1_maint:_multiple_names:_authors_list" title="Category:CS1 maint: multiple names: authors list">link</a>)</span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid9925784-16"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid9925784_16-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFRivasEddy1999" class="citation journal">Rivas E, Eddy SR (1999). "A dynamic programming algorithm for RNA structure prediction including pseudoknots". <i>J. Mol. Biol</i>. <b>285</b> (5): 2053–68. <a href="/wiki/ArXiv_(identifier)" class="mw-redirect" title="ArXiv (identifier)">arXiv</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//arxiv.org/abs/physics/9807048">physics/9807048</a></span>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1006%2Fjmbi.1998.2436">10.1006/jmbi.1998.2436</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/9925784">9925784</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J.+Mol.+Biol.&amp;rft.atitle=A+dynamic+programming+algorithm+for+RNA+structure+prediction+including+pseudoknots&amp;rft.volume=285&amp;rft.issue=5&amp;rft.pages=2053-68&amp;rft.date=1999&amp;rft_id=info%3Aarxiv%2Fphysics%2F9807048&amp;rft_id=info%3Apmid%2F9925784&amp;rft_id=info%3Adoi%2F10.1006%2Fjmbi.1998.2436&amp;rft.aulast=Rivas&amp;rft.aufirst=E&amp;rft.au=Eddy%2C+SR&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17478505-17"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17478505_17-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFReederSteffenGiegerich2007" class="citation journal">Reeder J, Steffen P, Giegerich R (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1933184">"pknotsRG: RNA pseudoknot folding including near-optimal structures and sliding windows"</a>. <i>Nucleic Acids Res</i>. <b>35</b> (Web Server issue): W320–4. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkm258">10.1093/nar/gkm258</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1933184">1933184</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17478505">17478505</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=pknotsRG%3A+RNA+pseudoknot+folding+including+near-optimal+structures+and+sliding+windows&amp;rft.volume=35&amp;rft.issue=Web+Server+issue&amp;rft.pages=W320-4&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1933184&amp;rft_id=info%3Apmid%2F17478505&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkm258&amp;rft.aulast=Reeder&amp;rft.aufirst=J&amp;rft.au=Steffen%2C+P&amp;rft.au=Giegerich%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1933184&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNAinverse-18"><span class="mw-cite-backlink">^ <a href="#cite_ref-RNAinverse_18-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-RNAinverse_18-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-RNAinverse_18-2"><sup><i><b>c</b></i></sup></a> <a href="#cite_ref-RNAinverse_18-3"><sup><i><b>d</b></i></sup></a> <a href="#cite_ref-RNAinverse_18-4"><sup><i><b>e</b></i></sup></a> <a href="#cite_ref-RNAinverse_18-5"><sup><i><b>f</b></i></sup></a> <a href="#cite_ref-RNAinverse_18-6"><sup><i><b>g</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFI.L._HofackerW._FontanaP.F._StadlerS._Bonhoeffer1994" class="citation journal">I.L. Hofacker; W. Fontana; P.F. Stadler; S. Bonhoeffer; M. Tacker; P. Schuster (1994). "Fast Folding and Comparison of RNA Secondary Structures". <i>Monatshefte für Chemie</i>. <b>125</b> (2): 167–188. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2FBF00818163">10.1007/BF00818163</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Monatshefte+f%C3%BCr+Chemie&amp;rft.atitle=Fast+Folding+and+Comparison+of+RNA+Secondary+Structures.&amp;rft.volume=125&amp;rft.issue=2&amp;rft.pages=167-188&amp;rft.date=1994&amp;rft_id=info%3Adoi%2F10.1007%2FBF00818163&amp;rft.au=I.L.+Hofacker&amp;rft.au=W.+Fontana&amp;rft.au=P.F.+Stadler&amp;rft.au=S.+Bonhoeffer&amp;rft.au=M.+Tacker&amp;rft.au=P.+Schuster&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid1695107-19"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid1695107_19-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMcCaskill_JS1990" class="citation journal">McCaskill JS (1990). "The equilibrium partition function and base pair binding probabilities for RNA secondary structure". <i>Biopolymers</i>. <b>29</b> (6–7): 1105–19. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2Fbip.360290621">10.1002/bip.360290621</a>. <a href="/wiki/Hdl_(identifier)" class="mw-redirect" title="Hdl (identifier)">hdl</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//hdl.handle.net/11858%2F00-001M-0000-0013-0DE3-9">11858/00-001M-0000-0013-0DE3-9</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/1695107">1695107</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biopolymers&amp;rft.atitle=The+equilibrium+partition+function+and+base+pair+binding+probabilities+for+RNA+secondary+structure&amp;rft.volume=29&amp;rft.issue=6%E2%80%937&amp;rft.pages=1105-19&amp;rft.date=1990&amp;rft_id=info%3Ahdl%2F11858%2F00-001M-0000-0013-0DE3-9&amp;rft_id=info%3Apmid%2F1695107&amp;rft_id=info%3Adoi%2F10.1002%2Fbip.360290621&amp;rft.au=McCaskill+JS&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16452114-20"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16452114_20-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHofackerStadler2006" class="citation journal">Hofacker IL, Stadler PF (2006). <a href="/wiki/Circular_RNA" title="Circular RNA">"Memory efficient folding algorithms for circular RNA secondary structures"</a>. <i>Bioinformatics</i>. <b>22</b> (10): 1172–6. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl023">10.1093/bioinformatics/btl023</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16452114">16452114</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Memory+efficient+folding+algorithms+for+circular+RNA+secondary+structures&amp;rft.volume=22&amp;rft.issue=10&amp;rft.pages=1172-6&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtl023&amp;rft_id=info%3Apmid%2F16452114&amp;rft.aulast=Hofacker&amp;rft.aufirst=IL&amp;rft.au=Stadler%2C+PF&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17611759-21"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17611759_21-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFBompfünewererBackofenBernhart2008" class="citation journal">Bompfünewerer AF, Backofen R, Bernhart SH,  et al. (2008). "Variations on RNA folding and alignment: lessons from Benasque". <i>J Math Biol</i>. <b>56</b> (1–2): 129–144. <a href="/wiki/CiteSeerX_(identifier)" class="mw-redirect" title="CiteSeerX (identifier)">CiteSeerX</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.188.1420">10.1.1.188.1420</a></span>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2Fs00285-007-0107-5">10.1007/s00285-007-0107-5</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17611759">17611759</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J+Math+Biol&amp;rft.atitle=Variations+on+RNA+folding+and+alignment%3A+lessons+from+Benasque&amp;rft.volume=56&amp;rft.issue=1%E2%80%932&amp;rft.pages=129-144&amp;rft.date=2008&amp;rft_id=%2F%2Fciteseerx.ist.psu.edu%2Fviewdoc%2Fsummary%3Fdoi%3D10.1.1.188.1420&amp;rft_id=info%3Apmid%2F17611759&amp;rft_id=info%3Adoi%2F10.1007%2Fs00285-007-0107-5&amp;rft.aulast=Bompf%C3%BCnewerer&amp;rft.aufirst=AF&amp;rft.au=Backofen%2C+R&amp;rft.au=Bernhart%2C+SH&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-abstract_shapes-22"><span class="mw-cite-backlink"><b><a href="#cite_ref-abstract_shapes_22-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFR._Giegerich,_B.Voß,_M._Rehmsmeier2004" class="citation journal">R. Giegerich, B.Voß, M. Rehmsmeier (2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC519098">"Abstract shapes of RNA"</a>. <i>Nucleic Acids Res</i>. <b>32</b> (16): 4843–4851. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkh779">10.1093/nar/gkh779</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC519098">519098</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15371549">15371549</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=Abstract+shapes+of+RNA.&amp;rft.volume=32&amp;rft.issue=16&amp;rft.pages=4843-4851&amp;rft.date=2004&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC519098&amp;rft_id=info%3Apmid%2F15371549&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkh779&amp;rft.au=R.+Giegerich%2C+B.Vo%C3%9F%2C+M.+Rehmsmeier&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC519098&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><span class="cs1-maint citation-comment">CS1 maint: multiple names: authors list (<a href="/wiki/Category:CS1_maint:_multiple_names:_authors_list" title="Category:CS1 maint: multiple names: authors list">link</a>)</span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-shape_probabilities-23"><span class="mw-cite-backlink"><b><a href="#cite_ref-shape_probabilities_23-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFB._VoßR._GiegerichM._Rehmsmeier2006" class="citation journal">B. Voß; R. Giegerich; M. Rehmsmeier (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1479382">"Complete probabilistic analysis of RNA shapes"</a>. <i>BMC Biology</i>. <b>4</b> (1): 5. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1741-7007-4-5">10.1186/1741-7007-4-5</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1479382">1479382</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16480488">16480488</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Biology&amp;rft.atitle=Complete+probabilistic+analysis+of+RNA+shapes.&amp;rft.volume=4&amp;rft.issue=1&amp;rft.pages=5&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1479382&amp;rft_id=info%3Apmid%2F16480488&amp;rft_id=info%3Adoi%2F10.1186%2F1741-7007-4-5&amp;rft.au=B.+Vo%C3%9F&amp;rft.au=R.+Giegerich&amp;rft.au=M.+Rehmsmeier&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1479382&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNAstructure-24"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNAstructure_24-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFD.H._MathewsM.D._DisneyJ._L._ChildsS.J._Schroeder2004" class="citation journal">D.H. Mathews; M.D. Disney; J. L. Childs; S.J. Schroeder; M. Zuker; D.H. Turner (2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC409911">"Incorporating chemical modification constraints into a dynamic programming algorothm for prediction of RNA secondary structure"</a>. <i>Proceedings of the National Academy of Sciences of the United States of America</i>. <b>101</b> (19): 7287–7292. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2004PNAS..101.7287M">2004PNAS..101.7287M</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.0401799101">10.1073/pnas.0401799101</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC409911">409911</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15123812">15123812</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;rft.atitle=Incorporating+chemical+modification+constraints+into+a+dynamic+programming+algorothm+for+prediction+of+RNA+secondary+structure.&amp;rft.volume=101&amp;rft.issue=19&amp;rft.pages=7287-7292&amp;rft.date=2004&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC409911&amp;rft_id=info%3Apmid%2F15123812&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0401799101&amp;rft_id=info%3Abibcode%2F2004PNAS..101.7287M&amp;rft.au=D.H.+Mathews&amp;rft.au=M.D.+Disney&amp;rft.au=J.+L.+Childs&amp;rft.au=S.J.+Schroeder&amp;rft.au=M.+Zuker&amp;rft.au=D.H.+Turner&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC409911&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNAstructure-partition-25"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNAstructure-partition_25-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFD.H._Mathews2004" class="citation journal">D.H. Mathews (2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1370608">"Using an RNA secondary structure partition function to determine confidence in base pairs predicted by free energy minimization"</a>. <i>RNA</i>. <b>10</b> (8): 1178–1190. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1261%2Frna.7650904">10.1261/rna.7650904</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1370608">1370608</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15272118">15272118</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=RNA&amp;rft.atitle=Using+an+RNA+secondary+structure+partition+function+to+determine+confidence+in+base+pairs+predicted+by+free+energy+minimization.&amp;rft.volume=10&amp;rft.issue=8&amp;rft.pages=1178-1190&amp;rft.date=2004&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1370608&amp;rft_id=info%3Apmid%2F15272118&amp;rft_id=info%3Adoi%2F10.1261%2Frna.7650904&amp;rft.au=D.H.+Mathews&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1370608&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-26"><span class="mw-cite-backlink"><b><a href="#cite_ref-26">^</a></b></span> <span class="reference-text"><cite id="CITEREFTsangWiese2010" class="citation journal">Tsang HH, Wiese KC (2010). "SARNA-Predict: accuracy improvement of RNA secondary structure prediction using permutation-based simulated annealing". <i>IEEE/ACM Transactions on Computational Biology and Bioinformatics</i>. <b>7</b> (4): 727–40. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1109%2FTCBB.2008.97">10.1109/TCBB.2008.97</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21030739">21030739</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=IEEE%2FACM+Transactions+on+Computational+Biology+and+Bioinformatics&amp;rft.atitle=SARNA-Predict%3A+accuracy+improvement+of+RNA+secondary+structure+prediction+using+permutation-based+simulated+annealing&amp;rft.volume=7&amp;rft.issue=4&amp;rft.pages=727-40&amp;rft.date=2010&amp;rft_id=info%3Adoi%2F10.1109%2FTCBB.2008.97&amp;rft_id=info%3Apmid%2F21030739&amp;rft.aulast=Tsang&amp;rft.aufirst=HH&amp;rft.au=Wiese%2C+KC&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid14654704-27"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid14654704_27-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDingLawrence2003" class="citation journal">Ding Y, Lawrence CE (2003). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC297010">"A statistical sampling algorithm for RNA secondary structure prediction"</a>. <i>Nucleic Acids Res</i>. <b>31</b> (24): 7280–301. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkg938">10.1093/nar/gkg938</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC297010">297010</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/14654704">14654704</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=A+statistical+sampling+algorithm+for+RNA+secondary+structure+prediction&amp;rft.volume=31&amp;rft.issue=24&amp;rft.pages=7280-301&amp;rft.date=2003&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC297010&amp;rft_id=info%3Apmid%2F14654704&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkg938&amp;rft.aulast=Ding&amp;rft.aufirst=Y&amp;rft.au=Lawrence%2C+CE&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC297010&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15215366-28"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15215366_28-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDingChanLawrence2004" class="citation journal">Ding Y, Chan CY, Lawrence CE (2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC441587">"Sfold web server for statistical folding and rational design of nucleic acids"</a>. <i>Nucleic Acids Res</i>. <b>32</b> (Web Server issue): W135–41. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkh449">10.1093/nar/gkh449</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC441587">441587</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15215366">15215366</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=Sfold+web+server+for+statistical+folding+and+rational+design+of+nucleic+acids&amp;rft.volume=32&amp;rft.issue=Web+Server+issue&amp;rft.pages=W135-41&amp;rft.date=2004&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC441587&amp;rft_id=info%3Apmid%2F15215366&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkh449&amp;rft.aulast=Ding&amp;rft.aufirst=Y&amp;rft.au=Chan%2C+CY&amp;rft.au=Lawrence%2C+CE&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC441587&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16043502-29"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16043502_29-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDingChanLawrence2005" class="citation journal">Ding Y, Chan CY, Lawrence CE (2005). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1370799">"RNA secondary structure prediction by centroids in a Boltzmann weighted ensemble"</a>. <i>RNA</i>. <b>11</b> (8): 1157–66. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1261%2Frna.2500605">10.1261/rna.2500605</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1370799">1370799</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16043502">16043502</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=RNA&amp;rft.atitle=RNA+secondary+structure+prediction+by+centroids+in+a+Boltzmann+weighted+ensemble&amp;rft.volume=11&amp;rft.issue=8&amp;rft.pages=1157-66&amp;rft.date=2005&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1370799&amp;rft_id=info%3Apmid%2F16043502&amp;rft_id=info%3Adoi%2F10.1261%2Frna.2500605&amp;rft.aulast=Ding&amp;rft.aufirst=Y&amp;rft.au=Chan%2C+CY&amp;rft.au=Lawrence%2C+CE&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1370799&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16109749-30"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16109749_30-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFChanLawrenceDing2005" class="citation journal">Chan CY, Lawrence CE, Ding Y (2005). "Structure clustering features on the Sfold Web server". <i>Bioinformatics</i>. <b>21</b> (20): 3926–8. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbti632">10.1093/bioinformatics/bti632</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16109749">16109749</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Structure+clustering+features+on+the+Sfold+Web+server&amp;rft.volume=21&amp;rft.issue=20&amp;rft.pages=3926-8&amp;rft.date=2005&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbti632&amp;rft_id=info%3Apmid%2F16109749&amp;rft.aulast=Chan&amp;rft.aufirst=CY&amp;rft.au=Lawrence%2C+CE&amp;rft.au=Ding%2C+Y&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-31"><span class="mw-cite-backlink"><b><a href="#cite_ref-31">^</a></b></span> <span class="reference-text"><cite id="CITEREFSinghHansonPaliwalZhou2019" class="citation journal">Singh, Jaswinder; Hanson, Jack; Paliwal, Kuldip; Zhou, Yaoqi (2019-11-27). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC6881452">"RNA secondary structure prediction using an ensemble of two-dimensional deep neural networks and transfer learning"</a>. <i>Nature Communications</i>. <b>10</b> (1): 5407. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2019NatCo..10.5407S">2019NatCo..10.5407S</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fs41467-019-13395-9">10.1038/s41467-019-13395-9</a>. <a href="/wiki/ISSN_(identifier)" class="mw-redirect" title="ISSN (identifier)">ISSN</a>&#160;<a rel="nofollow" class="external text" href="//www.worldcat.org/issn/2041-1723">2041-1723</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC6881452">6881452</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/31776342">31776342</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Communications&amp;rft.atitle=RNA+secondary+structure+prediction+using+an+ensemble+of+two-dimensional+deep+neural+networks+and+transfer+learning&amp;rft.volume=10&amp;rft.issue=1&amp;rft.pages=5407&amp;rft.date=2019-11-27&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6881452&amp;rft_id=info%3Abibcode%2F2019NatCo..10.5407S&amp;rft_id=info%3Apmid%2F31776342&amp;rft_id=info%3Adoi%2F10.1038%2Fs41467-019-13395-9&amp;rft.issn=2041-1723&amp;rft.aulast=Singh&amp;rft.aufirst=Jaswinder&amp;rft.au=Hanson%2C+Jack&amp;rft.au=Paliwal%2C+Kuldip&amp;rft.au=Zhou%2C+Yaoqi&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6881452&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid27378291-32"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid27378291_32-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFBarsacchiNovoaKellisBechini2016" class="citation journal">Barsacchi B, Novoa EM, Kellis M, Bechini A (2016). "SwiSpot: modeling riboswitches by spotting out switching sequences". <i>Bioinformatics</i>. <b>32</b> (21): 3252–3259. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtw401">10.1093/bioinformatics/btw401</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/27378291">27378291</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=SwiSpot%3A+modeling+riboswitches+by+spotting+out+switching+sequences.&amp;rft.volume=32&amp;rft.issue=21&amp;rft.pages=3252-3259&amp;rft.date=2016&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtw401&amp;rft_id=info%3Apmid%2F27378291&amp;rft.aulast=Barsacchi&amp;rft.aufirst=B&amp;rft.au=Novoa%2C+EM&amp;rft.au=Kellis%2C+M&amp;rft.au=Bechini%2C+A&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18712296-33"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18712296_33-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMarkhamZuker2008" class="citation book">Markham NR, Zuker M (2008). <i>UNAFold: software for nucleic acid folding and hybridization</i>. <i>Methods Mol Biol</i>. Methods in Molecular Biology™. <b>453</b>. pp.&#160;3–31. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2F978-1-60327-429-6_1">10.1007/978-1-60327-429-6_1</a>. <a href="/wiki/ISBN_(identifier)" class="mw-redirect" title="ISBN (identifier)">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-1-60327-428-9" title="Special:BookSources/978-1-60327-428-9"><bdi>978-1-60327-428-9</bdi></a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18712296">18712296</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=UNAFold%3A+software+for+nucleic+acid+folding+and+hybridization.&amp;rft.series=Methods+in+Molecular+Biology%E2%84%A2&amp;rft.pages=3-31&amp;rft.date=2008&amp;rft_id=info%3Apmid%2F18712296&amp;rft_id=info%3Adoi%2F10.1007%2F978-1-60327-429-6_1&amp;rft.isbn=978-1-60327-428-9&amp;rft.aulast=Markham&amp;rft.aufirst=NR&amp;rft.au=Zuker%2C+M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17878940-34"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17878940_34-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDawsonFujiwaraKawai2007" class="citation journal">Dawson WK, Fujiwara K, Kawai G (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1975678">"Prediction of RNA pseudoknots using heuristic modeling with mapping and sequential folding"</a>. <i>PLOS One</i>. <b>2</b> (9): e905. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2007PLoSO...2..905D">2007PLoSO...2..905D</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pone.0000905">10.1371/journal.pone.0000905</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1975678">1975678</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17878940">17878940</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+One&amp;rft.atitle=Prediction+of+RNA+pseudoknots+using+heuristic+modeling+with+mapping+and+sequential+folding&amp;rft.volume=2&amp;rft.issue=9&amp;rft.pages=e905&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1975678&amp;rft_id=info%3Apmid%2F17878940&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pone.0000905&amp;rft_id=info%3Abibcode%2F2007PLoSO...2..905D&amp;rft.aulast=Dawson&amp;rft.aufirst=WK&amp;rft.au=Fujiwara%2C+K&amp;rft.au=Kawai%2C+G&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1975678&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmidnone-35"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmidnone_35-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDawsonTakaiItoShimizu2014" class="citation journal">Dawson WK, Takai T, Ito N, Shimizu K, Kawai G (2014). "A new entropy model for RNA: part III. Is the folding free energy landscape of RNA funnel shaped?". <i>Journal of Nucleic Acids Investigation</i>. <b>5</b> (1): 2652. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.4081%2Fjnai.2014.2652">10.4081/jnai.2014.2652</a></span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Nucleic+Acids+Investigation&amp;rft.atitle=A+new+entropy+model+for+RNA%3A+part+III.+Is+the+folding+free+energy+landscape+of+RNA+funnel+shaped%3F&amp;rft.volume=5&amp;rft.issue=1&amp;rft.pages=2652&amp;rft.date=2014&amp;rft_id=info%3Adoi%2F10.4081%2Fjnai.2014.2652&amp;rft.aulast=Dawson&amp;rft.aufirst=WK&amp;rft.au=Takai%2C+T&amp;rft.au=Ito%2C+N&amp;rft.au=Shimizu%2C+K&amp;rft.au=Kawai%2C+G&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid19543381-37"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19543381_37-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFFrellsenMoltkeThiimMardia2009" class="citation journal">Frellsen J, Moltke I, Thiim M, Mardia KV, Ferkinghoff-Borg J, Hamelryck T (2009).  Gardner P (ed.). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2691987">"A probabilistic model of RNA conformational space"</a>. <i>PLOS Comput. Biol</i>. <b>5</b> (6): e1000406. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2009PLSCB...5E0406F">2009PLSCB...5E0406F</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pcbi.1000406">10.1371/journal.pcbi.1000406</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2691987">2691987</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19543381">19543381</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+Comput.+Biol.&amp;rft.atitle=A+probabilistic+model+of+RNA+conformational+space.&amp;rft.volume=5&amp;rft.issue=6&amp;rft.pages=e1000406&amp;rft.date=2009&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2691987&amp;rft_id=info%3Apmid%2F19543381&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.1000406&amp;rft_id=info%3Abibcode%2F2009PLSCB...5E0406F&amp;rft.aulast=Frellsen&amp;rft.aufirst=J&amp;rft.au=Moltke%2C+I&amp;rft.au=Thiim%2C+M&amp;rft.au=Mardia%2C+KV&amp;rft.au=Ferkinghoff-Borg%2C+J&amp;rft.au=Hamelryck%2C+T&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2691987&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17726102-38"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17726102_38-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDasBaker2007" class="citation journal">Das R, Baker D (September 2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1955458">"Automated de novo prediction of native-like RNA tertiary structures"</a>. <i>Proc. Natl. Acad. Sci. U.S.A</i>. <b>104</b> (37): 14664–9. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2007PNAS..10414664D">2007PNAS..10414664D</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.0703836104">10.1073/pnas.0703836104</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1955458">1955458</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17726102">17726102</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;rft.atitle=Automated+de+novo+prediction+of+native-like+RNA+tertiary+structures&amp;rft.volume=104&amp;rft.issue=37&amp;rft.pages=14664-9&amp;rft.date=2007-09&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1955458&amp;rft_id=info%3Apmid%2F17726102&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0703836104&amp;rft_id=info%3Abibcode%2F2007PNAS..10414664D&amp;rft.aulast=Das&amp;rft.aufirst=R&amp;rft.au=Baker%2C+D&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1955458&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18579566-39"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18579566_39-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSharmaDingDokholyan2008" class="citation journal">Sharma S, Ding F, Dokholyan NV (September 2008). <a rel="nofollow" class="external text" href="http://bioinformatics.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=18579566">"iFoldRNA: three-dimensional RNA structure prediction and folding"</a>. <i>Bioinformatics</i>. <b>24</b> (17): 1951–2. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtn328">10.1093/bioinformatics/btn328</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2559968">2559968</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18579566">18579566</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=iFoldRNA%3A+three-dimensional+RNA+structure+prediction+and+folding&amp;rft.volume=24&amp;rft.issue=17&amp;rft.pages=1951-2&amp;rft.date=2008-09&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2559968&amp;rft_id=info%3Apmid%2F18579566&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtn328&amp;rft.aulast=Sharma&amp;rft.aufirst=S&amp;rft.au=Ding%2C+F&amp;rft.au=Dokholyan%2C+NV&amp;rft_id=http%3A%2F%2Fbioinformatics.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D18579566&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18322526-40"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18322526_40-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFParisienMajor2008" class="citation journal">Parisien M, Major F (2008). "The MC-Fold and MC-Sym pipeline infers RNA structure from sequence data". <i>Nature</i>. <b>452</b> (1): 51–55. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2008Natur.452...51P">2008Natur.452...51P</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnature06684">10.1038/nature06684</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18322526">18322526</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=The+MC-Fold+and+MC-Sym+pipeline+infers+RNA+structure+from+sequence+data&amp;rft.volume=452&amp;rft.issue=1&amp;rft.pages=51-55&amp;rft.date=2008&amp;rft_id=info%3Apmid%2F18322526&amp;rft_id=info%3Adoi%2F10.1038%2Fnature06684&amp;rft_id=info%3Abibcode%2F2008Natur.452...51P&amp;rft.aulast=Parisien&amp;rft.aufirst=M&amp;rft.au=Major%2C+F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid19144906-41"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19144906_41-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSC_FloresRB_Altman2010" class="citation journal">SC Flores; RB Altman (September 2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2924536">"Coarse-grained modeling of large RNA molecules with knowledge-based potentials and structural filters"</a>. <i>RNA</i>. <b>15</b> (9): 1769–1778. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1261%2Frna.1270809">10.1261/rna.1270809</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2924536">2924536</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19144906">19144906</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=RNA&amp;rft.atitle=Coarse-grained+modeling+of+large+RNA+molecules+with+knowledge-based+potentials+and+structural+filters&amp;rft.volume=15&amp;rft.issue=9&amp;rft.pages=1769-1778&amp;rft.date=2010-09&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2924536&amp;rft_id=info%3Apmid%2F19144906&amp;rft_id=info%3Adoi%2F10.1261%2Frna.1270809&amp;rft.au=SC+Flores&amp;rft.au=RB+Altman&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2924536&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid20651028-42"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid20651028_42-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFJonikasRadmerLaederach2009" class="citation journal">Jonikas MA, Radmer RJ, Laederach A,  et al. (February 2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2648710">"Turning limited experimental information into 3D models of RNA"</a>. <i>RNA</i>. <b>16</b> (2): 189–99. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1261%2Frna.2112110">10.1261/rna.2112110</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2648710">2648710</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20651028">20651028</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=RNA&amp;rft.atitle=Turning+limited+experimental+information+into+3D+models+of+RNA&amp;rft.volume=16&amp;rft.issue=2&amp;rft.pages=189-99&amp;rft.date=2009-02&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2648710&amp;rft_id=info%3Apmid%2F20651028&amp;rft_id=info%3Adoi%2F10.1261%2Frna.2112110&amp;rft.aulast=Jonikas&amp;rft.aufirst=MA&amp;rft.au=Radmer%2C+RJ&amp;rft.au=Laederach%2C+A&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2648710&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid22539264-43"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid22539264_43-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFPopendaSzachniukAntczakPurzycka2012" class="citation journal">Popenda M, Szachniuk M, Antczak M, Purzycka KJ, Lukasiak P, Bartol N, Blazewicz J, Adamiak RW (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3413140">"Automated 3D structure composition for large RNAs"</a>. <i>Nucleic Acids Res</i>. <b>40</b> (14): 1–12. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgks339">10.1093/nar/gks339</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3413140">3413140</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22539264">22539264</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=Automated+3D+structure+composition+for+large+RNAs&amp;rft.volume=40&amp;rft.issue=14&amp;rft.pages=1-12&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3413140&amp;rft_id=info%3Apmid%2F22539264&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgks339&amp;rft.aulast=Popenda&amp;rft.aufirst=M&amp;rft.au=Szachniuk%2C+M&amp;rft.au=Antczak%2C+M&amp;rft.au=Purzycka%2C+KJ&amp;rft.au=Lukasiak%2C+P&amp;rft.au=Bartol%2C+N&amp;rft.au=Blazewicz%2C+J&amp;rft.au=Adamiak%2C+RW&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3413140&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid12499300-48"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid12499300_48-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFPerriquetTouzetDauchet2003" class="citation journal">Perriquet O, Touzet H, Dauchet M (2003). "Finding the common structure shared by two homologous RNAs". <i>Bioinformatics</i>. <b>19</b> (1): 108–16. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2F19.1.108">10.1093/bioinformatics/19.1.108</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/12499300">12499300</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Finding+the+common+structure+shared+by+two+homologous+RNAs.&amp;rft.volume=19&amp;rft.issue=1&amp;rft.pages=108-16&amp;rft.date=2003&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F19.1.108&amp;rft_id=info%3Apmid%2F12499300&amp;rft.aulast=Perriquet&amp;rft.aufirst=O&amp;rft.au=Touzet%2C+H&amp;rft.au=Dauchet%2C+M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15215367-49"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15215367_49-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFTouzetPerriquet2004" class="citation journal">Touzet H, Perriquet O (Jul 1, 2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC441553">"CARNAC: folding families of related RNAs"</a>. <i>Nucleic Acids Res</i>. 32. <b>32</b> (Web Server issue): W142–5. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkh415">10.1093/nar/gkh415</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC441553">441553</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15215367">15215367</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=CARNAC%3A+folding+families+of+related+RNAs.&amp;rft.volume=32&amp;rft.issue=Web+Server+issue&amp;rft.pages=W142-5&amp;rft.date=2004-07-01&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC441553&amp;rft_id=info%3Apmid%2F15215367&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkh415&amp;rft.aulast=Touzet&amp;rft.aufirst=H&amp;rft.au=Perriquet%2C+O&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC441553&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid20843778-50"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid20843778_50-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMichiaki_HamadaKengo_SatoKiyoshi_Asai2011" class="citation journal">Michiaki Hamada; Kengo Sato; Kiyoshi Asai (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3025558">"Improving the accuracy of predicting secondary structure for aligned RNA sequences"</a>. <i>Nucleic Acids Res</i>. <b>39</b> (2): 393–402. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkq792">10.1093/nar/gkq792</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3025558">3025558</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20843778">20843778</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=Improving+the+accuracy+of+predicting+secondary+structure+for+aligned+RNA+sequences&amp;rft.volume=39&amp;rft.issue=2&amp;rft.pages=393-402&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3025558&amp;rft_id=info%3Apmid%2F20843778&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkq792&amp;rft.au=Michiaki+Hamada&amp;rft.au=Kengo+Sato&amp;rft.au=Kiyoshi+Asai&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3025558&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid19808876-51"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19808876_51-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMichiaki_HamadaKengo_SatoHisanori_KiryuToutai_Mituyama2009" class="citation journal">Michiaki Hamada; Kengo Sato; Hisanori Kiryu; Toutai Mituyama; Kiyoshi Asai (2009). "CentroidAlign: fast and accurate aligner for structured RNAs by maximizing expected sum-of-pairs score". <i>Bioinformatics</i>. <b>25</b> (24): 3236–43. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp580">10.1093/bioinformatics/btp580</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19808876">19808876</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=CentroidAlign%3A+fast+and+accurate+aligner+for+structured+RNAs+by+maximizing+expected+sum-of-pairs+score&amp;rft.volume=25&amp;rft.issue=24&amp;rft.pages=3236-43&amp;rft.date=2009&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtp580&amp;rft_id=info%3Apmid%2F19808876&amp;rft.au=Michiaki+Hamada&amp;rft.au=Kengo+Sato&amp;rft.au=Hisanori+Kiryu&amp;rft.au=Toutai+Mituyama&amp;rft.au=Kiyoshi+Asai&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16357030-52"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16357030_52-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFYaoWeinbergRuzzo2006" class="citation journal">Yao Z, Weinberg Z, Ruzzo WL (2006). "CMfinder--a covariance model based RNA motif finding algorithm". <i>Bioinformatics</i>. <b>22</b> (4): 445–52. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtk008">10.1093/bioinformatics/btk008</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16357030">16357030</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=CMfinder--a+covariance+model+based+RNA+motif+finding+algorithm&amp;rft.volume=22&amp;rft.issue=4&amp;rft.pages=445-52&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtk008&amp;rft_id=info%3Apmid%2F16357030&amp;rft.aulast=Yao&amp;rft.aufirst=Z&amp;rft.au=Weinberg%2C+Z&amp;rft.au=Ruzzo%2C+WL&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16952317-53"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16952317_53-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDowellEddy2006" class="citation journal">Dowell RD, Eddy SR (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1579236">"Efficient pairwise RNA structure prediction and alignment using sequence alignment constraints"</a>. <i>BMC Bioinformatics</i>. <b>7</b> (1): 400. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-7-400">10.1186/1471-2105-7-400</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1579236">1579236</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16952317">16952317</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Efficient+pairwise+RNA+structure+prediction+and+alignment+using+sequence+alignment+constraints&amp;rft.volume=7&amp;rft.issue=1&amp;rft.pages=400&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1579236&amp;rft_id=info%3Apmid%2F16952317&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-7-400&amp;rft.aulast=Dowell&amp;rft.aufirst=RD&amp;rft.au=Eddy%2C+SR&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1579236&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid23060618-54"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid23060618_54-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSatoKatoAkutsuAsai2012" class="citation journal">Sato K, Kato Y, Akutsu T, Asai K, Sakakibara Y (2012). "DAFS: simultaneous aligning and folding of RNA sequences via dual decomposition". <i>Bioinformatics</i>. <b>28</b> (24): 3218–24. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts612">10.1093/bioinformatics/bts612</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23060618">23060618</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=DAFS%3A+simultaneous+aligning+and+folding+of+RNA+sequences+via+dual+decomposition&amp;rft.volume=28&amp;rft.issue=24&amp;rft.pages=3218-24&amp;rft.date=2012&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbts612&amp;rft_id=info%3Apmid%2F23060618&amp;rft.aulast=Sato&amp;rft.aufirst=K&amp;rft.au=Kato%2C+Y&amp;rft.au=Akutsu%2C+T&amp;rft.au=Asai%2C+K&amp;rft.au=Sakakibara%2C+Y&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid11902836-55"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid11902836_55-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMathewsTurner2002" class="citation journal">Mathews DH, Turner DH (2002). "Dynalign: an algorithm for finding the secondary structure common to two RNA sequences". <i>J. Mol. Biol</i>. <b>317</b> (2): 191–203. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1006%2Fjmbi.2001.5351">10.1006/jmbi.2001.5351</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/11902836">11902836</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J.+Mol.+Biol.&amp;rft.atitle=Dynalign%3A+an+algorithm+for+finding+the+secondary+structure+common+to+two+RNA+sequences&amp;rft.volume=317&amp;rft.issue=2&amp;rft.pages=191-203&amp;rft.date=2002&amp;rft_id=info%3Adoi%2F10.1006%2Fjmbi.2001.5351&amp;rft_id=info%3Apmid%2F11902836&amp;rft.aulast=Mathews&amp;rft.aufirst=DH&amp;rft.au=Turner%2C+DH&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15731207-56"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15731207_56-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMathews_DH2005" class="citation journal">Mathews DH (2005). "Predicting a set of minimal free energy RNA secondary structures common to two sequences". <i>Bioinformatics</i>. <b>21</b> (10): 2246–53. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbti349">10.1093/bioinformatics/bti349</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15731207">15731207</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Predicting+a+set+of+minimal+free+energy+RNA+secondary+structures+common+to+two+sequences&amp;rft.volume=21&amp;rft.issue=10&amp;rft.pages=2246-53&amp;rft.date=2005&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbti349&amp;rft_id=info%3Apmid%2F15731207&amp;rft.au=Mathews+DH&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17445273-57"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17445273_57-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHarmanciSharmaMathews2007" class="citation journal">Harmanci AO, Sharma G, Mathews DH (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1868766">"Efficient pairwise RNA structure prediction using probabilistic alignment constraints in Dynalign"</a>. <i>BMC Bioinformatics</i>. <b>8</b> (1): 130. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-8-130">10.1186/1471-2105-8-130</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1868766">1868766</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17445273">17445273</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Efficient+pairwise+RNA+structure+prediction+using+probabilistic+alignment+constraints+in+Dynalign&amp;rft.volume=8&amp;rft.issue=1&amp;rft.pages=130&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1868766&amp;rft_id=info%3Apmid%2F17445273&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-8-130&amp;rft.aulast=Harmanci&amp;rft.aufirst=AO&amp;rft.au=Sharma%2C+G&amp;rft.au=Mathews%2C+DH&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1868766&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17324941-58"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17324941_58-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFTorarinssonHavgaardGorodkin2007" class="citation journal">Torarinsson E, Havgaard JH, Gorodkin J (2007). "Multiple structural alignment and clustering of RNA sequences". <i>Bioinformatics</i>. <b>23</b> (8): 926–32. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtm049">10.1093/bioinformatics/btm049</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17324941">17324941</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Multiple+structural+alignment+and+clustering+of+RNA+sequences&amp;rft.volume=23&amp;rft.issue=8&amp;rft.pages=926-32&amp;rft.date=2007&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtm049&amp;rft_id=info%3Apmid%2F17324941&amp;rft.aulast=Torarinsson&amp;rft.aufirst=E&amp;rft.au=Havgaard%2C+JH&amp;rft.au=Gorodkin%2C+J&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-978-3-642-33122-0_11-59"><span class="mw-cite-backlink"><b><a href="#cite_ref-978-3-642-33122-0_11_59-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMilo_NimrodZakov_ShayKatzenelson_ErezBachmat_Eitan2012" class="citation journal">Milo Nimrod; Zakov Shay; Katzenelson Erez; Bachmat Eitan; Dinitz Yefim; Ziv-Ukelson Michal (2012). "RNA Tree Comparisons via Unrooted Unordered Alignments". <i>Algorithms in Bioinformatics</i>. Lecture Notes in Computer Science. <b>7534</b>: 135–148. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2F978-3-642-33122-0_11">10.1007/978-3-642-33122-0_11</a>. <a href="/wiki/ISBN_(identifier)" class="mw-redirect" title="ISBN (identifier)">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-3-642-33121-3" title="Special:BookSources/978-3-642-33121-3"><bdi>978-3-642-33121-3</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Algorithms+in+Bioinformatics&amp;rft.atitle=RNA+Tree+Comparisons+via+Unrooted+Unordered+Alignments&amp;rft.volume=7534&amp;rft.pages=135-148&amp;rft.date=2012&amp;rft_id=info%3Adoi%2F10.1007%2F978-3-642-33122-0_11&amp;rft.isbn=978-3-642-33121-3&amp;rft.au=Milo+Nimrod&amp;rft.au=Zakov+Shay&amp;rft.au=Katzenelson+Erez&amp;rft.au=Bachmat+Eitan&amp;rft.au=Dinitz+Yefim&amp;rft.au=Ziv-Ukelson+Michal&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid23590940-60"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid23590940_60-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMilo_NimrodZakov_ShayKatzenelson_ErezBachmat_Eitan2013" class="citation journal">Milo Nimrod; Zakov Shay; Katzenelson Erez; Bachmat Eitan; Dinitz Yefim; Ziv-Ukelson Michal (2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3765143">"Unrooted unordered homeomorphic subtree alignment of RNA trees"</a>. <i>Algorithms for Molecular Biology</i>. <b>8</b> (1): 13. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1748-7188-8-13">10.1186/1748-7188-8-13</a>. <a href="/wiki/ISSN_(identifier)" class="mw-redirect" title="ISSN (identifier)">ISSN</a>&#160;<a rel="nofollow" class="external text" href="//www.worldcat.org/issn/1748-7188">1748-7188</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3765143">3765143</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23590940">23590940</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Algorithms+for+Molecular+Biology&amp;rft.atitle=Unrooted+unordered+homeomorphic+subtree+alignment+of+RNA+trees&amp;rft.volume=8&amp;rft.issue=1&amp;rft.pages=13&amp;rft.date=2013&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3765143&amp;rft.issn=1748-7188&amp;rft_id=info%3Apmid%2F23590940&amp;rft_id=info%3Adoi%2F10.1186%2F1748-7188-8-13&amp;rft.au=Milo+Nimrod&amp;rft.au=Zakov+Shay&amp;rft.au=Katzenelson+Erez&amp;rft.au=Bachmat+Eitan&amp;rft.au=Dinitz+Yefim&amp;rft.au=Ziv-Ukelson+Michal&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3765143&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid22689765-61"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid22689765_61-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid22689765_61-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-pmid22689765_61-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFHeyneCostaRoseBackofen2012" class="citation journal">Heyne S, Costa F, Rose D, Backofen R (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3371856">"GraphClust: alignment-free structural clustering of local RNA secondary structures"</a>. <i>Bioinformatics</i>. <b>28</b> (12): i224–i232. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts224">10.1093/bioinformatics/bts224</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3371856">3371856</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22689765">22689765</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=GraphClust%3A+alignment-free+structural+clustering+of+local+RNA+secondary+structures&amp;rft.volume=28&amp;rft.issue=12&amp;rft.pages=i224-i232&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3371856&amp;rft_id=info%3Apmid%2F22689765&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbts224&amp;rft.aulast=Heyne&amp;rft.aufirst=S&amp;rft.au=Costa%2C+F&amp;rft.au=Rose%2C+D&amp;rft.au=Backofen%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3371856&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16495232-62"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16495232_62-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFBindewaldShapiro2006" class="citation journal">Bindewald E, Shapiro BA (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1383574">"RNA secondary structure prediction from sequence alignments using a network of k-nearest neighbor classifiers"</a>. <i>RNA</i>. <b>12</b> (3): 342–52. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1261%2Frna.2164906">10.1261/rna.2164906</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1383574">1383574</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16495232">16495232</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=RNA&amp;rft.atitle=RNA+secondary+structure+prediction+from+sequence+alignments+using+a+network+of+k-nearest+neighbor+classifiers&amp;rft.volume=12&amp;rft.issue=3&amp;rft.pages=342-52&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1383574&amp;rft_id=info%3Apmid%2F16495232&amp;rft_id=info%3Adoi%2F10.1261%2Frna.2164906&amp;rft.aulast=Bindewald&amp;rft.aufirst=E&amp;rft.au=Shapiro%2C+BA&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1383574&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17662141-63"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17662141_63-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFBauerKlauReinert2007" class="citation journal">Bauer M, Klau GW, Reinert K (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1955456">"Accurate multiple sequence-structure alignment of RNA sequences using combinatorial optimization"</a>. <i>BMC Bioinformatics</i>. <b>8</b> (1): 271. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-8-271">10.1186/1471-2105-8-271</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1955456">1955456</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17662141">17662141</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Accurate+multiple+sequence-structure+alignment+of+RNA+sequences+using+combinatorial+optimization.&amp;rft.volume=8&amp;rft.issue=1&amp;rft.pages=271&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1955456&amp;rft_id=info%3Apmid%2F17662141&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-8-271&amp;rft.aulast=Bauer&amp;rft.aufirst=M&amp;rft.au=Klau%2C+GW&amp;rft.au=Reinert%2C+K&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1955456&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17432929-64"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17432929_64-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWillReicheHofackerStadler2007" class="citation journal">Will S, Reiche K, Hofacker IL, Stadler PF, Backofen R (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1851984">"Inferring noncoding RNA families and classes by means of genome-scale structure-based clustering"</a>. <i>PLOS Comput. Biol</i>. <b>3</b> (4): e65. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2007PLSCB...3...65W">2007PLSCB...3...65W</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pcbi.0030065">10.1371/journal.pcbi.0030065</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1851984">1851984</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17432929">17432929</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+Comput.+Biol.&amp;rft.atitle=Inferring+noncoding+RNA+families+and+classes+by+means+of+genome-scale+structure-based+clustering.&amp;rft.volume=3&amp;rft.issue=4&amp;rft.pages=e65&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1851984&amp;rft_id=info%3Apmid%2F17432929&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.0030065&amp;rft_id=info%3Abibcode%2F2007PLSCB...3...65W&amp;rft.aulast=Will&amp;rft.aufirst=S&amp;rft.au=Reiche%2C+K&amp;rft.au=Hofacker%2C+IL&amp;rft.au=Stadler%2C+PF&amp;rft.au=Backofen%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1851984&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17038338-65"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17038338_65-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLindgreenGardnerKrogh2006" class="citation journal">Lindgreen S, Gardner PP, Krogh A (2006). "Measuring covariation in RNA alignments: physical realism improves information measures". <i>Bioinformatics</i>. <b>22</b> (24): 2988–95. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl514">10.1093/bioinformatics/btl514</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17038338">17038338</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Measuring+covariation+in+RNA+alignments%3A+physical+realism+improves+information+measures&amp;rft.volume=22&amp;rft.issue=24&amp;rft.pages=2988-95&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtl514&amp;rft_id=info%3Apmid%2F17038338&amp;rft.aulast=Lindgreen&amp;rft.aufirst=S&amp;rft.au=Gardner%2C+PP&amp;rft.au=Krogh%2C+A&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18006551-66"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18006551_66-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLindgreenGardnerKrogh2007" class="citation journal">Lindgreen S, Gardner PP, Krogh A (2007). "MASTR: multiple alignment and structure prediction of non-coding RNAs using simulated annealing". <i>Bioinformatics</i>. <b>23</b> (24): 3304–11. <a href="/wiki/CiteSeerX_(identifier)" class="mw-redirect" title="CiteSeerX (identifier)">CiteSeerX</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.563.7072">10.1.1.563.7072</a></span>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtm525">10.1093/bioinformatics/btm525</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18006551">18006551</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=MASTR%3A+multiple+alignment+and+structure+prediction+of+non-coding+RNAs+using+simulated+annealing&amp;rft.volume=23&amp;rft.issue=24&amp;rft.pages=3304-11&amp;rft.date=2007&amp;rft_id=%2F%2Fciteseerx.ist.psu.edu%2Fviewdoc%2Fsummary%3Fdoi%3D10.1.1.563.7072&amp;rft_id=info%3Apmid%2F18006551&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtm525&amp;rft.aulast=Lindgreen&amp;rft.aufirst=S&amp;rft.au=Gardner%2C+PP&amp;rft.au=Krogh%2C+A&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid21193521-67"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid21193521_67-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFXuMathews2011" class="citation journal">Xu Z, Mathews DH (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3042186">"Multilign: an algorithm to predict secondary structures conserved in multiple RNA sequences"</a>. <i>Bioinformatics</i>. <b>27</b> (5): 626–632. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq726">10.1093/bioinformatics/btq726</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3042186">3042186</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21193521">21193521</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Multilign%3A+an+algorithm+to+predict+secondary+structures+conserved+in+multiple+RNA+sequences&amp;rft.volume=27&amp;rft.issue=5&amp;rft.pages=626-632&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3042186&amp;rft_id=info%3Apmid%2F21193521&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtq726&amp;rft.aulast=Xu&amp;rft.aufirst=Z&amp;rft.au=Mathews%2C+DH&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3042186&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17459961-68"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17459961_68-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFKiryuTabeiKinAsai2007" class="citation journal">Kiryu H, Tabei Y, Kin T, Asai K (2007). "Murlet: a practical multiple alignment tool for structural RNA sequences". <i>Bioinformatics</i>. <b>23</b> (13): 1588–98. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtm146">10.1093/bioinformatics/btm146</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17459961">17459961</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Murlet%3A+a+practical+multiple+alignment+tool+for+structural+RNA+sequences&amp;rft.volume=23&amp;rft.issue=13&amp;rft.pages=1588-98&amp;rft.date=2007&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtm146&amp;rft_id=info%3Apmid%2F17459961&amp;rft.aulast=Kiryu&amp;rft.aufirst=H&amp;rft.au=Tabei%2C+Y&amp;rft.au=Kin%2C+T&amp;rft.au=Asai%2C+K&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-MXSCARNA-69"><span class="mw-cite-backlink"><b><a href="#cite_ref-MXSCARNA_69-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFTabeiKiryuKinAsai2008" class="citation journal">Tabei Y, Kiryu H, Kin T, Asai K (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2375124">"A fast structural multiple alignment method for long RNA sequences"</a>. <i>BMC Bioinformatics</i>. <b>9</b> (1): 33. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-9-33">10.1186/1471-2105-9-33</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2375124">2375124</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18215258">18215258</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=A+fast+structural+multiple+alignment+method+for+long+RNA+sequences&amp;rft.volume=9&amp;rft.issue=1&amp;rft.pages=33&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2375124&amp;rft_id=info%3Apmid%2F18215258&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-9-33&amp;rft.aulast=Tabei&amp;rft.aufirst=Y&amp;rft.au=Kiryu%2C+H&amp;rft.au=Kin%2C+T&amp;rft.au=Asai%2C+K&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2375124&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18304945-70"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18304945_70-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHarmanciSharmaMathews2008" class="citation journal">Harmanci AO, Sharma G, Mathews DH (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2367733">"PARTS: probabilistic alignment for RNA joinT secondary structure prediction"</a>. <i>Nucleic Acids Res</i>. <b>36</b> (7): 2406–17. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkn043">10.1093/nar/gkn043</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2367733">2367733</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18304945">18304945</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=PARTS%3A+probabilistic+alignment+for+RNA+joinT+secondary+structure+prediction.&amp;rft.volume=36&amp;rft.issue=7&amp;rft.pages=2406-17&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2367733&amp;rft_id=info%3Apmid%2F18304945&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkn043&amp;rft.aulast=Harmanci&amp;rft.aufirst=AO&amp;rft.au=Sharma%2C+G&amp;rft.au=Mathews%2C+DH&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2367733&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid10383470-71"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid10383470_71-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFKnudsenHein1999" class="citation journal">Knudsen B, Hein J (1999). "RNA secondary structure prediction using stochastic context-free grammars and evolutionary history". <i>Bioinformatics</i>. <b>15</b> (6): 446–54. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2F15.6.446">10.1093/bioinformatics/15.6.446</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/10383470">10383470</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RNA+secondary+structure+prediction+using+stochastic+context-free+grammars+and+evolutionary+history&amp;rft.volume=15&amp;rft.issue=6&amp;rft.pages=446-54&amp;rft.date=1999&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F15.6.446&amp;rft_id=info%3Apmid%2F10383470&amp;rft.aulast=Knudsen&amp;rft.aufirst=B&amp;rft.au=Hein%2C+J&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid12824339-72"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid12824339_72-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFKnudsenHein2003" class="citation journal">Knudsen B, Hein J (2003). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC169020">"Pfold: RNA secondary structure prediction using stochastic context-free grammars"</a>. <i>Nucleic Acids Res</i>. <b>31</b> (13): 3423–8. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkg614">10.1093/nar/gkg614</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC169020">169020</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/12824339">12824339</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=Pfold%3A+RNA+secondary+structure+prediction+using+stochastic+context-free+grammars&amp;rft.volume=31&amp;rft.issue=13&amp;rft.pages=3423-8&amp;rft.date=2003&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC169020&amp;rft_id=info%3Apmid%2F12824339&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkg614&amp;rft.aulast=Knudsen&amp;rft.aufirst=B&amp;rft.au=Hein%2C+J&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC169020&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18836192-73"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18836192_73-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSeemannGorodkinBackofen2008" class="citation journal">Seemann SE, Gorodkin J, Backofen R (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2582601">"Unifying evolutionary and thermodynamic information for RNA folding of multiple alignments"</a>. <i>Nucleic Acids Res</i>. <b>36</b> (20): 6355–62. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkn544">10.1093/nar/gkn544</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2582601">2582601</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18836192">18836192</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=Unifying+evolutionary+and+thermodynamic+information+for+RNA+folding+of+multiple+alignments&amp;rft.volume=36&amp;rft.issue=20&amp;rft.pages=6355-62&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2582601&amp;rft_id=info%3Apmid%2F18836192&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkn544&amp;rft.aulast=Seemann&amp;rft.aufirst=SE&amp;rft.au=Gorodkin%2C+J&amp;rft.au=Backofen%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2582601&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid22796961-74"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid22796961_74-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDooseMetzler2012" class="citation journal">Doose G, Metzler D (2012). "Bayesian sampling of evolutionarily conserved RNA secondary structures with pseudoknots". <i>Bioinformatics</i>. <b>28</b> (17): 2242–2248. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts369">10.1093/bioinformatics/bts369</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22796961">22796961</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Bayesian+sampling+of+evolutionarily+conserved+RNA+secondary+structures+with+pseudoknots.&amp;rft.volume=28&amp;rft.issue=17&amp;rft.pages=2242-2248&amp;rft.date=2012&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbts369&amp;rft_id=info%3Apmid%2F22796961&amp;rft.aulast=Doose&amp;rft.aufirst=G&amp;rft.au=Metzler%2C+D&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15073017-75"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15073017_75-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHofackerBernhartStadler2004" class="citation journal">Hofacker IL, Bernhart SH, Stadler PF (2004). "Alignment of RNA base pairing probability matrices". <i>Bioinformatics</i>. <b>20</b> (14): 2222–7. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbth229">10.1093/bioinformatics/bth229</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15073017">15073017</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Alignment+of+RNA+base+pairing+probability+matrices&amp;rft.volume=20&amp;rft.issue=14&amp;rft.pages=2222-7&amp;rft.date=2004&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbth229&amp;rft_id=info%3Apmid%2F15073017&amp;rft.aulast=Hofacker&amp;rft.aufirst=IL&amp;rft.au=Bernhart%2C+SH&amp;rft.au=Stadler%2C+PF&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid21788211-76"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid21788211_76-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWeiAlpertLawrence2011" class="citation journal">Wei D, Alpert LV, Lawrence CE (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3167047">"RNAG: a new Gibbs sampler for predicting RNA secondary structure for unaligned sequence"</a>. <i>Bioinformatics</i>. <b>27</b> (18): 2486–2493. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtr421">10.1093/bioinformatics/btr421</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3167047">3167047</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21788211">21788211</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RNAG%3A+a+new+Gibbs+sampler+for+predicting+RNA+secondary+structure+for+unaligned+sequence&amp;rft.volume=27&amp;rft.issue=18&amp;rft.pages=2486-2493&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3167047&amp;rft_id=info%3Apmid%2F21788211&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtr421&amp;rft.aulast=Wei&amp;rft.aufirst=D&amp;rft.au=Alpert%2C+LV&amp;rft.au=Lawrence%2C+CE&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3167047&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18420654-77"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18420654_77-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWilmHigginsNotredame2008" class="citation journal">Wilm A, Higgins DG, Notredame C (May 2008). <a rel="nofollow" class="external text" href="http://nar.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=18420654">"R-Coffee: a method for multiple alignment of non-coding RNA"</a>. <i>Nucleic Acids Res</i>. <b>36</b> (9): e52. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkn174">10.1093/nar/gkn174</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2396437">2396437</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18420654">18420654</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=R-Coffee%3A+a+method+for+multiple+alignment+of+non-coding+RNA&amp;rft.volume=36&amp;rft.issue=9&amp;rft.pages=e52&amp;rft.date=2008-05&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2396437&amp;rft_id=info%3Apmid%2F18420654&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkn174&amp;rft.aulast=Wilm&amp;rft.aufirst=A&amp;rft.au=Higgins%2C+DG&amp;rft.au=Notredame%2C+C&amp;rft_id=http%3A%2F%2Fnar.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D18420654&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18483080-78"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18483080_78-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMorettiWilmHigginsXenarios2008" class="citation journal">Moretti S, Wilm A, Higgins DG, Xenarios I, Notredame C (July 2008). <a rel="nofollow" class="external text" href="http://nar.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=18483080">"R-Coffee: a web server for accurately aligning noncoding RNA sequences"</a>. <i>Nucleic Acids Res</i>. <b>36</b> (Web Server issue): W10–3. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkn278">10.1093/nar/gkn278</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2447777">2447777</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18483080">18483080</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=R-Coffee%3A+a+web+server+for+accurately+aligning+noncoding+RNA+sequences&amp;rft.volume=36&amp;rft.issue=Web+Server+issue&amp;rft.pages=W10-3&amp;rft.date=2008-07&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2447777&amp;rft_id=info%3Apmid%2F18483080&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkn278&amp;rft.aulast=Moretti&amp;rft.aufirst=S&amp;rft.au=Wilm%2C+A&amp;rft.au=Higgins%2C+DG&amp;rft.au=Xenarios%2C+I&amp;rft.au=Notredame%2C+C&amp;rft_id=http%3A%2F%2Fnar.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D18483080&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid21507242-79"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid21507242_79-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHarmanciSharmaMathews2011" class="citation journal">Harmanci AO, Sharma G, Mathews DH (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3120699">"TurboFold: iterative probabilistic estimation of secondary structures for multiple RNA sequence"</a>. <i>BMC Bioinformatics</i>. <b>12</b> (1): 108. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-12-108">10.1186/1471-2105-12-108</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3120699">3120699</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21507242">21507242</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=TurboFold%3A+iterative+probabilistic+estimation+of+secondary+structures+for+multiple+RNA+sequence&amp;rft.volume=12&amp;rft.issue=1&amp;rft.pages=108&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3120699&amp;rft_id=info%3Apmid%2F21507242&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-12-108&amp;rft.aulast=Harmanci&amp;rft.aufirst=AO&amp;rft.au=Sharma%2C+G&amp;rft.au=Mathews%2C+DH&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3120699&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid22285566-80"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid22285566_80-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSeetinMathews2012" class="citation journal">Seetin MG, Mathews DH (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3307117">"TurboKnot: rapid prediction of conserved RNA secondary structures including pseudoknots"</a>. <i>Bioinformatics</i>. <b>28</b> (6): 792–798. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts044">10.1093/bioinformatics/bts044</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3307117">3307117</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22285566">22285566</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=TurboKnot%3A+rapid+prediction+of+conserved+RNA+secondary+structures+including+pseudoknots&amp;rft.volume=28&amp;rft.issue=6&amp;rft.pages=792-798&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3307117&amp;rft_id=info%3Apmid%2F22285566&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbts044&amp;rft.aulast=Seetin&amp;rft.aufirst=MG&amp;rft.au=Mathews%2C+DH&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3307117&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-81"><span class="mw-cite-backlink"><b><a href="#cite_ref-81">^</a></b></span> <span class="reference-text"><cite id="CITEREFRivasClementsEddy2017" class="citation journal">Rivas, E; Clements, J; Eddy, SR (January 2017). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC5554622">"A statistical test for conserved RNA structure shows lack of evidence for structure in lncRNAs"</a>. <i>Nature Methods</i>. <b>14</b> (1): 45–48. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnmeth.4066">10.1038/nmeth.4066</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC5554622">5554622</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/27819659">27819659</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Methods&amp;rft.atitle=A+statistical+test+for+conserved+RNA+structure+shows+lack+of+evidence+for+structure+in+lncRNAs.&amp;rft.volume=14&amp;rft.issue=1&amp;rft.pages=45-48&amp;rft.date=2017-01&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5554622&amp;rft_id=info%3Apmid%2F27819659&amp;rft_id=info%3Adoi%2F10.1038%2Fnmeth.4066&amp;rft.aulast=Rivas&amp;rft.aufirst=E&amp;rft.au=Clements%2C+J&amp;rft.au=Eddy%2C+SR&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5554622&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid12079347-82"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid12079347_82-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHofackerFeketeStadler2002" class="citation journal">Hofacker IL, Fekete M, Stadler PF (2002). "Secondary structure prediction for aligned RNA sequences". <i>J. Mol. Biol</i>. <b>319</b> (5): 1059–66. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2FS0022-2836%2802%2900308-X">10.1016/S0022-2836(02)00308-X</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/12079347">12079347</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J.+Mol.+Biol.&amp;rft.atitle=Secondary+structure+prediction+for+aligned+RNA+sequences&amp;rft.volume=319&amp;rft.issue=5&amp;rft.pages=1059-66&amp;rft.date=2002&amp;rft_id=info%3Adoi%2F10.1016%2FS0022-2836%2802%2900308-X&amp;rft_id=info%3Apmid%2F12079347&amp;rft.aulast=Hofacker&amp;rft.aufirst=IL&amp;rft.au=Fekete%2C+M&amp;rft.au=Stadler%2C+PF&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17020924-83"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17020924_83-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFVoß,_Björn2006" class="citation journal">Voß, Björn (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1636479">"Structural analysis of aligned RNAs"</a>. <i>Nucleic Acids Research</i>. <b>34</b> (19): 5471–5481. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkl692">10.1093/nar/gkl692</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1636479">1636479</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17020924">17020924</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=Structural+analysis+of+aligned+RNAs&amp;rft.volume=34&amp;rft.issue=19&amp;rft.pages=5471-5481&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1636479&amp;rft_id=info%3Apmid%2F17020924&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkl692&amp;rft.au=Vo%C3%9F%2C+Bj%C3%B6rn&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1636479&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16020472-84"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16020472_84-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFReederGiegerich2005" class="citation journal">Reeder J, Giegerich R (2005). "Consensus shapes: an alternative to the Sankoff algorithm for RNA consensus structure prediction". <i>Bioinformatics</i>. <b>21</b> (17): 3516–23. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbti577">10.1093/bioinformatics/bti577</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16020472">16020472</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Consensus+shapes%3A+an+alternative+to+the+Sankoff+algorithm+for+RNA+consensus+structure+prediction&amp;rft.volume=21&amp;rft.issue=17&amp;rft.pages=3516-23&amp;rft.date=2005&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbti577&amp;rft_id=info%3Apmid%2F16020472&amp;rft.aulast=Reeder&amp;rft.aufirst=J&amp;rft.au=Giegerich%2C+R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16452790-85"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16452790_85-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHöchsmannTöllerGiegerichKurtz2003" class="citation journal">Höchsmann M, Töller T, Giegerich R, Kurtz S (2003). "Local similarity in RNA secondary structures". <i>Proc IEEE Comput Soc Bioinform Conf</i>. <b>2</b>: 159–68. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16452790">16452790</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proc+IEEE+Comput+Soc+Bioinform+Conf&amp;rft.atitle=Local+similarity+in+RNA+secondary+structures&amp;rft.volume=2&amp;rft.pages=159-68&amp;rft.date=2003&amp;rft_id=info%3Apmid%2F16452790&amp;rft.aulast=H%C3%B6chsmann&amp;rft.aufirst=M&amp;rft.au=T%C3%B6ller%2C+T&amp;rft.au=Giegerich%2C+R&amp;rft.au=Kurtz%2C+S&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17048408-86"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17048408_86-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHöchsmannVossGiegerich2004" class="citation journal">Höchsmann M, Voss B, Giegerich R (2004). "Pure multiple RNA secondary structure alignments: a progressive profile approach". <i>IEEE/ACM Trans Comput Biol Bioinform</i>. <b>1</b> (1): 53–62. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1109%2FTCBB.2004.11">10.1109/TCBB.2004.11</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17048408">17048408</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=IEEE%2FACM+Trans+Comput+Biol+Bioinform&amp;rft.atitle=Pure+multiple+RNA+secondary+structure+alignments%3A+a+progressive+profile+approach&amp;rft.volume=1&amp;rft.issue=1&amp;rft.pages=53-62&amp;rft.date=2004&amp;rft_id=info%3Adoi%2F10.1109%2FTCBB.2004.11&amp;rft_id=info%3Apmid%2F17048408&amp;rft.aulast=H%C3%B6chsmann&amp;rft.aufirst=M&amp;rft.au=Voss%2C+B&amp;rft.au=Giegerich%2C+R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16908501-87"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16908501_87-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHamadaTsudaKudoKin2006" class="citation journal">Hamada M, Tsuda K, Kudo T, Kin T, Asai K (2006). "Mining frequent stem patterns from unaligned RNA sequences". <i>Bioinformatics</i>. <b>22</b> (20): 2480–7. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl431">10.1093/bioinformatics/btl431</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16908501">16908501</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Mining+frequent+stem+patterns+from+unaligned+RNA+sequences&amp;rft.volume=22&amp;rft.issue=20&amp;rft.pages=2480-7&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtl431&amp;rft_id=info%3Apmid%2F16908501&amp;rft.aulast=Hamada&amp;rft.aufirst=M&amp;rft.au=Tsuda%2C+K&amp;rft.au=Kudo%2C+T&amp;rft.au=Kin%2C+T&amp;rft.au=Asai%2C+K&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17537756-88"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17537756_88-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFXuJiStormo2007" class="citation journal">Xu X, Ji Y, Stormo GD (2007). "RNA Sampler: a new sampling based algorithm for common RNA secondary structure prediction and structural alignment". <i>Bioinformatics</i>. <b>23</b> (15): 1883–91. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtm272">10.1093/bioinformatics/btm272</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17537756">17537756</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RNA+Sampler%3A+a+new+sampling+based+algorithm+for+common+RNA+secondary+structure+prediction+and+structural+alignment&amp;rft.volume=23&amp;rft.issue=15&amp;rft.pages=1883-91&amp;rft.date=2007&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtm272&amp;rft_id=info%3Apmid%2F17537756&amp;rft.aulast=Xu&amp;rft.aufirst=X&amp;rft.au=Ji%2C+Y&amp;rft.au=Stormo%2C+GD&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16690634-89"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16690634_89-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFTabeiTsudaKinAsai2006" class="citation journal">Tabei Y, Tsuda K, Kin T, Asai K (2006). "SCARNA: fast and accurate structural alignment of RNA sequences by matching fixed-length stem fragments". <i>Bioinformatics</i>. <b>22</b> (14): 1723–9. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl177">10.1093/bioinformatics/btl177</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16690634">16690634</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=SCARNA%3A+fast+and+accurate+structural+alignment+of+RNA+sequences+by+matching+fixed-length+stem+fragments&amp;rft.volume=22&amp;rft.issue=14&amp;rft.pages=1723-9&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtl177&amp;rft_id=info%3Apmid%2F16690634&amp;rft.aulast=Tabei&amp;rft.aufirst=Y&amp;rft.au=Tsuda%2C+K&amp;rft.au=Kin%2C+T&amp;rft.au=Asai%2C+K&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17696604-90"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17696604_90-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMeyerMiklós2007" class="citation journal">Meyer IM, Miklós I (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1941756">"SimulFold: simultaneously inferring RNA structures including pseudoknots, alignments, and trees using a Bayesian MCMC framework"</a>. <i>PLOS Comput. Biol</i>. <b>3</b> (8): e149. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2007PLSCB...3..149M">2007PLSCB...3..149M</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pcbi.0030149">10.1371/journal.pcbi.0030149</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1941756">1941756</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17696604">17696604</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+Comput.+Biol.&amp;rft.atitle=SimulFold%3A+simultaneously+inferring+RNA+structures+including+pseudoknots%2C+alignments%2C+and+trees+using+a+Bayesian+MCMC+framework&amp;rft.volume=3&amp;rft.issue=8&amp;rft.pages=e149&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1941756&amp;rft_id=info%3Apmid%2F17696604&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.0030149&amp;rft_id=info%3Abibcode%2F2007PLSCB...3..149M&amp;rft.aulast=Meyer&amp;rft.aufirst=IM&amp;rft.au=Mikl%C3%B3s%2C+I&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1941756&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15790387-91"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15790387_91-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHolmes_I2005" class="citation journal">Holmes I (2005). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1090553">"Accelerated probabilistic inference of RNA structure evolution"</a>. <i>BMC Bioinformatics</i>. <b>6</b> (1): 73. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-6-73">10.1186/1471-2105-6-73</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1090553">1090553</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15790387">15790387</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Accelerated+probabilistic+inference+of+RNA+structure+evolution&amp;rft.volume=6&amp;rft.issue=1&amp;rft.pages=73&amp;rft.date=2005&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1090553&amp;rft_id=info%3Apmid%2F15790387&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-6-73&amp;rft.au=Holmes+I&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1090553&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16613908-92"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16613908_92-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDalliWilmMainzSteger2006" class="citation journal">Dalli D, Wilm A, Mainz I, Steger G (2006). "STRAL: progressive alignment of non-coding RNA using base pairing probability vectors in quadratic time". <i>Bioinformatics</i>. <b>22</b> (13): 1593–9. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl142">10.1093/bioinformatics/btl142</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16613908">16613908</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=STRAL%3A+progressive+alignment+of+non-coding+RNA+using+base+pairing+probability+vectors+in+quadratic+time&amp;rft.volume=22&amp;rft.issue=13&amp;rft.pages=1593-9&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtl142&amp;rft_id=info%3Apmid%2F16613908&amp;rft.aulast=Dalli&amp;rft.aufirst=D&amp;rft.au=Wilm%2C+A&amp;rft.au=Mainz%2C+I&amp;rft.au=Steger%2C+G&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-&#123;&#123;&#80;MID&#124;20047957&#125;&#125;-93"><span class="mw-cite-backlink"><b><a href="#cite_ref-{{PMID|20047957}}_93-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFEngelenTahi2010" class="citation journal">Engelen S, Tahi F (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2853104">"Tfold: efficient in silico prediction of non-coding RNA secondary structures"</a>. <i>Nucleic Acids Res</i>. <b>38</b> (7): 2453–66. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkp1067">10.1093/nar/gkp1067</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2853104">2853104</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20047957">20047957</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=Tfold%3A+efficient+in+silico+prediction+of+non-coding+RNA+secondary+structures&amp;rft.volume=38&amp;rft.issue=7&amp;rft.pages=2453-66&amp;rft.date=2010&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2853104&amp;rft_id=info%3Apmid%2F20047957&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkp1067&amp;rft.aulast=Engelen&amp;rft.aufirst=S&amp;rft.au=Tahi%2C+F&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2853104&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18492721-94"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18492721_94-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFTorarinssonLindgreen2008" class="citation journal">Torarinsson E, Lindgreen S (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2447782">"WAR: Webserver for aligning structural RNAs"</a>. <i>Nucleic Acids Res</i>. <b>36</b> (Web Server issue): W79–84. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkn275">10.1093/nar/gkn275</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2447782">2447782</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18492721">18492721</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=WAR%3A+Webserver+for+aligning+structural+RNAs.&amp;rft.volume=36&amp;rft.issue=Web+Server+issue&amp;rft.pages=W79-84&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2447782&amp;rft_id=info%3Apmid%2F18492721&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkn275&amp;rft.aulast=Torarinsson&amp;rft.aufirst=E&amp;rft.au=Lindgreen%2C+S&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2447782&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17018148-95"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid17018148_95-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid17018148_95-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFKlostermanUzilovBendañaBradley2006" class="citation journal">Klosterman PS, Uzilov AV, Bendaña YR, Bradley RK, Chao S, Kosiol C, Goldman N, Holmes I (October 2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1622757">"XRate: a fast prototyping, training and annotation tool for phylo-grammars"</a>. <i>BMC Bioinformatics</i>. <b>7</b> (1): 428. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-7-428">10.1186/1471-2105-7-428</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1622757">1622757</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17018148">17018148</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=XRate%3A+a+fast+prototyping%2C+training+and+annotation+tool+for+phylo-grammars&amp;rft.volume=7&amp;rft.issue=1&amp;rft.pages=428&amp;rft.date=2006-10&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1622757&amp;rft_id=info%3Apmid%2F17018148&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-7-428&amp;rft.aulast=Klosterman&amp;rft.aufirst=PS&amp;rft.au=Uzilov%2C+AV&amp;rft.au=Benda%C3%B1a%2C+YR&amp;rft.au=Bradley%2C+RK&amp;rft.au=Chao%2C+S&amp;rft.au=Kosiol%2C+C&amp;rft.au=Goldman%2C+N&amp;rft.au=Holmes%2C+I&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1622757&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-PMC3125805-96"><span class="mw-cite-backlink"><b><a href="#cite_ref-PMC3125805_96-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFEggenhofer,_Tafer,_Stadler,_Hofacker2011" class="citation journal">Eggenhofer, Tafer, Stadler, Hofacker (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3125805">"RNApredator: fast accessibility-based prediction of sRNA targets"</a>. <i>Nucleic Acids Res</i>. <b>39</b> (suppl 2: W149–W154): W149–W154. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkr467">10.1093/nar/gkr467</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3125805">3125805</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21672960">21672960</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=RNApredator%3A+fast+accessibility-based+prediction+of+sRNA+targets.&amp;rft.volume=39&amp;rft.issue=suppl+2%3A+W149%E2%80%93W154&amp;rft.pages=W149-W154&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3125805&amp;rft_id=info%3Apmid%2F21672960&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkr467&amp;rft.au=Eggenhofer%2C+Tafer%2C+Stadler%2C+Hofacker&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3125805&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><span class="cs1-maint citation-comment">CS1 maint: multiple names: authors list (<a href="/wiki/Category:CS1_maint:_multiple_names:_authors_list" title="Category:CS1 maint: multiple names: authors list">link</a>)</span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16403789-97"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16403789_97-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFGerlachGiegerich2006" class="citation journal">Gerlach W, Giegerich R (2006). "GUUGle: a utility for fast exact matching under RNA complementary rules including G-U base pairing". <i>Bioinformatics</i>. <b>22</b> (6): 762–764. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtk041">10.1093/bioinformatics/btk041</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16403789">16403789</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=GUUGle%3A+a+utility+for+fast+exact+matching+under+RNA+complementary+rules+including+G-U+base+pairing.&amp;rft.volume=22&amp;rft.issue=6&amp;rft.pages=762-764&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtk041&amp;rft_id=info%3Apmid%2F16403789&amp;rft.aulast=Gerlach&amp;rft.aufirst=W&amp;rft.au=Giegerich%2C+R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid28472523-98"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid28472523_98-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMannWrightBackofen2017" class="citation journal">Mann M, Wright PR, Backofen R (2017). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC5570192">"IntaRNA 2.0: enhanced and customizable prediction of RNA–RNA interactions"</a>. <i>Nucleic Acids Res</i>. <b>45</b> (Web Server): W435–W439. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkx279">10.1093/nar/gkx279</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC5570192">5570192</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/28472523">28472523</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=IntaRNA+2.0%3A+enhanced+and+customizable+prediction+of+RNA%E2%80%93RNA+interactions.&amp;rft.volume=45&amp;rft.issue=Web+Server&amp;rft.pages=W435-W439&amp;rft.date=2017&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5570192&amp;rft_id=info%3Apmid%2F28472523&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkx279&amp;rft.aulast=Mann&amp;rft.aufirst=M&amp;rft.au=Wright%2C+PR&amp;rft.au=Backofen%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5570192&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid24838564-99"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid24838564_99-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid24838564_99-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFWrightGeorgMannSorescu2014" class="citation journal">Wright PR, Georg J, Mann M, Sorescu DA, Richter AS, Lott S, Kleinkauf R, Hess WR, Backofen R (2014). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4086077">"CopraRNA and IntaRNA: predicting small RNA targets, networks and interaction domains"</a>. <i>Nucleic Acids Res</i>. <b>42</b> (Web Server): W119–23. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgku359">10.1093/nar/gku359</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4086077">4086077</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24838564">24838564</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=CopraRNA+and+IntaRNA%3A+predicting+small+RNA+targets%2C+networks+and+interaction+domains.&amp;rft.volume=42&amp;rft.issue=Web+Server&amp;rft.pages=W119-23&amp;rft.date=2014&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4086077&amp;rft_id=info%3Apmid%2F24838564&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgku359&amp;rft.aulast=Wright&amp;rft.aufirst=PR&amp;rft.au=Georg%2C+J&amp;rft.au=Mann%2C+M&amp;rft.au=Sorescu%2C+DA&amp;rft.au=Richter%2C+AS&amp;rft.au=Lott%2C+S&amp;rft.au=Kleinkauf%2C+R&amp;rft.au=Hess%2C+WR&amp;rft.au=Backofen%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4086077&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18940824-100"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18940824_100-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFBuschRichterBackofen2008" class="citation journal">Busch A, Richter AS, Backofen R (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2639303">"IntaRNA: efficient prediction of bacterial sRNA targets incorporating target site accessibility and seed regions"</a>. <i>Bioinformatics</i>. <b>24</b> (24): 2849–56. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtn544">10.1093/bioinformatics/btn544</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2639303">2639303</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18940824">18940824</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=IntaRNA%3A+efficient+prediction+of+bacterial+sRNA+targets+incorporating+target+site+accessibility+and+seed+regions.&amp;rft.volume=24&amp;rft.issue=24&amp;rft.pages=2849-56&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2639303&amp;rft_id=info%3Apmid%2F18940824&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtn544&amp;rft.aulast=Busch&amp;rft.aufirst=A&amp;rft.au=Richter%2C+AS&amp;rft.au=Backofen%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2639303&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid19850757-101"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19850757_101-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFRichterSchlebergerBackofenSteglich2010" class="citation journal">Richter AS, Schleberger C, Backofen R, Steglich C (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2796815">"Seed-based INTARNA prediction combined with GFP-reporter system identifies mRNA targets of the small RNA Yfr1"</a>. <i>Bioinformatics</i>. <b>26</b> (1): 1–5. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp609">10.1093/bioinformatics/btp609</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2796815">2796815</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19850757">19850757</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Seed-based+INTARNA+prediction+combined+with+GFP-reporter+system+identifies+mRNA+targets+of+the+small+RNA+Yfr1.&amp;rft.volume=26&amp;rft.issue=1&amp;rft.pages=1-5&amp;rft.date=2010&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2796815&amp;rft_id=info%3Apmid%2F19850757&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtp609&amp;rft.aulast=Richter&amp;rft.aufirst=AS&amp;rft.au=Schleberger%2C+C&amp;rft.au=Backofen%2C+R&amp;rft.au=Steglich%2C+C&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2796815&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid20444875-102"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid20444875_102-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSmithHeyneRichterWill2010" class="citation journal">Smith C, Heyne S, Richter AS, Will S, Backofen R (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2896085">"Freiburg RNA Tools: a web server integrating INTARNA, EXPARNA and LOCARNA"</a>. <i>Nucleic Acids Res</i>. <b>38</b> (Web Server): W373–7. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkq316">10.1093/nar/gkq316</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2896085">2896085</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20444875">20444875</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=Freiburg+RNA+Tools%3A+a+web+server+integrating+INTARNA%2C+EXPARNA+and+LOCARNA.&amp;rft.volume=38&amp;rft.issue=Web+Server&amp;rft.pages=W373-7&amp;rft.date=2010&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2896085&amp;rft_id=info%3Apmid%2F20444875&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkq316&amp;rft.aulast=Smith&amp;rft.aufirst=C&amp;rft.au=Heyne%2C+S&amp;rft.au=Richter%2C+AS&amp;rft.au=Will%2C+S&amp;rft.au=Backofen%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2896085&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid23980183-103"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid23980183_103-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWrightRichterPapenfortMann2013" class="citation journal">Wright PR, Richter AS, Papenfort K, Mann M, Vogel J, Hess WR, Backofen R, Georg J (2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3773804">"Comparative genomics boosts target prediction for bacterial small RNAs"</a>. <i>Proc Natl Acad Sci U S A</i>. <b>110</b> (37): E3487–E3496. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2013PNAS..110E3487W">2013PNAS..110E3487W</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.1303248110">10.1073/pnas.1303248110</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3773804">3773804</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23980183">23980183</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proc+Natl+Acad+Sci+U+S+A&amp;rft.atitle=Comparative+genomics+boosts+target+prediction+for+bacterial+small+RNAs.&amp;rft.volume=110&amp;rft.issue=37&amp;rft.pages=E3487-E3496&amp;rft.date=2013&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3773804&amp;rft_id=info%3Apmid%2F23980183&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.1303248110&amp;rft_id=info%3Abibcode%2F2013PNAS..110E3487W&amp;rft.aulast=Wright&amp;rft.aufirst=PR&amp;rft.au=Richter%2C+AS&amp;rft.au=Papenfort%2C+K&amp;rft.au=Mann%2C+M&amp;rft.au=Vogel%2C+J&amp;rft.au=Hess%2C+WR&amp;rft.au=Backofen%2C+R&amp;rft.au=Georg%2C+J&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3773804&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-MINT-RNA-NAR-104"><span class="mw-cite-backlink"><b><a href="#cite_ref-MINT-RNA-NAR_104-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFGórskaJasińskiTrylska2015" class="citation journal">Górska A, Jasiński M, Trylska J (2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4787793">"MINT: software to identify motifs and short-range interactions in trajectories of nucleic acids"</a>. <i>Nucleic Acids Research</i>. <b>43</b> (17): e114. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkv559">10.1093/nar/gkv559</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4787793">4787793</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/26024667">26024667</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=MINT%3A+software+to+identify+motifs+and+short-range+interactions+in+trajectories+of+nucleic+acids.&amp;rft.volume=43&amp;rft.issue=17&amp;rft.pages=e114&amp;rft.date=2015&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4787793&amp;rft_id=info%3Apmid%2F26024667&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkv559&amp;rft.aulast=G%C3%B3rska&amp;rft.aufirst=A&amp;rft.au=Jasi%C5%84ski%2C+M&amp;rft.au=Trylska%2C+J&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4787793&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-NUPACK-105"><span class="mw-cite-backlink"><b><a href="#cite_ref-NUPACK_105-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFR.M._DirksJ.S._BoisJ.M._SchaefferE._Winfree2007" class="citation journal">R.M. Dirks; J.S. Bois; J.M. Schaeffer; E. Winfree; N.A. Pierce (2007). "Thermodynamic Analysis of Interacting Nucleic Acid Strands". <i>SIAM Review</i>. <b>49</b> (1): 65–88. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2007SIAMR..49...65D">2007SIAMR..49...65D</a>. <a href="/wiki/CiteSeerX_(identifier)" class="mw-redirect" title="CiteSeerX (identifier)">CiteSeerX</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.523.4764">10.1.1.523.4764</a></span>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1137%2F060651100">10.1137/060651100</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=SIAM+Review&amp;rft.atitle=Thermodynamic+Analysis+of+Interacting+Nucleic+Acid+Strands&amp;rft.volume=49&amp;rft.issue=1&amp;rft.pages=65-88&amp;rft.date=2007&amp;rft_id=%2F%2Fciteseerx.ist.psu.edu%2Fviewdoc%2Fsummary%3Fdoi%3D10.1.1.523.4764&amp;rft_id=info%3Adoi%2F10.1137%2F060651100&amp;rft_id=info%3Abibcode%2F2007SIAMR..49...65D&amp;rft.au=R.M.+Dirks&amp;rft.au=J.S.+Bois&amp;rft.au=J.M.+Schaeffer&amp;rft.au=E.+Winfree&amp;rft.au=N.A.+Pierce&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-oligowalk-106"><span class="mw-cite-backlink"><b><a href="#cite_ref-oligowalk_106-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFD.H._MathewsM.E._BurkardS.M._FreierD.H._Turner1999" class="citation journal">D.H. Mathews; M.E. Burkard; S.M. Freier; D.H. Turner (1999). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1369867">"Predicting Oligonucleotide Affinity to RNA Targets"</a>. <i>RNA</i>. <b>5</b> (11): 1458–1469. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1017%2FS1355838299991148">10.1017/S1355838299991148</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1369867">1369867</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/10580474">10580474</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=RNA&amp;rft.atitle=Predicting+Oligonucleotide+Affinity+to+RNA+Targets.&amp;rft.volume=5&amp;rft.issue=11&amp;rft.pages=1458-1469&amp;rft.date=1999&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1369867&amp;rft_id=info%3Apmid%2F10580474&amp;rft_id=info%3Adoi%2F10.1017%2FS1355838299991148&amp;rft.au=D.H.+Mathews&amp;rft.au=M.E.+Burkard&amp;rft.au=S.M.+Freier&amp;rft.au=D.H.+Turner&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1369867&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-piRNA-107"><span class="mw-cite-backlink"><b><a href="#cite_ref-piRNA_107-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFH._ChitsazR._SalariS.C._SahinalpR._Backofen2009" class="citation journal">H. Chitsaz; R. Salari; S.C. Sahinalp; R. Backofen (2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2687966">"A Partition Function Algorithm for Interacting Nucleic Acid Strands"</a>. <i>Bioinformatics</i>. <b>25</b> (12): i365–i373. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp212">10.1093/bioinformatics/btp212</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2687966">2687966</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19478011">19478011</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=A+Partition+Function+Algorithm+for+Interacting+Nucleic+Acid+Strands.&amp;rft.volume=25&amp;rft.issue=12&amp;rft.pages=i365-i373&amp;rft.date=2009&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687966&amp;rft_id=info%3Apmid%2F19478011&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtp212&amp;rft.au=H.+Chitsaz&amp;rft.au=R.+Salari&amp;rft.au=S.C.+Sahinalp&amp;rft.au=R.+Backofen&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687966&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNAripalign-108"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNAripalign_108-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFAndrew_Xiang_LiJing_QinManja_MarzChristian_M._Reidys2011" class="citation journal">Andrew Xiang Li; Jing Qin; Manja Marz; Christian M. Reidys (2011). "RNA–RNA interaction prediction based on multiple sequence alignments". <i>Bioinformatics</i>. <b>27</b> (4): 456–463. <a href="/wiki/ArXiv_(identifier)" class="mw-redirect" title="ArXiv (identifier)">arXiv</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//arxiv.org/abs/1003.3987">1003.3987</a></span>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq659">10.1093/bioinformatics/btq659</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21134894">21134894</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RNA%E2%80%93RNA+interaction+prediction+based+on+multiple+sequence+alignments.&amp;rft.volume=27&amp;rft.issue=4&amp;rft.pages=456-463&amp;rft.date=2011&amp;rft_id=info%3Aarxiv%2F1003.3987&amp;rft_id=info%3Apmid%2F21134894&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtq659&amp;rft.au=Andrew+Xiang+Li&amp;rft.au=Jing+Qin&amp;rft.au=Manja+Marz&amp;rft.au=Christian+M.+Reidys&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid20823308-109"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid20823308_109-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFKatoSatoHamadaWatanabe2010" class="citation journal">Kato Y, Sato K, Hamada M, Watanabe Y, Asai K, Akutsu T (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2935440">"RactIP: fast and accurate prediction of RNA-RNA interaction using integer programming"</a>. <i>Bioinformatics</i>. <b>26</b> (18): i460-6. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq372">10.1093/bioinformatics/btq372</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2935440">2935440</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20823308">20823308</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RactIP%3A+fast+and+accurate+prediction+of+RNA-RNA+interaction+using+integer+programming&amp;rft.volume=26&amp;rft.issue=18&amp;rft.pages=i460-6&amp;rft.date=2010&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2935440&amp;rft_id=info%3Apmid%2F20823308&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtq372&amp;rft.aulast=Kato&amp;rft.aufirst=Y&amp;rft.au=Sato%2C+K&amp;rft.au=Hamada%2C+M&amp;rft.au=Watanabe%2C+Y&amp;rft.au=Asai%2C+K&amp;rft.au=Akutsu%2C+T&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2935440&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16722605-110"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16722605_110-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFBernhartTaferMücksteinFlamm2006" class="citation journal">Bernhart SH, Tafer H, Mückstein U, Flamm C, Stadler PF, Hofacker IL (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1459172">"Partition function and base pairing probabilities of RNA heterodimers"</a>. <i>Algorithms Mol Biol</i>. <b>1</b> (1): 3. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1748-7188-1-3">10.1186/1748-7188-1-3</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1459172">1459172</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16722605">16722605</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Algorithms+Mol+Biol&amp;rft.atitle=Partition+function+and+base+pairing+probabilities+of+RNA+heterodimers&amp;rft.volume=1&amp;rft.issue=1&amp;rft.pages=3&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1459172&amp;rft_id=info%3Apmid%2F16722605&amp;rft_id=info%3Adoi%2F10.1186%2F1748-7188-1-3&amp;rft.aulast=Bernhart&amp;rft.aufirst=SH&amp;rft.au=Tafer%2C+H&amp;rft.au=M%C3%BCckstein%2C+U&amp;rft.au=Flamm%2C+C&amp;rft.au=Stadler%2C+PF&amp;rft.au=Hofacker%2C+IL&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1459172&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15383676-111"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid15383676_111-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid15383676_111-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-pmid15383676_111-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFRehmsmeierSteffenHochsmannGiegerich2004" class="citation journal">Rehmsmeier M, Steffen P, Hochsmann M, Giegerich R (2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1370637">"Fast and effective prediction of microRNA/target duplexes"</a>. <i>RNA</i>. <b>10</b> (10): 1507–17. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1261%2Frna.5248604">10.1261/rna.5248604</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1370637">1370637</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15383676">15383676</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=RNA&amp;rft.atitle=Fast+and+effective+prediction+of+microRNA%2Ftarget+duplexes&amp;rft.volume=10&amp;rft.issue=10&amp;rft.pages=1507-17&amp;rft.date=2004&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1370637&amp;rft_id=info%3Apmid%2F15383676&amp;rft_id=info%3Adoi%2F10.1261%2Frna.5248604&amp;rft.aulast=Rehmsmeier&amp;rft.aufirst=M&amp;rft.au=Steffen%2C+P&amp;rft.au=Hochsmann%2C+M&amp;rft.au=Giegerich%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1370637&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16845047-112"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid16845047_112-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid16845047_112-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-pmid16845047_112-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFKrügerRehmsmeier2006" class="citation journal">Krüger J, Rehmsmeier M (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1538877">"RNAhybrid: microRNA target prediction easy, fast and flexible"</a>. <i>Nucleic Acids Res</i>. <b>34</b> (Web Server issue): W451–4. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkl243">10.1093/nar/gkl243</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1538877">1538877</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16845047">16845047</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=RNAhybrid%3A+microRNA+target+prediction+easy%2C+fast+and+flexible&amp;rft.volume=34&amp;rft.issue=Web+Server+issue&amp;rft.pages=W451-4&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1538877&amp;rft_id=info%3Apmid%2F16845047&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkl243&amp;rft.aulast=Kr%C3%BCger&amp;rft.aufirst=J&amp;rft.au=Rehmsmeier%2C+M&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1538877&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16446276-113"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16446276_113-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMücksteinTaferHackermüllerBernhart2006" class="citation journal">Mückstein U, Tafer H, Hackermüller J, Bernhart SH, Stadler PF, Hofacker IL (2006). "Thermodynamics of RNA-RNA binding". <i>Bioinformatics</i>. <b>22</b> (10): 1177–82. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl024">10.1093/bioinformatics/btl024</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16446276">16446276</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Thermodynamics+of+RNA-RNA+binding&amp;rft.volume=22&amp;rft.issue=10&amp;rft.pages=1177-82&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtl024&amp;rft_id=info%3Apmid%2F16446276&amp;rft.aulast=M%C3%BCckstein&amp;rft.aufirst=U&amp;rft.au=Tafer%2C+H&amp;rft.au=Hackerm%C3%BCller%2C+J&amp;rft.au=Bernhart%2C+SH&amp;rft.au=Stadler%2C+PF&amp;rft.au=Hofacker%2C+IL&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-114"><span class="mw-cite-backlink"><b><a href="#cite_ref-114">^</a></b></span> <span class="reference-text"><cite id="CITEREFChorosteckiPalatnik2014" class="citation journal">Chorostecki U, Palatnik JF (July 2014). "comTAR: a web tool for the prediction and characterization of conserved microRNA targets in plants". <i>Bioinformatics</i>. <b>30</b> (14): 2066–7. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu147">10.1093/bioinformatics/btu147</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24632500">24632500</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=comTAR%3A+a+web+tool+for+the+prediction+and+characterization+of+conserved+microRNA+targets+in+plants&amp;rft.volume=30&amp;rft.issue=14&amp;rft.pages=2066-7&amp;rft.date=2014-07&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtu147&amp;rft_id=info%3Apmid%2F24632500&amp;rft.aulast=Chorostecki&amp;rft.aufirst=U&amp;rft.au=Palatnik%2C+JF&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16990141-115"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid16990141_115-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid16990141_115-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFMirandaHuynhTayAng2006" class="citation journal">Miranda KC, Huynh T, Tay Y, Ang YS, Tam WL, Thomson AM, Lim B, Rigoutsos I (2006). "A pattern-based method for the identification of MicroRNA binding sites and their corresponding heteroduplexes". <i>Cell</i>. <b>126</b> (6): 1203–17. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.cell.2006.07.031">10.1016/j.cell.2006.07.031</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16990141">16990141</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Cell&amp;rft.atitle=A+pattern-based+method+for+the+identification+of+MicroRNA+binding+sites+and+their+corresponding+heteroduplexes.&amp;rft.volume=126&amp;rft.issue=6&amp;rft.pages=1203-17&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1016%2Fj.cell.2006.07.031&amp;rft_id=info%3Apmid%2F16990141&amp;rft.aulast=Miranda&amp;rft.aufirst=KC&amp;rft.au=Huynh%2C+T&amp;rft.au=Tay%2C+Y&amp;rft.au=Ang%2C+YS&amp;rft.au=Tam%2C+WL&amp;rft.au=Thomson%2C+AM&amp;rft.au=Lim%2C+B&amp;rft.au=Rigoutsos%2C+I&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-116"><span class="mw-cite-backlink"><b><a href="#cite_ref-116">^</a></b></span> <span class="reference-text"><cite id="CITEREFWeillLisiScottDallaire2015" class="citation journal">Weill N, Lisi V, Scott N, Dallaire P, Pelloux J, Major F (August 2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4538818">"MiRBooking simulates the stoichiometric mode of action of microRNAs"</a>. <i>Nucleic Acids Research</i>. <b>43</b> (14): 6730–8. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkv619">10.1093/nar/gkv619</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4538818">4538818</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/26089388">26089388</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=MiRBooking+simulates+the+stoichiometric+mode+of+action+of+microRNAs&amp;rft.volume=43&amp;rft.issue=14&amp;rft.pages=6730-8&amp;rft.date=2015-08&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4538818&amp;rft_id=info%3Apmid%2F26089388&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkv619&amp;rft.aulast=Weill&amp;rft.aufirst=N&amp;rft.au=Lisi%2C+V&amp;rft.au=Scott%2C+N&amp;rft.au=Dallaire%2C+P&amp;rft.au=Pelloux%2C+J&amp;rft.au=Major%2C+F&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4538818&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18668037-117"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18668037_117-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFBaekVillénShinCamargo2008" class="citation journal">Baek D, Villén J, Shin C, Camargo FD, Gygi SP, Bartel DP (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2745094">"The impact of microRNAs on protein output"</a>. <i>Nature</i>. <b>455</b> (7209): 64–71. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2008Natur.455...64B">2008Natur.455...64B</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnature07242">10.1038/nature07242</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2745094">2745094</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18668037">18668037</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=The+impact+of+microRNAs+on+protein+output.&amp;rft.volume=455&amp;rft.issue=7209&amp;rft.pages=64-71&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2745094&amp;rft_id=info%3Apmid%2F18668037&amp;rft_id=info%3Adoi%2F10.1038%2Fnature07242&amp;rft_id=info%3Abibcode%2F2008Natur.455...64B&amp;rft.aulast=Baek&amp;rft.aufirst=D&amp;rft.au=Vill%C3%A9n%2C+J&amp;rft.au=Shin%2C+C&amp;rft.au=Camargo%2C+FD&amp;rft.au=Gygi%2C+SP&amp;rft.au=Bartel%2C+DP&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2745094&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid19789267-118"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19789267_118-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFAlexiouMaragkakisPapadopoulosReczko2009" class="citation journal">Alexiou P, Maragkakis M, Papadopoulos GL, Reczko M, Hatzigeorgiou AG (2009). "Lost in translation: an assessment and perspective for computational microRNA target identification". <i>Bioinformatics</i>. <b>25</b> (23): 3049–55. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp565">10.1093/bioinformatics/btp565</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19789267">19789267</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Lost+in+translation%3A+an+assessment+and+perspective+for+computational+microRNA+target+identification.&amp;rft.volume=25&amp;rft.issue=23&amp;rft.pages=3049-55&amp;rft.date=2009&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtp565&amp;rft_id=info%3Apmid%2F19789267&amp;rft.aulast=Alexiou&amp;rft.aufirst=P&amp;rft.au=Maragkakis%2C+M&amp;rft.au=Papadopoulos%2C+GL&amp;rft.au=Reczko%2C+M&amp;rft.au=Hatzigeorgiou%2C+AG&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid19478799-119"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19478799_119-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFRitchieFlamantRasko2009" class="citation journal">Ritchie W, Flamant S, Rasko JE (2009). "Predicting microRNA targets and functions: traps for the unwary". <i>Nature Methods</i>. <b>6</b> (6): 3978–398. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnmeth0609-397">10.1038/nmeth0609-397</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19478799">19478799</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Methods&amp;rft.atitle=Predicting+microRNA+targets+and+functions%3A+traps+for+the+unwary&amp;rft.volume=6&amp;rft.issue=6&amp;rft.pages=3978-398&amp;rft.date=2009&amp;rft_id=info%3Adoi%2F10.1038%2Fnmeth0609-397&amp;rft_id=info%3Apmid%2F19478799&amp;rft.aulast=Ritchie&amp;rft.aufirst=W&amp;rft.au=Flamant%2C+S&amp;rft.au=Rasko%2C+JE&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-Chiu_et._al,_Genome_Res_2015-120"><span class="mw-cite-backlink"><b><a href="#cite_ref-Chiu_et._al,_Genome_Res_2015_120-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFChiuLlobet-NavasYangChung2015" class="citation journal">Chiu HS, Llobet-Navas D, Yang X, Chung WJ, Ambesi-Impiombato A, Iyer A, Kim HR, Seviour EG, Luo Z, Sehgal V, Moss T, Lu Y, Ram P, Silva J, Mills GB, Califano A, Sumazin P (February 2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4315299">"Cupid: simultaneous reconstruction of microRNA-target and ceRNA networks"</a>. <i>Genome Research</i>. <b>25</b> (2): 257–67. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1101%2Fgr.178194.114">10.1101/gr.178194.114</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4315299">4315299</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/25378249">25378249</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Genome+Research&amp;rft.atitle=Cupid%3A+simultaneous+reconstruction+of+microRNA-target+and+ceRNA+networks&amp;rft.volume=25&amp;rft.issue=2&amp;rft.pages=257-67&amp;rft.date=2015-02&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4315299&amp;rft_id=info%3Apmid%2F25378249&amp;rft_id=info%3Adoi%2F10.1101%2Fgr.178194.114&amp;rft.aulast=Chiu&amp;rft.aufirst=HS&amp;rft.au=Llobet-Navas%2C+D&amp;rft.au=Yang%2C+X&amp;rft.au=Chung%2C+WJ&amp;rft.au=Ambesi-Impiombato%2C+A&amp;rft.au=Iyer%2C+A&amp;rft.au=Kim%2C+HR&amp;rft.au=Seviour%2C+EG&amp;rft.au=Luo%2C+Z&amp;rft.au=Sehgal%2C+V&amp;rft.au=Moss%2C+T&amp;rft.au=Lu%2C+Y&amp;rft.au=Ram%2C+P&amp;rft.au=Silva%2C+J&amp;rft.au=Mills%2C+GB&amp;rft.au=Califano%2C+A&amp;rft.au=Sumazin%2C+P&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4315299&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid19765283-121"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19765283_121-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMaragkakisAlexiouPapadopoulosReczko2009" class="citation journal">Maragkakis M, Alexiou P, Papadopoulos GL, Reczko M, Dalamagas T, Giannopoulos G, Goumas G, Koukis E, Kourtis K, Simossis VA, Sethupathy P, Vergoulis T, Koziris N, Sellis T, Tsanakas P, Hatzigeorgiou AG (2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2752464">"Accurate microRNA target prediction correlates with protein repression levels"</a>. <i>BMC Bioinformatics</i>. <b>10</b> (1): 295. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-10-295">10.1186/1471-2105-10-295</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2752464">2752464</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19765283">19765283</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Accurate+microRNA+target+prediction+correlates+with+protein+repression+levels.&amp;rft.volume=10&amp;rft.issue=1&amp;rft.pages=295&amp;rft.date=2009&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2752464&amp;rft_id=info%3Apmid%2F19765283&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-10-295&amp;rft.aulast=Maragkakis&amp;rft.aufirst=M&amp;rft.au=Alexiou%2C+P&amp;rft.au=Papadopoulos%2C+GL&amp;rft.au=Reczko%2C+M&amp;rft.au=Dalamagas%2C+T&amp;rft.au=Giannopoulos%2C+G&amp;rft.au=Goumas%2C+G&amp;rft.au=Koukis%2C+E&amp;rft.au=Kourtis%2C+K&amp;rft.au=Simossis%2C+VA&amp;rft.au=Sethupathy%2C+P&amp;rft.au=Vergoulis%2C+T&amp;rft.au=Koziris%2C+N&amp;rft.au=Sellis%2C+T&amp;rft.au=Tsanakas%2C+P&amp;rft.au=Hatzigeorgiou%2C+AG&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2752464&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17254305-122"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17254305_122-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFThadaniTammi2006" class="citation journal">Thadani R, Tammi MT (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1764477">"MicroTar: predicting microRNA targets from RNA duplexes"</a>. <i>BMC Bioinformatics</i>. 7. <b>7</b> (Suppl 5): S20. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-7-S5-S20">10.1186/1471-2105-7-S5-S20</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1764477">1764477</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17254305">17254305</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=MicroTar%3A+predicting+microRNA+targets+from+RNA+duplexes.&amp;rft.volume=7&amp;rft.issue=Suppl+5&amp;rft.pages=S20&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1764477&amp;rft_id=info%3Apmid%2F17254305&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-7-S5-S20&amp;rft.aulast=Thadani&amp;rft.aufirst=R&amp;rft.au=Tammi%2C+MT&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1764477&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16978421-123"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16978421_123-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFKimNamRheeLee2006" class="citation journal">Kim SK, Nam JW, Rhee JK, Lee WJ, Zhang BT (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1594580">"miTarget: microRNA target gene prediction using a support vector machine"</a>. <i>BMC Bioinformatics</i>. <b>7</b> (1): 411. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-7-411">10.1186/1471-2105-7-411</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1594580">1594580</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16978421">16978421</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=miTarget%3A+microRNA+target+gene+prediction+using+a+support+vector+machine.&amp;rft.volume=7&amp;rft.issue=1&amp;rft.pages=411&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1594580&amp;rft_id=info%3Apmid%2F16978421&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-7-411&amp;rft.aulast=Kim&amp;rft.aufirst=SK&amp;rft.au=Nam%2C+JW&amp;rft.au=Rhee%2C+JK&amp;rft.au=Lee%2C+WJ&amp;rft.au=Zhang%2C+BT&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1594580&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-124"><span class="mw-cite-backlink"><b><a href="#cite_ref-124">^</a></b></span> <span class="reference-text"><cite id="CITEREFFriedmanNaamatiLinial2010" class="citation journal">Friedman Y, Naamati G, Linial M (August 2010). "MiRror: a combinatorial analysis web tool for ensembles of microRNAs and their targets". <i>Bioinformatics</i>. <b>26</b> (15): 1920–1. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq298">10.1093/bioinformatics/btq298</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20529892">20529892</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=MiRror%3A+a+combinatorial+analysis+web+tool+for+ensembles+of+microRNAs+and+their+targets&amp;rft.volume=26&amp;rft.issue=15&amp;rft.pages=1920-1&amp;rft.date=2010-08&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtq298&amp;rft_id=info%3Apmid%2F20529892&amp;rft.aulast=Friedman&amp;rft.aufirst=Y&amp;rft.au=Naamati%2C+G&amp;rft.au=Linial%2C+M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-125"><span class="mw-cite-backlink"><b><a href="#cite_ref-125">^</a></b></span> <span class="reference-text"><cite id="CITEREFBalagaFriedmanLinial2012" class="citation journal">Balaga O, Friedman Y, Linial M (October 2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3479204">"Toward a combinatorial nature of microRNA regulation in human cells"</a>. <i>Nucleic Acids Research</i>. <b>40</b> (19): 9404–16. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgks759">10.1093/nar/gks759</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3479204">3479204</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22904063">22904063</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=Toward+a+combinatorial+nature+of+microRNA+regulation+in+human+cells&amp;rft.volume=40&amp;rft.issue=19&amp;rft.pages=9404-16&amp;rft.date=2012-10&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3479204&amp;rft_id=info%3Apmid%2F22904063&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgks759&amp;rft.aulast=Balaga&amp;rft.aufirst=O&amp;rft.au=Friedman%2C+Y&amp;rft.au=Linial%2C+M&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3479204&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15806104-126"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15806104_126-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFKrekGrünPoyWolf2005" class="citation journal">Krek A, Grün D, Poy MN, Wolf R, Rosenberg L, Epstein EJ, MacMenamin P, da Piedade I, Gunsalus KC, Stoffel M, Rajewsky N (2005). "Combinatorial microRNA target predictions". <i>Nat Genet</i>. <b>37</b> (5): 495–500. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fng1536">10.1038/ng1536</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15806104">15806104</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nat+Genet&amp;rft.atitle=Combinatorial+microRNA+target+predictions.&amp;rft.volume=37&amp;rft.issue=5&amp;rft.pages=495-500&amp;rft.date=2005&amp;rft_id=info%3Adoi%2F10.1038%2Fng1536&amp;rft_id=info%3Apmid%2F15806104&amp;rft.aulast=Krek&amp;rft.aufirst=A&amp;rft.au=Gr%C3%BCn%2C+D&amp;rft.au=Poy%2C+MN&amp;rft.au=Wolf%2C+R&amp;rft.au=Rosenberg%2C+L&amp;rft.au=Epstein%2C+EJ&amp;rft.au=MacMenamin%2C+P&amp;rft.au=da+Piedade%2C+I&amp;rft.au=Gunsalus%2C+KC&amp;rft.au=Stoffel%2C+M&amp;rft.au=Rajewsky%2C+N&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17893677-127"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17893677_127-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFKerteszIovinoUnnerstallGaul2007" class="citation journal">Kertesz M, Iovino N, Unnerstall U, Gaul U, Segal E (2007). "The role of site accessibility in microRNA target recognition". <i>Nat Genet</i>. <b>39</b> (10): 1278–84. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fng2135">10.1038/ng2135</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17893677">17893677</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nat+Genet&amp;rft.atitle=The+role+of+site+accessibility+in+microRNA+target+recognition.&amp;rft.volume=39&amp;rft.issue=10&amp;rft.pages=1278-84&amp;rft.date=2007&amp;rft_id=info%3Adoi%2F10.1038%2Fng2135&amp;rft_id=info%3Apmid%2F17893677&amp;rft.aulast=Kertesz&amp;rft.aufirst=M&amp;rft.au=Iovino%2C+N&amp;rft.au=Unnerstall%2C+U&amp;rft.au=Gaul%2C+U&amp;rft.au=Segal%2C+E&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18978784-128"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18978784_128-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFvan_DongenAbreu-GoodgerEnright2008" class="citation journal">van Dongen S, Abreu-Goodger C, Enright AJ (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2635553">"Detecting microRNA binding and siRNA off-target effects from expression data"</a>. <i>Nat Methods</i>. <b>5</b> (12): 1023–5. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnmeth.1267">10.1038/nmeth.1267</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2635553">2635553</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18978784">18978784</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nat+Methods&amp;rft.atitle=Detecting+microRNA+binding+and+siRNA+off-target+effects+from+expression+data.&amp;rft.volume=5&amp;rft.issue=12&amp;rft.pages=1023-5&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2635553&amp;rft_id=info%3Apmid%2F18978784&amp;rft_id=info%3Adoi%2F10.1038%2Fnmeth.1267&amp;rft.aulast=van+Dongen&amp;rft.aufirst=S&amp;rft.au=Abreu-Goodger%2C+C&amp;rft.au=Enright%2C+AJ&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2635553&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid20871108-129"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid20871108_129-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFBartonicekEnright2010" class="citation journal">Bartonicek N, Enright AJ (2010). "SylArray: A web-server for automated detection of miRNA effects from expression data". <i>Bioinformatics</i>. <b>26</b> (22): 2900–1. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq545">10.1093/bioinformatics/btq545</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20871108">20871108</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=SylArray%3A+A+web-server+for+automated+detection+of+miRNA+effects+from+expression+data.&amp;rft.volume=26&amp;rft.issue=22&amp;rft.pages=2900-1&amp;rft.date=2010&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtq545&amp;rft_id=info%3Apmid%2F20871108&amp;rft.aulast=Bartonicek&amp;rft.aufirst=N&amp;rft.au=Enright%2C+AJ&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid=20413915-130"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid=20413915_130-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFR._HeikhamR._Shankar2010" class="citation journal">R. Heikham &amp; R. Shankar (2010). "Flanking region sequence information to refine microRNA target predictions". <i>Journal of Biosciences</i>. <b>35</b> (1): 105–18. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2Fs12038-010-0013-7">10.1007/s12038-010-0013-7</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20413915">20413915</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Biosciences&amp;rft.atitle=Flanking+region+sequence+information+to+refine+microRNA+target+predictions.&amp;rft.volume=35&amp;rft.issue=1&amp;rft.pages=105-18&amp;rft.date=2010&amp;rft_id=info%3Adoi%2F10.1007%2Fs12038-010-0013-7&amp;rft_id=info%3Apmid%2F20413915&amp;rft.au=R.+Heikham&amp;rft.au=R.+Shankar&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid14697198-131"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid14697198_131-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLewisShihJones-RhoadesBartel2003" class="citation journal">Lewis BP, Shih IH, Jones-Rhoades MW, Bartel DP, Burge CB (December 2003). "Prediction of mammalian microRNA targets". <i>Cell</i>. <b>115</b> (7): 787–98. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1016%2FS0092-8674%2803%2901018-3">10.1016/S0092-8674(03)01018-3</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/14697198">14697198</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Cell&amp;rft.atitle=Prediction+of+mammalian+microRNA+targets&amp;rft.volume=115&amp;rft.issue=7&amp;rft.pages=787-98&amp;rft.date=2003-12&amp;rft_id=info%3Adoi%2F10.1016%2FS0092-8674%2803%2901018-3&amp;rft_id=info%3Apmid%2F14697198&amp;rft.aulast=Lewis&amp;rft.aufirst=BP&amp;rft.au=Shih%2C+IH&amp;rft.au=Jones-Rhoades%2C+MW&amp;rft.au=Bartel%2C+DP&amp;rft.au=Burge%2C+CB&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15652477-132"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15652477_132-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLewisBurgeBartel2005" class="citation journal">Lewis BP, Burge CB, Bartel DP (January 2005). "Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets". <i>Cell</i>. <b>120</b> (1): 15–20. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.cell.2004.12.035">10.1016/j.cell.2004.12.035</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15652477">15652477</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Cell&amp;rft.atitle=Conserved+seed+pairing%2C+often+flanked+by+adenosines%2C+indicates+that+thousands+of+human+genes+are+microRNA+targets&amp;rft.volume=120&amp;rft.issue=1&amp;rft.pages=15-20&amp;rft.date=2005-01&amp;rft_id=info%3Adoi%2F10.1016%2Fj.cell.2004.12.035&amp;rft_id=info%3Apmid%2F15652477&amp;rft.aulast=Lewis&amp;rft.aufirst=BP&amp;rft.au=Burge%2C+CB&amp;rft.au=Bartel%2C+DP&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17612493-133"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17612493_133-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFGrimsonFarhJohnstonGarrett-Engele2007" class="citation journal">Grimson A, Farh KK, Johnston WK, Garrett-Engele P, Lim LP, Bartel DP (July 2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3800283">"MicroRNA targeting specificity in mammals: determinants beyond seed pairing"</a>. <i>Molecular Cell</i>. <b>27</b> (1): 91–105. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.molcel.2007.06.017">10.1016/j.molcel.2007.06.017</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3800283">3800283</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17612493">17612493</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Molecular+Cell&amp;rft.atitle=MicroRNA+targeting+specificity+in+mammals%3A+determinants+beyond+seed+pairing&amp;rft.volume=27&amp;rft.issue=1&amp;rft.pages=91-105&amp;rft.date=2007-07&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3800283&amp;rft_id=info%3Apmid%2F17612493&amp;rft_id=info%3Adoi%2F10.1016%2Fj.molcel.2007.06.017&amp;rft.aulast=Grimson&amp;rft.aufirst=A&amp;rft.au=Farh%2C+KK&amp;rft.au=Johnston%2C+WK&amp;rft.au=Garrett-Engele%2C+P&amp;rft.au=Lim%2C+LP&amp;rft.au=Bartel%2C+DP&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3800283&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid21909094-134"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid21909094_134-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFGarciaBaekShinBell2011" class="citation journal">Garcia DM, Baek D, Shin C, Bell GW, Grimson A, Bartel DP (September 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3190056">"Weak seed-pairing stability and high target-site abundance decrease the proficiency of lsy-6 and other microRNAs"</a>. <i>Nature Structural &amp; Molecular Biology</i>. <b>18</b> (10): 1139–46. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnsmb.2115">10.1038/nsmb.2115</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3190056">3190056</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21909094">21909094</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Structural+%26+Molecular+Biology&amp;rft.atitle=Weak+seed-pairing+stability+and+high+target-site+abundance+decrease+the+proficiency+of+lsy-6+and+other+microRNAs&amp;rft.volume=18&amp;rft.issue=10&amp;rft.pages=1139-46&amp;rft.date=2011-09&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3190056&amp;rft_id=info%3Apmid%2F21909094&amp;rft_id=info%3Adoi%2F10.1038%2Fnsmb.2115&amp;rft.aulast=Garcia&amp;rft.aufirst=DM&amp;rft.au=Baek%2C+D&amp;rft.au=Shin%2C+C&amp;rft.au=Bell%2C+GW&amp;rft.au=Grimson%2C+A&amp;rft.au=Bartel%2C+DP&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3190056&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-135"><span class="mw-cite-backlink"><b><a href="#cite_ref-135">^</a></b></span> <span class="reference-text"><cite id="CITEREFAgarwalBellNamBartel2015" class="citation journal">Agarwal V, Bell GW, Nam JW, Bartel DP (August 2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4532895">"Predicting effective microRNA target sites in mammalian mRNAs"</a>. <i>eLife</i>. <b>4</b>: e05005. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.7554%2FeLife.05005">10.7554/eLife.05005</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4532895">4532895</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/26267216">26267216</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=eLife&amp;rft.atitle=Predicting+effective+microRNA+target+sites+in+mammalian+mRNAs&amp;rft.volume=4&amp;rft.pages=e05005&amp;rft.date=2015-08&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4532895&amp;rft_id=info%3Apmid%2F26267216&amp;rft_id=info%3Adoi%2F10.7554%2FeLife.05005&amp;rft.aulast=Agarwal&amp;rft.aufirst=V&amp;rft.au=Bell%2C+GW&amp;rft.au=Nam%2C+JW&amp;rft.au=Bartel%2C+DP&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4532895&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-136"><span class="mw-cite-backlink"><b><a href="#cite_ref-136">^</a></b></span> <span class="reference-text"><cite id="CITEREFAgarwalSubtelnyThiruUlitsky2018" class="citation journal">Agarwal, V; Subtelny, AO; Thiru, P; Ulitsky, I; Bartel, DP (4 October 2018). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC6172730">"Predicting microRNA targeting efficacy in Drosophila"</a>. <i>Genome Biology</i>. <b>19</b> (1): 152. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2Fs13059-018-1504-3">10.1186/s13059-018-1504-3</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC6172730">6172730</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/30286781">30286781</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Genome+Biology&amp;rft.atitle=Predicting+microRNA+targeting+efficacy+in+Drosophila.&amp;rft.volume=19&amp;rft.issue=1&amp;rft.pages=152&amp;rft.date=2018-10-04&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6172730&amp;rft_id=info%3Apmid%2F30286781&amp;rft_id=info%3Adoi%2F10.1186%2Fs13059-018-1504-3&amp;rft.aulast=Agarwal&amp;rft.aufirst=V&amp;rft.au=Subtelny%2C+AO&amp;rft.au=Thiru%2C+P&amp;rft.au=Ulitsky%2C+I&amp;rft.au=Bartel%2C+DP&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6172730&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15313604-140"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15313604_140-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWashietlHofacker2004" class="citation journal">Washietl S, Hofacker IL (2004). "Consensus folding of aligned sequences as a new measure for the detection of functional RNAs by comparative genomics". <i>J. Mol. Biol</i>. <b>342</b> (1): 19–30. <a href="/wiki/CiteSeerX_(identifier)" class="mw-redirect" title="CiteSeerX (identifier)">CiteSeerX</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.58.6251">10.1.1.58.6251</a></span>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.jmb.2004.07.018">10.1016/j.jmb.2004.07.018</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15313604">15313604</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J.+Mol.+Biol.&amp;rft.atitle=Consensus+folding+of+aligned+sequences+as+a+new+measure+for+the+detection+of+functional+RNAs+by+comparative+genomics&amp;rft.volume=342&amp;rft.issue=1&amp;rft.pages=19-30&amp;rft.date=2004&amp;rft_id=%2F%2Fciteseerx.ist.psu.edu%2Fviewdoc%2Fsummary%3Fdoi%3D10.1.1.58.6251&amp;rft_id=info%3Apmid%2F15313604&amp;rft_id=info%3Adoi%2F10.1016%2Fj.jmb.2004.07.018&amp;rft.aulast=Washietl&amp;rft.aufirst=S&amp;rft.au=Hofacker%2C+IL&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16628248-141"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16628248_141-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFPedersenBejeranoSiepel2006" class="citation journal">Pedersen JS, Bejerano G, Siepel A,  et al. (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1440920">"Identification and classification of conserved RNA secondary structures in the human genome"</a>. <i>PLOS Comput. Biol</i>. <b>2</b> (4): e33. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2006PLSCB...2...33P">2006PLSCB...2...33P</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pcbi.0020033">10.1371/journal.pcbi.0020033</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1440920">1440920</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16628248">16628248</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+Comput.+Biol.&amp;rft.atitle=Identification+and+classification+of+conserved+RNA+secondary+structures+in+the+human+genome&amp;rft.volume=2&amp;rft.issue=4&amp;rft.pages=e33&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1440920&amp;rft_id=info%3Apmid%2F16628248&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.0020033&amp;rft_id=info%3Abibcode%2F2006PLSCB...2...33P&amp;rft.aulast=Pedersen&amp;rft.aufirst=JS&amp;rft.au=Bejerano%2C+G&amp;rft.au=Siepel%2C+A&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1440920&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-142"><span class="mw-cite-backlink"><b><a href="#cite_ref-142">^</a></b></span> <span class="reference-text"><cite id="CITEREFCoventry_A,_Kleitman_DJ,_Berger_BA2004" class="citation journal">Coventry A, Kleitman DJ, <a href="/wiki/Bonnie_Berger" title="Bonnie Berger">Berger BA</a> (2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC514400">"MSARI: Multiple sequence alignments for statistical detection of RNA secondary structure"</a>. <i>PNAS</i>. <b>101</b> (33): 12102–12107. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2004PNAS..10112102C">2004PNAS..10112102C</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.0404193101">10.1073/pnas.0404193101</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC514400">514400</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15304649">15304649</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PNAS&amp;rft.atitle=MSARI%3A+Multiple+sequence+alignments+for+statistical+detection+of+RNA+secondary+structure&amp;rft.volume=101&amp;rft.issue=33&amp;rft.pages=12102-12107&amp;rft.date=2004&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC514400&amp;rft_id=info%3Apmid%2F15304649&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0404193101&amp;rft_id=info%3Abibcode%2F2004PNAS..10112102C&amp;rft.au=Coventry+A%2C+Kleitman+DJ%2C+Berger+BA&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC514400&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><span class="cs1-maint citation-comment">CS1 maint: multiple names: authors list (<a href="/wiki/Category:CS1_maint:_multiple_names:_authors_list" title="Category:CS1 maint: multiple names: authors list">link</a>)</span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid11801179-143"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid11801179_143-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFRivasEddy2001" class="citation journal">Rivas E, Eddy SR (2001). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC64605">"Noncoding RNA gene detection using comparative sequence analysis"</a>. <i>BMC Bioinformatics</i>. <b>2</b> (1): 8. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-2-8">10.1186/1471-2105-2-8</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC64605">64605</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/11801179">11801179</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Noncoding+RNA+gene+detection+using+comparative+sequence+analysis&amp;rft.volume=2&amp;rft.issue=1&amp;rft.pages=8&amp;rft.date=2001&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC64605&amp;rft_id=info%3Apmid%2F11801179&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-2-8&amp;rft.aulast=Rivas&amp;rft.aufirst=E&amp;rft.au=Eddy%2C+SR&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC64605&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid11553332-144"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid11553332_144-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFRivasKleinJonesEddy2001" class="citation journal">Rivas E, Klein RJ, Jones TA, Eddy SR (2001). "Computational identification of noncoding RNAs in E. coli by comparative genomics". <i>Curr. Biol</i>. <b>11</b> (17): 1369–73. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1016%2FS0960-9822%2801%2900401-8">10.1016/S0960-9822(01)00401-8</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/11553332">11553332</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Curr.+Biol.&amp;rft.atitle=Computational+identification+of+noncoding+RNAs+in+E.+coli+by+comparative+genomics&amp;rft.volume=11&amp;rft.issue=17&amp;rft.pages=1369-73&amp;rft.date=2001&amp;rft_id=info%3Adoi%2F10.1016%2FS0960-9822%2801%2900401-8&amp;rft_id=info%3Apmid%2F11553332&amp;rft.aulast=Rivas&amp;rft.aufirst=E&amp;rft.au=Klein%2C+RJ&amp;rft.au=Jones%2C+TA&amp;rft.au=Eddy%2C+SR&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15665081-145"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15665081_145-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWashietlHofackerStadler2005" class="citation journal">Washietl S, Hofacker IL, Stadler PF (2005). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC548974">"Fast and reliable prediction of noncoding RNAs"</a>. <i>Proc. Natl. Acad. Sci. U.S.A</i>. <b>102</b> (7): 2454–9. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2005PNAS..102.2454W">2005PNAS..102.2454W</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.0409169102">10.1073/pnas.0409169102</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC548974">548974</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15665081">15665081</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;rft.atitle=Fast+and+reliable+prediction+of+noncoding+RNAs&amp;rft.volume=102&amp;rft.issue=7&amp;rft.pages=2454-9&amp;rft.date=2005&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC548974&amp;rft_id=info%3Apmid%2F15665081&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0409169102&amp;rft_id=info%3Abibcode%2F2005PNAS..102.2454W&amp;rft.aulast=Washietl&amp;rft.aufirst=S&amp;rft.au=Hofacker%2C+IL&amp;rft.au=Stadler%2C+PF&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC548974&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17452347-146"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17452347_146-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFGruberNeuböckHofackerWashietl2007" class="citation journal">Gruber AR, Neuböck R, Hofacker IL, Washietl S (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1933143">"The RNAz web server: prediction of thermodynamically stable and evolutionarily conserved RNA structures"</a>. <i>Nucleic Acids Res</i>. <b>35</b> (Web Server issue): W335–8. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkm222">10.1093/nar/gkm222</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1933143">1933143</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17452347">17452347</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=The+RNAz+web+server%3A+prediction+of+thermodynamically+stable+and+evolutionarily+conserved+RNA+structures&amp;rft.volume=35&amp;rft.issue=Web+Server+issue&amp;rft.pages=W335-8&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1933143&amp;rft_id=info%3Apmid%2F17452347&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkm222&amp;rft.aulast=Gruber&amp;rft.aufirst=AR&amp;rft.au=Neub%C3%B6ck%2C+R&amp;rft.au=Hofacker%2C+IL&amp;rft.au=Washietl%2C+S&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1933143&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17993695-147"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17993695_147-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWashietl_S2007" class="citation journal">Washietl S (2007). "Prediction of Structural Noncoding RNAs With RNAz". <i>Methods Mol. Biol</i>. Methods in Molecular Biology. <b>395</b>: 503–26. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2F978-1-59745-514-5_32">10.1007/978-1-59745-514-5_32</a>. <a href="/wiki/ISBN_(identifier)" class="mw-redirect" title="ISBN (identifier)">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-1-58829-693-1" title="Special:BookSources/978-1-58829-693-1"><bdi>978-1-58829-693-1</bdi></a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17993695">17993695</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Methods+Mol.+Biol.&amp;rft.atitle=Prediction+of+Structural+Noncoding+RNAs+With+RNAz&amp;rft.volume=395&amp;rft.pages=503-26&amp;rft.date=2007&amp;rft_id=info%3Apmid%2F17993695&amp;rft_id=info%3Adoi%2F10.1007%2F978-1-59745-514-5_32&amp;rft.isbn=978-1-58829-693-1&amp;rft.au=Washietl+S&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-148"><span class="mw-cite-backlink"><b><a href="#cite_ref-148">^</a></b></span> <span class="reference-text"><cite id="CITEREFAndrewsRocheMoss2018" class="citation journal">Andrews RJ, Roche J, Moss WN (2018). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC6317755">"ScanFold: an approach for genome-wide discovery of local RNA structural elements—applications to Zika virus and HIV"</a>. <i>PeerJ</i>. <b>6</b>: e6136. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.7717%2Fpeerj.6136">10.7717/peerj.6136</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC6317755">6317755</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/30627482">30627482</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PeerJ&amp;rft.atitle=ScanFold%3A+an+approach+for+genome-wide+discovery+of+local+RNA+structural+elements%E2%80%94applications+to+Zika+virus+and+HIV&amp;rft.volume=6&amp;rft.pages=e6136&amp;rft.date=2018&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6317755&amp;rft_id=info%3Apmid%2F30627482&amp;rft_id=info%3Adoi%2F10.7717%2Fpeerj.6136&amp;rft.aulast=Andrews&amp;rft.aufirst=RJ&amp;rft.au=Roche%2C+J&amp;rft.au=Moss%2C+WN&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6317755&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid14704338-149"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid14704338_149-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLaslettCanback2004" class="citation journal">Laslett D, Canback B (2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC373265">"ARAGORN, a program to detect tRNA genes and tmRNA genes in nucleotide sequences"</a>. <i>Nucleic Acids Res</i>. <b>32</b> (1): 11–6. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkh152">10.1093/nar/gkh152</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC373265">373265</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/14704338">14704338</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=ARAGORN%2C+a+program+to+detect+tRNA+genes+and+tmRNA+genes+in+nucleotide+sequences.&amp;rft.volume=32&amp;rft.issue=1&amp;rft.pages=11-6&amp;rft.date=2004&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC373265&amp;rft_id=info%3Apmid%2F14704338&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkh152&amp;rft.aulast=Laslett&amp;rft.aufirst=D&amp;rft.au=Canback%2C+B&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC373265&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid23805282-150"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid23805282_150-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFJhaShankar2013" class="citation journal">Jha A, Shankar R (2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3689854">"miReader: Discovering novel miRNAs in species without sequenced genome"</a>. <i>PLOS One</i>. <b>8</b> (6): e66857. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2013PLoSO...866857J">2013PLoSO...866857J</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pone.0066857">10.1371/journal.pone.0066857</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3689854">3689854</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23805282">23805282</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+One&amp;rft.atitle=miReader%3A+Discovering+novel+miRNAs+in+species+without+sequenced+genome.&amp;rft.volume=8&amp;rft.issue=6&amp;rft.pages=e66857&amp;rft.date=2013&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3689854&amp;rft_id=info%3Apmid%2F23805282&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pone.0066857&amp;rft_id=info%3Abibcode%2F2013PLoSO...866857J&amp;rft.aulast=Jha&amp;rft.aufirst=A&amp;rft.au=Shankar%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3689854&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18215311-151"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18215311_151-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFArtziKiezunShomron2008" class="citation journal">Artzi S, Kiezun A, Shomron N (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2258288">"miRNAminer: a tool for homologous microRNA gene search"</a>. <i>BMC Bioinformatics</i>. <b>9</b> (1): 39. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-9-39">10.1186/1471-2105-9-39</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2258288">2258288</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18215311">18215311</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=miRNAminer%3A+a+tool+for+homologous+microRNA+gene+search.&amp;rft.volume=9&amp;rft.issue=1&amp;rft.pages=39&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2258288&amp;rft_id=info%3Apmid%2F18215311&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-9-39&amp;rft.aulast=Artzi&amp;rft.aufirst=S&amp;rft.au=Kiezun%2C+A&amp;rft.au=Shomron%2C+N&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2258288&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-riscbinder-152"><span class="mw-cite-backlink"><b><a href="#cite_ref-riscbinder_152-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFAhmedAnsariRaghava2009" class="citation journal">Ahmed F, Ansari HR, Raghava GP (2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2676257">"Prediction of guide strand of microRNAs from its sequence and secondary structure"</a>. <i>BMC Bioinformatics</i>. <b>10</b> (1): 105. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-10-105">10.1186/1471-2105-10-105</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2676257">2676257</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19358699">19358699</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Prediction+of+guide+strand+of+microRNAs+from+its+sequence+and+secondary+structure&amp;rft.volume=10&amp;rft.issue=1&amp;rft.pages=105&amp;rft.date=2009&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2676257&amp;rft_id=info%3Apmid%2F19358699&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-10-105&amp;rft.aulast=Ahmed&amp;rft.aufirst=F&amp;rft.au=Ansari%2C+HR&amp;rft.au=Raghava%2C+GP&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2676257&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16873472-153"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16873472_153-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHertelStadler2006" class="citation journal">Hertel J, Stadler PF (2006). "Hairpins in a Haystack: recognizing microRNA precursors in comparative genomics data". <i>Bioinformatics</i>. <b>22</b> (14): e197–202. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl257">10.1093/bioinformatics/btl257</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16873472">16873472</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Hairpins+in+a+Haystack%3A+recognizing+microRNA+precursors+in+comparative+genomics+data.&amp;rft.volume=22&amp;rft.issue=14&amp;rft.pages=e197-202&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtl257&amp;rft_id=info%3Apmid%2F16873472&amp;rft.aulast=Hertel&amp;rft.aufirst=J&amp;rft.au=Stadler%2C+PF&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid14681368-154"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid14681368_154-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWuytsPerrièreVan_De_Peer2004" class="citation journal">Wuyts J, Perrière G, Van De Peer Y (2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC308799">"The European ribosomal RNA database"</a>. <i>Nucleic Acids Res</i>. <b>32</b> (Database issue): D101–3. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkh065">10.1093/nar/gkh065</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC308799">308799</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/14681368">14681368</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=The+European+ribosomal+RNA+database.&amp;rft.volume=32&amp;rft.issue=Database+issue&amp;rft.pages=D101-3&amp;rft.date=2004&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC308799&amp;rft_id=info%3Apmid%2F14681368&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkh065&amp;rft.aulast=Wuyts&amp;rft.aufirst=J&amp;rft.au=Perri%C3%A8re%2C+G&amp;rft.au=Van+De+Peer%2C+Y&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC308799&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid11752286-155"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid11752286_155-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSzymanskiBarciszewskaErdmannBarciszewski2002" class="citation journal">Szymanski M, Barciszewska MZ, Erdmann VA, Barciszewski J (2002). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC99124">"5S Ribosomal RNA Database"</a>. <i>Nucleic Acids Res</i>. <b>30</b> (1): 176–8. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F30.1.176">10.1093/nar/30.1.176</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC99124">99124</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/11752286">11752286</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=5S+Ribosomal+RNA+Database.&amp;rft.volume=30&amp;rft.issue=1&amp;rft.pages=176-8&amp;rft.date=2002&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC99124&amp;rft_id=info%3Apmid%2F11752286&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F30.1.176&amp;rft.aulast=Szymanski&amp;rft.aufirst=M&amp;rft.au=Barciszewska%2C+MZ&amp;rft.au=Erdmann%2C+VA&amp;rft.au=Barciszewski%2C+J&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC99124&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17452365-156"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17452365_156-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLagesenHallinRødlandStaerfeldt2007" class="citation journal">Lagesen K, Hallin P, Rødland EA, Staerfeldt HH, Rognes T, Ussery DW (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1888812">"RNAmmer: consistent and rapid annotation of ribosomal RNA genes"</a>. <i>Nucleic Acids Res</i>. <b>35</b> (9): 3100–8. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkm160">10.1093/nar/gkm160</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1888812">1888812</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17452365">17452365</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=RNAmmer%3A+consistent+and+rapid+annotation+of+ribosomal+RNA+genes.&amp;rft.volume=35&amp;rft.issue=9&amp;rft.pages=3100-8&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1888812&amp;rft_id=info%3Apmid%2F17452365&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkm160&amp;rft.aulast=Lagesen&amp;rft.aufirst=K&amp;rft.au=Hallin%2C+P&amp;rft.au=R%C3%B8dland%2C+EA&amp;rft.au=Staerfeldt%2C+HH&amp;rft.au=Rognes%2C+T&amp;rft.au=Ussery%2C+DW&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1888812&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17895272-157"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17895272_157-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHertelHofackerStadler2008" class="citation journal">Hertel J, Hofacker IL, Stadler PF (2008). "SnoReport: computational identification of snoRNAs with unknown targets". <i>Bioinformatics</i>. <b>24</b> (2): 158–64. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtm464">10.1093/bioinformatics/btm464</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17895272">17895272</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=SnoReport%3A+computational+identification+of+snoRNAs+with+unknown+targets.&amp;rft.volume=24&amp;rft.issue=2&amp;rft.pages=158-64&amp;rft.date=2008&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtm464&amp;rft_id=info%3Apmid%2F17895272&amp;rft.aulast=Hertel&amp;rft.aufirst=J&amp;rft.au=Hofacker%2C+IL&amp;rft.au=Stadler%2C+PF&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid10024243-158"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid10024243_158-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLoweEddy1999" class="citation journal">Lowe TM, Eddy SR (February 1999). "A computational screen for methylation guide snoRNAs in yeast". <i>Science</i>. <b>283</b> (5405): 1168–71. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1999Sci...283.1168L">1999Sci...283.1168L</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.283.5405.1168">10.1126/science.283.5405.1168</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/10024243">10024243</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=A+computational+screen+for+methylation+guide+snoRNAs+in+yeast&amp;rft.volume=283&amp;rft.issue=5405&amp;rft.pages=1168-71&amp;rft.date=1999-02&amp;rft_id=info%3Apmid%2F10024243&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.283.5405.1168&amp;rft_id=info%3Abibcode%2F1999Sci...283.1168L&amp;rft.aulast=Lowe&amp;rft.aufirst=TM&amp;rft.au=Eddy%2C+SR&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15980563-159"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid15980563_159-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid15980563_159-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite id="CITEREFSchattnerBrooksLowe2005" class="citation journal">Schattner P, Brooks AN, Lowe TM (July 2005). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1160127">"The tRNAscan-SE, snoscan and snoGPS web servers for the detection of tRNAs and snoRNAs"</a>. <i>Nucleic Acids Research</i>. <b>33</b> (Web Server issue): W686-9. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgki366">10.1093/nar/gki366</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1160127">1160127</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15980563">15980563</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=The+tRNAscan-SE%2C+snoscan+and+snoGPS+web+servers+for+the+detection+of+tRNAs+and+snoRNAs&amp;rft.volume=33&amp;rft.issue=Web+Server+issue&amp;rft.pages=W686-9&amp;rft.date=2005-07&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1160127&amp;rft_id=info%3Apmid%2F15980563&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgki366&amp;rft.aulast=Schattner&amp;rft.aufirst=P&amp;rft.au=Brooks%2C+AN&amp;rft.au=Lowe%2C+TM&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1160127&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid9023104-160"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid9023104_160-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLoweEddy1997" class="citation journal">Lowe TM, Eddy SR (1997). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC146525">"tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence"</a>. <i>Nucleic Acids Res</i>. <b>25</b> (5): 955–64. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F25.5.955">10.1093/nar/25.5.955</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC146525">146525</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/9023104">9023104</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence.&amp;rft.volume=25&amp;rft.issue=5&amp;rft.pages=955-64&amp;rft.date=1997&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC146525&amp;rft_id=info%3Apmid%2F9023104&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F25.5.955&amp;rft.aulast=Lowe&amp;rft.aufirst=TM&amp;rft.au=Eddy%2C+SR&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC146525&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid22362754-161"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid22362754_161-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFTempelTahi2012" class="citation journal">Tempel S, Tahi F (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3367186">"A fast ab-initio method for predicting miRNA precursors in genomes"</a>. <i>Nucleic Acids Res</i>. <b>40</b> (11): 955–64. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgks146">10.1093/nar/gks146</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3367186">3367186</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22362754">22362754</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=A+fast+ab-initio+method+for+predicting+miRNA+precursors+in+genomes.&amp;rft.volume=40&amp;rft.issue=11&amp;rft.pages=955-64&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3367186&amp;rft_id=info%3Apmid%2F22362754&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgks146&amp;rft.aulast=Tempel&amp;rft.aufirst=S&amp;rft.au=Tahi%2C+F&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3367186&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid11700055-162"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid11700055_162-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFGautheretLambert2001" class="citation journal">Gautheret D, Lambert A (2001). "Direct RNA motif definition and identification from multiple sequence alignments using secondary structure profiles". <i>J Mol Biol</i>. <b>313</b> (5): 1003–11. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1006%2Fjmbi.2001.5102">10.1006/jmbi.2001.5102</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/11700055">11700055</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J+Mol+Biol&amp;rft.atitle=Direct+RNA+motif+definition+and+identification+from+multiple+sequence+alignments+using+secondary+structure+profiles.&amp;rft.volume=313&amp;rft.issue=5&amp;rft.pages=1003-11&amp;rft.date=2001&amp;rft_id=info%3Adoi%2F10.1006%2Fjmbi.2001.5102&amp;rft_id=info%3Apmid%2F11700055&amp;rft.aulast=Gautheret&amp;rft.aufirst=D&amp;rft.au=Lambert%2C+A&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15215371-163"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15215371_163-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLambertFontaineLegendreLeclerc2004" class="citation journal">Lambert A, Fontaine JF, Legendre M, Leclerc F, Permal E, Major F, Putzer H, Delfour O, Michot B, Gautheret D (2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC441556">"The ERPIN server: an interface to profile-based RNA motif identification"</a>. <i>Nucleic Acids Res</i>. <b>32</b> (Web Server issue): W160–5. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkh418">10.1093/nar/gkh418</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC441556">441556</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15215371">15215371</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=The+ERPIN+server%3A+an+interface+to+profile-based+RNA+motif+identification.&amp;rft.volume=32&amp;rft.issue=Web+Server+issue&amp;rft.pages=W160-5&amp;rft.date=2004&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC441556&amp;rft_id=info%3Apmid%2F15215371&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkh418&amp;rft.aulast=Lambert&amp;rft.aufirst=A&amp;rft.au=Fontaine%2C+JF&amp;rft.au=Legendre%2C+M&amp;rft.au=Leclerc%2C+F&amp;rft.au=Permal%2C+E&amp;rft.au=Major%2C+F&amp;rft.au=Putzer%2C+H&amp;rft.au=Delfour%2C+O&amp;rft.au=Michot%2C+B&amp;rft.au=Gautheret%2C+D&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC441556&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15892887-164"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15892887_164-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLambertLegendreFontaineGautheret2005" class="citation journal">Lambert A, Legendre M, Fontaine JF, Gautheret D (2005). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1168889">"Computing expectation values for RNA motifs using discrete convolutions"</a>. <i>BMC Bioinformatics</i>. <b>6</b> (1): 118. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-6-118">10.1186/1471-2105-6-118</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1168889">1168889</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15892887">15892887</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Computing+expectation+values+for+RNA+motifs+using+discrete+convolutions.&amp;rft.volume=6&amp;rft.issue=1&amp;rft.pages=118&amp;rft.date=2005&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1168889&amp;rft_id=info%3Apmid%2F15892887&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-6-118&amp;rft.aulast=Lambert&amp;rft.aufirst=A&amp;rft.au=Legendre%2C+M&amp;rft.au=Fontaine%2C+JF&amp;rft.au=Gautheret%2C+D&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1168889&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17397253-165"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17397253_165-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFNawrockiEddy2007" class="citation journal">Nawrocki EP, Eddy SR (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1847999">"Query-dependent banding (QDB) for faster RNA similarity searches"</a>. <i>PLOS Comput. Biol</i>. <b>3</b> (3): e56. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2007PLSCB...3...56N">2007PLSCB...3...56N</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pcbi.0030056">10.1371/journal.pcbi.0030056</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1847999">1847999</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17397253">17397253</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+Comput.+Biol.&amp;rft.atitle=Query-dependent+banding+%28QDB%29+for+faster+RNA+similarity+searches.&amp;rft.volume=3&amp;rft.issue=3&amp;rft.pages=e56&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1847999&amp;rft_id=info%3Apmid%2F17397253&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.0030056&amp;rft_id=info%3Abibcode%2F2007PLSCB...3...56N&amp;rft.aulast=Nawrocki&amp;rft.aufirst=EP&amp;rft.au=Eddy%2C+SR&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1847999&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid12095421-166"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid12095421_166-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFEddy_SR2002" class="citation journal">Eddy SR (2002). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC119854">"A memory-efficient dynamic programming algorithm for optimal alignment of a sequence to an RNA secondary structure"</a>. <i>BMC Bioinformatics</i>. <b>3</b> (1): 18. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-3-18">10.1186/1471-2105-3-18</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC119854">119854</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/12095421">12095421</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=A+memory-efficient+dynamic+programming+algorithm+for+optimal+alignment+of+a+sequence+to+an+RNA+secondary+structure.&amp;rft.volume=3&amp;rft.issue=1&amp;rft.pages=18&amp;rft.date=2002&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC119854&amp;rft_id=info%3Apmid%2F12095421&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-3-18&amp;rft.au=Eddy+SR&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC119854&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid8029015-167"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid8029015_167-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFEddyDurbin1994" class="citation journal">Eddy SR, Durbin R (1994). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC308124">"RNA sequence analysis using covariance models"</a>. <i>Nucleic Acids Res</i>. <b>22</b> (11): 2079–88. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F22.11.2079">10.1093/nar/22.11.2079</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC308124">308124</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/8029015">8029015</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=RNA+sequence+analysis+using+covariance+models.&amp;rft.volume=22&amp;rft.issue=11&amp;rft.pages=2079-88&amp;rft.date=1994&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC308124&amp;rft_id=info%3Apmid%2F8029015&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F22.11.2079&amp;rft.aulast=Eddy&amp;rft.aufirst=SR&amp;rft.au=Durbin%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC308124&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16204111-168"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16204111_168-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSatoSakakibara2005" class="citation journal">Sato K, Sakakibara Y (2005). "RNA secondary structural alignment with conditional random fields". <i>Bioinformatics</i>. 21. Suppl 2 (suppl_2): ii237–42. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbti1139">10.1093/bioinformatics/bti1139</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16204111">16204111</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RNA+secondary+structural+alignment+with+conditional+random+fields.&amp;rft.volume=Suppl+2&amp;rft.issue=suppl_2&amp;rft.pages=ii237-42&amp;rft.date=2005&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbti1139&amp;rft_id=info%3Apmid%2F16204111&amp;rft.aulast=Sato&amp;rft.aufirst=K&amp;rft.au=Sakakibara%2C+Y&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15262817-169"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15262817_169-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWeinbergRuzzo2004" class="citation journal">Weinberg Z, Ruzzo WL (2004). "Exploiting conserved structure for faster annotation of non-coding RNAs without loss of accuracy". <i>Bioinformatics</i>. 20. Suppl 1 (suppl_1): i334–41. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbth925">10.1093/bioinformatics/bth925</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15262817">15262817</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Exploiting+conserved+structure+for+faster+annotation+of+non-coding+RNAs+without+loss+of+accuracy.&amp;rft.volume=Suppl+1&amp;rft.issue=suppl_1&amp;rft.pages=i334-41&amp;rft.date=2004&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbth925&amp;rft_id=info%3Apmid%2F15262817&amp;rft.aulast=Weinberg&amp;rft.aufirst=Z&amp;rft.au=Ruzzo%2C+WL&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16267089-170"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16267089_170-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWeinbergRuzzo2006" class="citation journal">Weinberg Z, Ruzzo WL (2006). "Sequence-based heuristics for faster annotation of non-coding RNA families". <i>Bioinformatics</i>. <b>22</b> (1): 35–9. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbti743">10.1093/bioinformatics/bti743</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16267089">16267089</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Sequence-based+heuristics+for+faster+annotation+of+non-coding+RNA+families.&amp;rft.volume=22&amp;rft.issue=1&amp;rft.pages=35-9&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbti743&amp;rft_id=info%3Apmid%2F16267089&amp;rft.aulast=Weinberg&amp;rft.aufirst=Z&amp;rft.au=Ruzzo%2C+WL&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid14499004-171"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid14499004_171-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFKleinEddy2003" class="citation journal">Klein RJ, Eddy SR (2003). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC239859">"RSEARCH: finding homologs of single structured RNA sequences"</a>. <i>BMC Bioinformatics</i>. <b>4</b> (1): 44. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-4-44">10.1186/1471-2105-4-44</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC239859">239859</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/14499004">14499004</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=RSEARCH%3A+finding+homologs+of+single+structured+RNA+sequences.&amp;rft.volume=4&amp;rft.issue=1&amp;rft.pages=44&amp;rft.date=2003&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC239859&amp;rft_id=info%3Apmid%2F14499004&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-4-44&amp;rft.aulast=Klein&amp;rft.aufirst=RJ&amp;rft.au=Eddy%2C+SR&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC239859&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid21619640-172"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid21619640_172-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMeyerKurtzBackofenWill2011" class="citation journal">Meyer F, Kurtz S, Backofen R, Will S, Beckstette M (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3154205">"Structator: fast index-based search for RNA sequence-structure patterns"</a>. <i>BMC Bioinformatics</i>. <b>12</b> (1): 214. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-12-214">10.1186/1471-2105-12-214</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3154205">3154205</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21619640">21619640</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Structator%3A+fast+index-based+search+for+RNA+sequence-structure+patterns&amp;rft.volume=12&amp;rft.issue=1&amp;rft.pages=214&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3154205&amp;rft_id=info%3Apmid%2F21619640&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-12-214&amp;rft.aulast=Meyer&amp;rft.aufirst=F&amp;rft.au=Kurtz%2C+S&amp;rft.au=Backofen%2C+R&amp;rft.au=Will%2C+S&amp;rft.au=Beckstette%2C+M&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3154205&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-173"><span class="mw-cite-backlink"><b><a href="#cite_ref-173">^</a></b></span> <span class="reference-text"><cite id="CITEREFMeyerKurtzBeckstette2013" class="citation journal">Meyer F, Kurtz S, Beckstette M (July 2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3765529">"Fast online and index-based algorithms for approximate search of RNA sequence-structure patterns"</a>. <i>BMC Bioinformatics</i>. <b>14</b> (1): 226. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-14-226">10.1186/1471-2105-14-226</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3765529">3765529</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23865810">23865810</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Fast+online+and+index-based+algorithms+for+approximate+search+of+RNA+sequence-structure+patterns&amp;rft.volume=14&amp;rft.issue=1&amp;rft.pages=226&amp;rft.date=2013-07&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3765529&amp;rft_id=info%3Apmid%2F23865810&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-14-226&amp;rft.aulast=Meyer&amp;rft.aufirst=F&amp;rft.au=Kurtz%2C+S&amp;rft.au=Beckstette%2C+M&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3765529&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15458580-176"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15458580_176-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFGardnerGiegerich2004" class="citation journal">Gardner PP, Giegerich R (2004). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC526219">"A comprehensive comparison of comparative RNA structure prediction approaches"</a>. <i>BMC Bioinformatics</i>. <b>5</b> (1): 140. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-5-140">10.1186/1471-2105-5-140</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC526219">526219</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15458580">15458580</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=A+comprehensive+comparison+of+comparative+RNA+structure+prediction+approaches&amp;rft.volume=5&amp;rft.issue=1&amp;rft.pages=140&amp;rft.date=2004&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC526219&amp;rft_id=info%3Apmid%2F15458580&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-5-140&amp;rft.aulast=Gardner&amp;rft.aufirst=PP&amp;rft.au=Giegerich%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC526219&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15860779-177"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15860779_177-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFGardnerWilmWashietl2005" class="citation journal">Gardner PP, Wilm A, Washietl S (2005). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1087786">"A benchmark of multiple sequence alignment programs upon structural RNAs"</a>. <i>Nucleic Acids Res</i>. <b>33</b> (8): 2433–9. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgki541">10.1093/nar/gki541</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1087786">1087786</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15860779">15860779</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.atitle=A+benchmark+of+multiple+sequence+alignment+programs+upon+structural+RNAs&amp;rft.volume=33&amp;rft.issue=8&amp;rft.pages=2433-9&amp;rft.date=2005&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1087786&amp;rft_id=info%3Apmid%2F15860779&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgki541&amp;rft.aulast=Gardner&amp;rft.aufirst=PP&amp;rft.au=Wilm%2C+A&amp;rft.au=Washietl%2C+S&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1087786&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17062125-178"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17062125_178-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWilmMainzSteger2006" class="citation journal">Wilm A, Mainz I, Steger G (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1635699">"An enhanced RNA alignment benchmark for sequence alignment programs"</a>. <i>Algorithms Mol Biol</i>. <b>1</b> (1): 19. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1748-7188-1-19">10.1186/1748-7188-1-19</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1635699">1635699</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17062125">17062125</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Algorithms+Mol+Biol&amp;rft.atitle=An+enhanced+RNA+alignment+benchmark+for+sequence+alignment+programs.&amp;rft.volume=1&amp;rft.issue=1&amp;rft.pages=19&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1635699&amp;rft_id=info%3Apmid%2F17062125&amp;rft_id=info%3Adoi%2F10.1186%2F1748-7188-1-19&amp;rft.aulast=Wilm&amp;rft.aufirst=A&amp;rft.au=Mainz%2C+I&amp;rft.au=Steger%2C+G&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1635699&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17151342-179"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17151342_179-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFFreyhultBollbackGardner2007" class="citation journal">Freyhult EK, Bollback JP, Gardner PP (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1716261">"Exploring genomic dark matter: a critical assessment of the performance of homology search methods on noncoding RNA"</a>. <i>Genome Res</i>. <b>17</b> (1): 117–25. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1101%2Fgr.5890907">10.1101/gr.5890907</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1716261">1716261</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17151342">17151342</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Genome+Res.&amp;rft.atitle=Exploring+genomic+dark+matter%3A+a+critical+assessment+of+the+performance+of+homology+search+methods+on+noncoding+RNA&amp;rft.volume=17&amp;rft.issue=1&amp;rft.pages=117-25&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1716261&amp;rft_id=info%3Apmid%2F17151342&amp;rft_id=info%3Adoi%2F10.1101%2Fgr.5890907&amp;rft.aulast=Freyhult&amp;rft.aufirst=EK&amp;rft.au=Bollback%2C+JP&amp;rft.au=Gardner%2C+PP&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1716261&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid23435231-180"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid23435231_180-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFPutonKozlowskiRotherBujnicki2013" class="citation journal">Puton T, Kozlowski LP, Rother KM, Bujnicki JM (2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3627593">"CompaRNA: a server for continuous benchmarking of automated methods for RNA secondary structure prediction"</a>. <i>Nucleic Acids Research</i>. <b>41</b> (7): 4307–23. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkt101">10.1093/nar/gkt101</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3627593">3627593</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23435231">23435231</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=CompaRNA%3A+a+server+for+continuous+benchmarking+of+automated+methods+for+RNA+secondary+structure+prediction&amp;rft.volume=41&amp;rft.issue=7&amp;rft.pages=4307-23&amp;rft.date=2013&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3627593&amp;rft_id=info%3Apmid%2F23435231&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkt101&amp;rft.aulast=Puton&amp;rft.aufirst=T&amp;rft.au=Kozlowski%2C+LP&amp;rft.au=Rother%2C+KM&amp;rft.au=Bujnicki%2C+JM&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3627593&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid32005745-181"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid32005745_181-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWright2020" class="citation journal">Wright ES (2020). "RNAconTest: Comparing tools for non-coding RNA multiple sequence alignment based on structural consistency". <i>RNA</i>: rna.073015.119. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1261%2Frna.073015.119">10.1261/rna.073015.119</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/32005745">32005745</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=RNA&amp;rft.atitle=RNAconTest%3A+Comparing+tools+for+non-coding+RNA+multiple+sequence+alignment+based+on+structural+consistency&amp;rft.pages=rna.073015.119&amp;rft.date=2020&amp;rft_id=info%3Adoi%2F10.1261%2Frna.073015.119&amp;rft_id=info%3Apmid%2F32005745&amp;rft.aulast=Wright&amp;rft.aufirst=ES&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17101042-184"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17101042_184-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFSeibelMüllerDandekarSchultz2006" class="citation journal">Seibel PN, Müller T, Dandekar T, Schultz J, Wolf M (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1637121">"4SALE--a tool for synchronous RNA sequence and secondary structure alignment and editing"</a>. <i>BMC Bioinformatics</i>. <b>7</b> (1): 498. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-7-498">10.1186/1471-2105-7-498</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1637121">1637121</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17101042">17101042</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=4SALE--a+tool+for+synchronous+RNA+sequence+and+secondary+structure+alignment+and+editing&amp;rft.volume=7&amp;rft.issue=1&amp;rft.pages=498&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1637121&amp;rft_id=info%3Apmid%2F17101042&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-7-498&amp;rft.aulast=Seibel&amp;rft.aufirst=PN&amp;rft.au=M%C3%BCller%2C+T&amp;rft.au=Dandekar%2C+T&amp;rft.au=Schultz%2C+J&amp;rft.au=Wolf%2C+M&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1637121&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-colorstock-185"><span class="mw-cite-backlink"><b><a href="#cite_ref-colorstock_185-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFBendanaHolmes2008" class="citation journal">Bendana YR, Holmes IH (2008). "Colorstock, SScolor, Rat ́on: RNA Alignment Visualization Tools". <i>Bioinformatics</i>. <b>24</b> (4): 579–80. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtm635">10.1093/bioinformatics/btm635</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18218657">18218657</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Colorstock%2C+SScolor%2C+Rat+%CC%81on%3A+RNA+Alignment+Visualization+Tools&amp;rft.volume=24&amp;rft.issue=4&amp;rft.pages=579-80&amp;rft.date=2008&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtm635&amp;rft_id=info%3Apmid%2F18218657&amp;rft.aulast=Bendana&amp;rft.aufirst=YR&amp;rft.au=Holmes%2C+IH&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-binf_igb_paper-186"><span class="mw-cite-backlink"><b><a href="#cite_ref-binf_igb_paper_186-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFNicolHeltBlanchard_SG_JrRaja2009" class="citation journal">Nicol JW, Helt GA, Blanchard SG Jr, Raja A, Loraine AE (2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2759552">"The Integrated Genome Browser: Free software for distribution and exploration of genome-scale data sets"</a>. <i>Bioinformatics</i>. <b>25</b> (20): 2730–2731. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp472">10.1093/bioinformatics/btp472</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2759552">2759552</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19654113">19654113</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=The+Integrated+Genome+Browser%3A+Free+software+for+distribution+and+exploration+of+genome-scale+data+sets.&amp;rft.volume=25&amp;rft.issue=20&amp;rft.pages=2730-2731&amp;rft.date=2009&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2759552&amp;rft_id=info%3Apmid%2F19654113&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtp472&amp;rft.aulast=Nicol&amp;rft.aufirst=JW&amp;rft.au=Helt%2C+GA&amp;rft.au=Blanchard+SG+Jr&amp;rft.au=Raja%2C+A&amp;rft.au=Loraine%2C+AE&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2759552&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid19151095-187"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19151095_187-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFWaterhouseProcterMartinClamp2009" class="citation journal">Waterhouse AM, Procter JB, Martin DM, Clamp M, Barton GJ (2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2672624">"Jalview Version 2--a multiple sequence alignment editor and analysis workbench"</a>. <i>Bioinformatics</i>. <b>25</b> (9): 1189–91. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp033">10.1093/bioinformatics/btp033</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2672624">2672624</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19151095">19151095</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Jalview+Version+2--a+multiple+sequence+alignment+editor+and+analysis+workbench.&amp;rft.volume=25&amp;rft.issue=9&amp;rft.pages=1189-91&amp;rft.date=2009&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2672624&amp;rft_id=info%3Apmid%2F19151095&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtp033&amp;rft.aulast=Waterhouse&amp;rft.aufirst=AM&amp;rft.au=Procter%2C+JB&amp;rft.au=Martin%2C+DM&amp;rft.au=Clamp%2C+M&amp;rft.au=Barton%2C+GJ&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2672624&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid14960472-188"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid14960472_188-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFClampCuffSearleBarton2004" class="citation journal">Clamp M, Cuff J, Searle SM, Barton GJ (2004). "The Jalview Java alignment editor". <i>Bioinformatics</i>. <b>20</b> (3): 426–7. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtg430">10.1093/bioinformatics/btg430</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/14960472">14960472</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=The+Jalview+Java+alignment+editor.&amp;rft.volume=20&amp;rft.issue=3&amp;rft.pages=426-7&amp;rft.date=2004&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtg430&amp;rft_id=info%3Apmid%2F14960472&amp;rft.aulast=Clamp&amp;rft.aufirst=M&amp;rft.au=Cuff%2C+J&amp;rft.au=Searle%2C+SM&amp;rft.au=Barton%2C+GJ&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid15377506-189"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid15377506_189-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFGriffiths-Jones_S2005" class="citation journal">Griffiths-Jones S (2005). "RALEE--RNA ALignment editor in Emacs". <i>Bioinformatics</i>. <b>21</b> (2): 257–9. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbth489">10.1093/bioinformatics/bth489</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15377506">15377506</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RALEE--RNA+ALignment+editor+in+Emacs&amp;rft.volume=21&amp;rft.issue=2&amp;rft.pages=257-9&amp;rft.date=2005&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbth489&amp;rft_id=info%3Apmid%2F15377506&amp;rft.au=Griffiths-Jones+S&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17804647-190"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17804647_190-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFAndersenLind-ThomsenKnudsen2007" class="citation journal">Andersen ES, Lind-Thomsen A, Knudsen B,  et al. (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2040093">"Semiautomated improvement of RNA alignments"</a>. <i>RNA</i>. <b>13</b> (11): 1850–9. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1261%2Frna.215407">10.1261/rna.215407</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2040093">2040093</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17804647">17804647</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=RNA&amp;rft.atitle=Semiautomated+improvement+of+RNA+alignments&amp;rft.volume=13&amp;rft.issue=11&amp;rft.pages=1850-9&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2040093&amp;rft_id=info%3Apmid%2F17804647&amp;rft_id=info%3Adoi%2F10.1261%2Frna.215407&amp;rft.aulast=Andersen&amp;rft.aufirst=ES&amp;rft.au=Lind-Thomsen%2C+A&amp;rft.au=Knudsen%2C+B&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2040093&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-EteRNA-191"><span class="mw-cite-backlink"><b><a href="#cite_ref-EteRNA_191-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFLee,_J._and_Kladwang,_W._and_Lee,_M._and_Cantu,_D._and_Azizyan,_M._and_Kim,_H._and_Limpaecher,_A._and_Yoon,_S._and_Treuille,_A._and_Das,_R.2014" class="citation journal">Lee, J. and Kladwang, W. and Lee, M. and Cantu, D. and Azizyan, M. and Kim, H. and Limpaecher, A. and Yoon, S. and Treuille, A. and Das, R. (2014). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3926058">"RNA design rules from a massive open laboratory"</a>. <i>PNAS</i>. <b>111</b> (6): 2122–2127. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2014PNAS..111.2122L">2014PNAS..111.2122L</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.1313039111">10.1073/pnas.1313039111</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3926058">3926058</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24469816">24469816</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PNAS&amp;rft.atitle=RNA+design+rules+from+a+massive+open+laboratory&amp;rft.volume=111&amp;rft.issue=6&amp;rft.pages=2122-2127&amp;rft.date=2014&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3926058&amp;rft_id=info%3Apmid%2F24469816&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.1313039111&amp;rft_id=info%3Abibcode%2F2014PNAS..111.2122L&amp;rft.au=Lee%2C+J.+and+Kladwang%2C+W.+and+Lee%2C+M.+and+Cantu%2C+D.+and+Azizyan%2C+M.+and+Kim%2C+H.+and+Limpaecher%2C+A.+and+Yoon%2C+S.+and+Treuille%2C+A.+and+Das%2C+R.&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3926058&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><span class="cs1-maint citation-comment">CS1 maint: multiple names: authors list (<a href="/wiki/Category:CS1_maint:_multiple_names:_authors_list" title="Category:CS1 maint: multiple names: authors list">link</a>)</span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNAiFold-192"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNAiFold_192-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFJ._A._Garcia-MartinP._CloteI._Dotu2013" class="citation journal">J. A. Garcia-Martin; P. Clote; I. Dotu (2013). "RNAiFold: A constraint programming algorithm for RNA inverse folding and molecular design". <i>Journal of Bioinformatics and Computational Biology</i>. <b>11</b> (2): 1350001. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1142%2FS0219720013500017">10.1142/S0219720013500017</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23600819">23600819</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Bioinformatics+and+Computational+Biology&amp;rft.atitle=RNAiFold%3A+A+constraint+programming+algorithm+for+RNA+inverse+folding+and+molecular+design&amp;rft.volume=11&amp;rft.issue=2&amp;rft.pages=1350001&amp;rft.date=2013&amp;rft_id=info%3Adoi%2F10.1142%2FS0219720013500017&amp;rft_id=info%3Apmid%2F23600819&amp;rft.au=J.+A.+Garcia-Martin&amp;rft.au=P.+Clote&amp;rft.au=I.+Dotu&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNAiFoldWeb-193"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNAiFoldWeb_193-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFJ._A._Garcia-MartinP._CloteI._Dotu2013" class="citation journal">J. A. Garcia-Martin; P. Clote; I. Dotu (2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3692061">"RNAiFold: a web server for RNA inverse folding and molecular design"</a>. <i>Nucleic Acids Research</i>. <b>41</b> (W1): W465-70. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkt280">10.1093/nar/gkt280</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3692061">3692061</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23700314">23700314</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=RNAiFold%3A+a+web+server+for+RNA+inverse+folding+and+molecular+design&amp;rft.volume=41&amp;rft.issue=W1&amp;rft.pages=W465-70&amp;rft.date=2013&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3692061&amp;rft_id=info%3Apmid%2F23700314&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkt280&amp;rft.au=J.+A.+Garcia-Martin&amp;rft.au=P.+Clote&amp;rft.au=I.+Dotu&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3692061&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNAiFold2Web-194"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNAiFold2Web_194-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFJ._A._Garcia-MartinI._DotuP._Clote2015" class="citation journal">J. A. Garcia-Martin; I. Dotu; P. Clote (2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4489274">"RNAiFold 2.0: a web server and software to design custom and Rfam-based RNA molecules"</a>. <i>Nucleic Acids Research</i>. <b>43</b> (W1): W513-21. <a href="/wiki/ArXiv_(identifier)" class="mw-redirect" title="ArXiv (identifier)">arXiv</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//arxiv.org/abs/1505.04210">1505.04210</a></span>. <a href="/wiki/Bibcode_(identifier)" class="mw-redirect" title="Bibcode (identifier)">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2015arXiv150504210G">2015arXiv150504210G</a>. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkv460">10.1093/nar/gkv460</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4489274">4489274</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/26019176">26019176</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=RNAiFold+2.0%3A+a+web+server+and+software+to+design+custom+and+Rfam-based+RNA+molecules&amp;rft.volume=43&amp;rft.issue=W1&amp;rft.pages=W513-21&amp;rft.date=2015&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4489274&amp;rft_id=info%3Abibcode%2F2015arXiv150504210G&amp;rft_id=info%3Aarxiv%2F1505.04210&amp;rft_id=info%3Apmid%2F26019176&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkv460&amp;rft.au=J.+A.+Garcia-Martin&amp;rft.au=I.+Dotu&amp;rft.au=P.+Clote&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4489274&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNA-SSD-195"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNA-SSD_195-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFM_AndronescuA_P_FejesF_HutterH_H_Hoos2004" class="citation journal">M Andronescu; A P Fejes; F Hutter; H H Hoos; A Condon (2004). "A new algorithm for RNA secondary structure design". <i>Journal of Molecular Biology</i>. <b>336</b> (3): 607–624. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.jmb.2003.12.041">10.1016/j.jmb.2003.12.041</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15095976">15095976</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Molecular+Biology&amp;rft.atitle=A+new+algorithm+for+RNA+secondary+structure+design&amp;rft.volume=336&amp;rft.issue=3&amp;rft.pages=607-624&amp;rft.date=2004&amp;rft_id=info%3Adoi%2F10.1016%2Fj.jmb.2003.12.041&amp;rft_id=info%3Apmid%2F15095976&amp;rft.au=M+Andronescu&amp;rft.au=A+P+Fejes&amp;rft.au=F+Hutter&amp;rft.au=H+H+Hoos&amp;rft.au=A+Condon&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-INFO-RNA-196"><span class="mw-cite-backlink"><b><a href="#cite_ref-INFO-RNA_196-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFA_BuschR_Backofen2006" class="citation journal">A Busch &amp; R Backofen (2006). "INFO-RNA--a fast approach to inverse RNA folding". <i>Bioinformatics</i>. <b>22</b> (15): 1823–1831. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl194">10.1093/bioinformatics/btl194</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16709587">16709587</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=INFO-RNA--a+fast+approach+to+inverse+RNA+folding&amp;rft.volume=22&amp;rft.issue=15&amp;rft.pages=1823-1831&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtl194&amp;rft_id=info%3Apmid%2F16709587&amp;rft.au=A+Busch&amp;rft.au=R+Backofen&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-INFO-RNA-WebServer-197"><span class="mw-cite-backlink"><b><a href="#cite_ref-INFO-RNA-WebServer_197-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFA_BuschR_Backofen2007" class="citation journal">A Busch &amp; R Backofen (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1933236">"INFO-RNA--a server for fast inverse RNA folding satisfying sequence constraints"</a>. <i>Nucleic Acids Research</i>. <b>35</b> (Web Server Issue): W310-3. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkm218">10.1093/nar/gkm218</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1933236">1933236</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17452349">17452349</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=INFO-RNA--a+server+for+fast+inverse+RNA+folding+satisfying+sequence+constraints&amp;rft.volume=35&amp;rft.issue=Web+Server+Issue&amp;rft.pages=W310-3&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1933236&amp;rft_id=info%3Apmid%2F17452349&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkm218&amp;rft.au=A+Busch&amp;rft.au=R+Backofen&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1933236&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNAexinv-198"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNAexinv_198-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFA_Avihoo,_A_ChurkinD_Barash2011" class="citation journal">A Avihoo, A Churkin &amp; D Barash (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3176266">"RNAexinv: An extended inverse RNA folding from shape and physical attributes to sequences"</a>. <i>BMC Bioinformatics</i>. <b>12</b> (319): 319. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-12-319">10.1186/1471-2105-12-319</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3176266">3176266</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21813013">21813013</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=RNAexinv%3A+An+extended+inverse+RNA+folding+from+shape+and+physical+attributes+to+sequences&amp;rft.volume=12&amp;rft.issue=319&amp;rft.pages=319&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3176266&amp;rft_id=info%3Apmid%2F21813013&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-12-319&amp;rft.au=A+Avihoo%2C+A+Churkin&amp;rft.au=D+Barash&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3176266&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNA-ensign-199"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNA-ensign_199-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFA._LevinM._LisY._PontyC._W._O’Donnell2012" class="citation journal">A. Levin; M. Lis; Y. Ponty; C. W. O’Donnell; S. Devadas; B. Berger &amp; J. Waldispühl (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3488226">"A global sampling approach to designing and reengineering RNA secondary structures"</a>. <i>Nucleic Acids Research</i>. <b>40</b> (20): 10041–10052. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgks768">10.1093/nar/gks768</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3488226">3488226</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22941632">22941632</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=A+global+sampling+approach+to+designing+and+reengineering+RNA+secondary+structures&amp;rft.volume=40&amp;rft.issue=20&amp;rft.pages=10041-10052&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3488226&amp;rft_id=info%3Apmid%2F22941632&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgks768&amp;rft.au=A.+Levin&amp;rft.au=M.+Lis&amp;rft.au=Y.+Ponty&amp;rft.au=C.+W.+O%E2%80%99Donnell&amp;rft.au=S.+Devadas&amp;rft.au=B.+Berger&amp;rft.au=J.+Waldisp%C3%BChl&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3488226&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-IncaRNAtion-200"><span class="mw-cite-backlink"><b><a href="#cite_ref-IncaRNAtion_200-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFV_Reinharz,_Y._PontyJérôme_Waldispühl2013" class="citation journal">V Reinharz, Y. Ponty &amp; Jérôme Waldispühl (2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3694657">"A weighted sampling algorithm for the design of RNA sequences with targeted secondary structure and nucleotide distribution"</a>. <i>Bioinformatics</i>. <b>29</b> (13): i308–i315. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtt217">10.1093/bioinformatics/btt217</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3694657">3694657</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23812999">23812999</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=A+weighted+sampling+algorithm+for+the+design+of+RNA+sequences+with+targeted+secondary+structure+and+nucleotide+distribution&amp;rft.volume=29&amp;rft.issue=13&amp;rft.pages=i308-i315&amp;rft.date=2013&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3694657&amp;rft_id=info%3Apmid%2F23812999&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtt217&amp;rft.au=V+Reinharz%2C+Y.+Ponty&amp;rft.au=J%C3%A9r%C3%B4me+Waldisp%C3%BChl&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3694657&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-DSS-Opt-201"><span class="mw-cite-backlink"><b><a href="#cite_ref-DSS-Opt_201-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFM._C._MatthiesS._BienertA._E._Torda2012" class="citation journal">M. C. Matthies; S. Bienert &amp; A. E. Torda (2012). "Dynamics in Sequence Space for RNA Secondary Structure Design". <i>Journal of Chemical Theory and Computation</i>. <b>8</b> (10): 3663–3670. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1021%2Fct300267j">10.1021/ct300267j</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/26593011">26593011</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Chemical+Theory+and+Computation&amp;rft.atitle=Dynamics+in+Sequence+Space+for+RNA+Secondary+Structure+Design&amp;rft.volume=8&amp;rft.issue=10&amp;rft.pages=3663-3670&amp;rft.date=2012&amp;rft_id=info%3Adoi%2F10.1021%2Fct300267j&amp;rft_id=info%3Apmid%2F26593011&amp;rft.au=M.+C.+Matthies&amp;rft.au=S.+Bienert&amp;rft.au=A.+E.+Torda&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-MODENA-202"><span class="mw-cite-backlink"><b><a href="#cite_ref-MODENA_202-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFA._Taneda2011" class="citation journal">A. Taneda (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3169953">"MODENA: a multi-objective RNA inverse folding"</a>. <i>Advances and Applications in Bioinformatics and Chemistry</i>. <b>4</b>: 1–12. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.2147%2Faabc.s14335">10.2147/aabc.s14335</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3169953">3169953</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21918633">21918633</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Advances+and+Applications+in+Bioinformatics+and+Chemistry&amp;rft.atitle=MODENA%3A+a+multi-objective+RNA+inverse+folding&amp;rft.volume=4&amp;rft.pages=1-12&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3169953&amp;rft_id=info%3Apmid%2F21918633&amp;rft_id=info%3Adoi%2F10.2147%2Faabc.s14335&amp;rft.au=A.+Taneda&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3169953&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-MODENA_E-203"><span class="mw-cite-backlink"><b><a href="#cite_ref-MODENA_E_203-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFA._Taneda2012" class="citation journal">A. Taneda (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3337422">"Multi-Objective Genetic Algorithm for Pseudoknotted RNA Sequence Design"</a>. <i>Frontiers in Genetics</i>. <b>3</b>: 36. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.3389%2Ffgene.2012.00036">10.3389/fgene.2012.00036</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3337422">3337422</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22558001">22558001</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Frontiers+in+Genetics&amp;rft.atitle=Multi-Objective+Genetic+Algorithm+for+Pseudoknotted+RNA+Sequence+Design&amp;rft.volume=3&amp;rft.pages=36&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3337422&amp;rft_id=info%3Apmid%2F22558001&amp;rft_id=info%3Adoi%2F10.3389%2Ffgene.2012.00036&amp;rft.au=A.+Taneda&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3337422&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-ERD-204"><span class="mw-cite-backlink"><b><a href="#cite_ref-ERD_204-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFA_Esmaili-TaheriM_GanjtabeshM_Mohammad-Noori2014" class="citation journal">A Esmaili-Taheri; M Ganjtabesh; M Mohammad-Noori (2014). "Evolutionary solution for the RNA design problem". <i>Bioinformatics</i>. <b>30</b> (9): 1250–1258. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu001">10.1093/bioinformatics/btu001</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24407223">24407223</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Evolutionary+solution+for+the+RNA+design+problem.&amp;rft.volume=30&amp;rft.issue=9&amp;rft.pages=1250-1258&amp;rft.date=2014&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtu001&amp;rft_id=info%3Apmid%2F24407223&amp;rft.au=A+Esmaili-Taheri&amp;rft.au=M+Ganjtabesh&amp;rft.au=M+Mohammad-Noori&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-antaRNA_basis-205"><span class="mw-cite-backlink"><b><a href="#cite_ref-antaRNA_basis_205-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFR_KleinkaufM_MannR_Backofen2015" class="citation journal">R Kleinkauf; M Mann; R Backofen (2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4576691">"antaRNA: ant colony-based RNA sequence design"</a>. <i>Bioinformatics</i>. <b>31</b> (19): 3114–3121. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv319">10.1093/bioinformatics/btv319</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4576691">4576691</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/26023105">26023105</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=antaRNA%3A+ant+colony-based+RNA+sequence+design&amp;rft.volume=31&amp;rft.issue=19&amp;rft.pages=3114-3121&amp;rft.date=2015&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4576691&amp;rft_id=info%3Apmid%2F26023105&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtv319&amp;rft.au=R+Kleinkauf&amp;rft.au=M+Mann&amp;rft.au=R+Backofen&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4576691&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-antaRNA_pk-206"><span class="mw-cite-backlink"><b><a href="#cite_ref-antaRNA_pk_206-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFR_KleinkaufT_HouwaartR_BackofenM_Mann2015" class="citation journal">R Kleinkauf; T Houwaart; R Backofen; M Mann (2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4652366">"antaRNA – Multi-objective inverse folding of pseudoknot RNA using ant-colony optimization"</a>. <i>BMC Bioinformatics</i>. <b>16</b> (389): 389. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2Fs12859-015-0815-6">10.1186/s12859-015-0815-6</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4652366">4652366</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/26581440">26581440</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=antaRNA+%E2%80%93+Multi-objective+inverse+folding+of+pseudoknot+RNA+using+ant-colony+optimization&amp;rft.volume=16&amp;rft.issue=389&amp;rft.pages=389&amp;rft.date=2015&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4652366&amp;rft_id=info%3Apmid%2F26581440&amp;rft_id=info%3Adoi%2F10.1186%2Fs12859-015-0815-6&amp;rft.au=R+Kleinkauf&amp;rft.au=T+Houwaart&amp;rft.au=R+Backofen&amp;rft.au=M+Mann&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4652366&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-switch.pl-207"><span class="mw-cite-backlink"><b><a href="#cite_ref-switch.pl_207-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFC_FlammI_L_HofackerS_Maurer-StrohP_F_Stadler2001" class="citation journal">C Flamm; I L Hofacker; S Maurer-Stroh; P F Stadler; M Zehl (2001). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1370083">"Design of multistable RNA molecules"</a>. <i>RNA</i>. <b>7</b> (2): 254–265. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1017%2Fs1355838201000863">10.1017/s1355838201000863</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1370083">1370083</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/11233982">11233982</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=RNA&amp;rft.atitle=Design+of+multistable+RNA+molecules.&amp;rft.volume=7&amp;rft.issue=2&amp;rft.pages=254-265&amp;rft.date=2001&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1370083&amp;rft_id=info%3Apmid%2F11233982&amp;rft_id=info%3Adoi%2F10.1017%2Fs1355838201000863&amp;rft.au=C+Flamm&amp;rft.au=I+L+Hofacker&amp;rft.au=S+Maurer-Stroh&amp;rft.au=P+F+Stadler&amp;rft.au=M+Zehl&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1370083&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RiboMaker-208"><span class="mw-cite-backlink"><b><a href="#cite_ref-RiboMaker_208-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFG_Rodrigo_GA_Jaramillo2014" class="citation journal">G Rodrigo G &amp; A Jaramillo (2014). "RiboMaker: computational design of conformation-based riboregulation". <i>Bioinformatics</i>. <b>30</b> (17): 2508–2510. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu335">10.1093/bioinformatics/btu335</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24833802">24833802</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RiboMaker%3A+computational+design+of+conformation-based+riboregulation.&amp;rft.volume=30&amp;rft.issue=17&amp;rft.pages=2508-2510&amp;rft.date=2014&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtu335&amp;rft_id=info%3Apmid%2F24833802&amp;rft.au=G+Rodrigo+G&amp;rft.au=A+Jaramillo&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNAblueprint-209"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNAblueprint_209-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFS_HammerB_TschiatschekC_FlammI_L_Hofacker2017" class="citation journal">S Hammer; B Tschiatschek; C Flamm; I L Hofacker &amp; S Findeiß (2017). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC5870862">"RNAblueprint: flexible multiple target nucleic acid sequence design"</a>. <i>Bioinformatics</i>. <b>33</b> (18): 2850–2858. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtx263">10.1093/bioinformatics/btx263</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC5870862">5870862</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/28449031">28449031</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RNAblueprint%3A+flexible+multiple+target+nucleic+acid+sequence+design&amp;rft.volume=33&amp;rft.issue=18&amp;rft.pages=2850-2858&amp;rft.date=2017&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5870862&amp;rft_id=info%3Apmid%2F28449031&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtx263&amp;rft.au=S+Hammer&amp;rft.au=B+Tschiatschek&amp;rft.au=C+Flamm&amp;rft.au=I+L+Hofacker&amp;rft.au=S+Findei%C3%9F&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5870862&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-RNADesign-210"><span class="mw-cite-backlink"><b><a href="#cite_ref-RNADesign_210-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFC_Höner_zu_SiederdissenS_HammerI_AbfalterI_L_Hofacker2013" class="citation journal">C Höner zu Siederdissen; S Hammer; I Abfalter; I L Hofacker; C Flamm &amp; P F Stadler (2013). "Computational Design of RNAs with Complex Energy Landscapes". <i>Biopolymers</i>. <b>99</b> (12): 1124–1136. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2Fbip.22337">10.1002/bip.22337</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23818234">23818234</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biopolymers&amp;rft.atitle=Computational+Design+of+RNAs+with+Complex+Energy+Landscapes&amp;rft.volume=99&amp;rft.issue=12&amp;rft.pages=1124-1136&amp;rft.date=2013&amp;rft_id=info%3Adoi%2F10.1002%2Fbip.22337&amp;rft_id=info%3Apmid%2F23818234&amp;rft.au=C+H%C3%B6ner+zu+Siederdissen&amp;rft.au=S+Hammer&amp;rft.au=I+Abfalter&amp;rft.au=I+L+Hofacker&amp;rft.au=C+Flamm&amp;rft.au=P+F+Stadler&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-Frnakenstein-211"><span class="mw-cite-backlink"><b><a href="#cite_ref-Frnakenstein_211-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFRB_LyngsøJ_W_J_AndersonE_SizikovaA_Badugu2012" class="citation journal">RB Lyngsø; J W J Anderson; E Sizikova; A Badugu; T Hyland &amp; Jotun Hein (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3534541">"Frnakenstein: multiple target inverse RNA folding"</a>. <i>BMC Bioinformatics</i>. <b>13</b> (260): 260. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-13-260">10.1186/1471-2105-13-260</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3534541">3534541</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23043260">23043260</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=Frnakenstein%3A+multiple+target+inverse+RNA+folding&amp;rft.volume=13&amp;rft.issue=260&amp;rft.pages=260&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3534541&amp;rft_id=info%3Apmid%2F23043260&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-13-260&amp;rft.au=RB+Lyngs%C3%B8&amp;rft.au=J+W+J+Anderson&amp;rft.au=E+Sizikova&amp;rft.au=A+Badugu&amp;rft.au=T+Hyland&amp;rft.au=Jotun+Hein&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3534541&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-ARDesigner-212"><span class="mw-cite-backlink"><b><a href="#cite_ref-ARDesigner_212-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFW._ShuM._LiuH._ChenX._Bo2010" class="citation journal">W. Shu; M. Liu; H. Chen; X. Bo; S. Wang (2010). "ARDesigner: A web-based system for allosteric RNA design". <i>Journal of Biotechnology</i>. <b>150</b> (4): 466–473. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.jbiotec.2010.10.067">10.1016/j.jbiotec.2010.10.067</a>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20969900">20969900</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Biotechnology&amp;rft.atitle=ARDesigner%3A+A+web-based+system+for+allosteric+RNA+design&amp;rft.volume=150&amp;rft.issue=4&amp;rft.pages=466-473&amp;rft.date=2010&amp;rft_id=info%3Adoi%2F10.1016%2Fj.jbiotec.2010.10.067&amp;rft_id=info%3Apmid%2F20969900&amp;rft.au=W.+Shu&amp;rft.au=M.+Liu&amp;rft.au=H.+Chen&amp;rft.au=X.+Bo&amp;rft.au=S.+Wang&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid19369500-213"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19369500_213-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFByunHan2009" class="citation journal">Byun Y, Han K (2009). "PseudoViewer3: generating planar drawings of large-scale RNA structures with pseudoknots". <i>Bioinformatics</i>. <b>25</b> (11): 1435–7. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp252">10.1093/bioinformatics/btp252</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19369500">19369500</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=PseudoViewer3%3A+generating+planar+drawings+of+large-scale+RNA+structures+with+pseudoknots.&amp;rft.volume=25&amp;rft.issue=11&amp;rft.pages=1435-7&amp;rft.date=2009&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtp252&amp;rft_id=info%3Apmid%2F19369500&amp;rft.aulast=Byun&amp;rft.aufirst=Y&amp;rft.au=Han%2C+K&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid16845039-214"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid16845039_214-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFByunHan2006" class="citation journal">Byun Y, Han K (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1538805">"PseudoViewer: web application and web service for visualizing RNA pseudoknots and secondary structures"</a>. <i>Nucleic Acids Res</i>. <b>34</b> (Web Server issue): W416–22. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkl210">10.1093/nar/gkl210</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1538805">1538805</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16845039">16845039</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=PseudoViewer%3A+web+application+and+web+service+for+visualizing+RNA+pseudoknots+and+secondary+structures.&amp;rft.volume=34&amp;rft.issue=Web+Server+issue&amp;rft.pages=W416-22&amp;rft.date=2006&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1538805&amp;rft_id=info%3Apmid%2F16845039&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkl210&amp;rft.aulast=Byun&amp;rft.aufirst=Y&amp;rft.au=Han%2C+K&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1538805&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid12824341-215"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid12824341_215-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHanByun2003" class="citation journal">Han K, Byun Y (2003). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC168946">"PSEUDOVIEWER2: Visualization of RNA pseudoknots of any type"</a>. <i>Nucleic Acids Res</i>. <b>31</b> (13): 3432–40. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkg539">10.1093/nar/gkg539</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC168946">168946</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/12824341">12824341</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=PSEUDOVIEWER2%3A+Visualization+of+RNA+pseudoknots+of+any+type.&amp;rft.volume=31&amp;rft.issue=13&amp;rft.pages=3432-40&amp;rft.date=2003&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC168946&amp;rft_id=info%3Apmid%2F12824341&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkg539&amp;rft.aulast=Han&amp;rft.aufirst=K&amp;rft.au=Byun%2C+Y&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC168946&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid12169562-216"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid12169562_216-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFHanLeeKim2002" class="citation journal">Han K, Lee Y, Kim W (2002). "PseudoViewer: automatic visualization of RNA pseudoknots". <i>Bioinformatics</i>. 18. <b>18</b> (Suppl 1): S321–8. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2F18.suppl_1.S321">10.1093/bioinformatics/18.suppl_1.S321</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/12169562">12169562</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=PseudoViewer%3A+automatic+visualization+of+RNA+pseudoknots.&amp;rft.volume=18&amp;rft.issue=Suppl+1&amp;rft.pages=S321-8&amp;rft.date=2002&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F18.suppl_1.S321&amp;rft_id=info%3Apmid%2F12169562&amp;rft.aulast=Han&amp;rft.aufirst=K&amp;rft.au=Lee%2C+Y&amp;rft.au=Kim%2C+W&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid17567618-217"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17567618_217-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFKaiserKrügerEvers2007" class="citation journal">Kaiser A, Krüger J, Evers DJ (2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1933240">"RNA Movies 2: sequential animation of RNA secondary structures"</a>. <i>Nucleic Acids Res</i>. <b>35</b> (Web Server issue): W330–4. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkm309">10.1093/nar/gkm309</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1933240">1933240</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17567618">17567618</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=RNA+Movies+2%3A+sequential+animation+of+RNA+secondary+structures.&amp;rft.volume=35&amp;rft.issue=Web+Server+issue&amp;rft.pages=W330-4&amp;rft.date=2007&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1933240&amp;rft_id=info%3Apmid%2F17567618&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkm309&amp;rft.aulast=Kaiser&amp;rft.aufirst=A&amp;rft.au=Kr%C3%BCger%2C+J&amp;rft.au=Evers%2C+DJ&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1933240&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid10068690-218"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid10068690_218-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFEversGiegerich1999" class="citation journal">Evers D, Giegerich R (1999). "RNA movies: visualizing RNA secondary structure spaces". <i>Bioinformatics</i>. <b>15</b> (1): 32–7. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2F15.1.32">10.1093/bioinformatics/15.1.32</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/10068690">10068690</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RNA+movies%3A+visualizing+RNA+secondary+structure+spaces.&amp;rft.volume=15&amp;rft.issue=1&amp;rft.pages=32-7&amp;rft.date=1999&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F15.1.32&amp;rft_id=info%3Apmid%2F10068690&amp;rft.aulast=Evers&amp;rft.aufirst=D&amp;rft.au=Giegerich%2C+R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-219"><span class="mw-cite-backlink"><b><a href="#cite_ref-219">^</a></b></span> <span class="reference-text"><cite id="CITEREFTsangDai2012" class="citation journal">Tsang HH, Dai DC (2012). "RNA-DV: an interactive tool for editing and visualizing RNA secondary structures". <i>Proceeding BCB '12 Proceedings of the ACM Conference on Bioinformatics, Computational Biology and Biomedicine</i>: 601–603. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1145%2F2382936.2383036">10.1145/2382936.2383036</a>. <a href="/wiki/ISBN_(identifier)" class="mw-redirect" title="ISBN (identifier)">ISBN</a>&#160;<a href="/wiki/Special:BookSources/9781450316705" title="Special:BookSources/9781450316705"><bdi>9781450316705</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proceeding+BCB+%2712+Proceedings+of+the+ACM+Conference+on+Bioinformatics%2C+Computational+Biology+and+Biomedicine&amp;rft.atitle=RNA-DV%3A+an+interactive+tool+for+editing+and+visualizing+RNA+secondary+structures&amp;rft.pages=601-603&amp;rft.date=2012&amp;rft_id=info%3Adoi%2F10.1145%2F2382936.2383036&amp;rft.isbn=9781450316705&amp;rft.aulast=Tsang&amp;rft.aufirst=Herbert+H.&amp;rft.au=Dai%2C+Denny+C.&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid18399701-220"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18399701_220-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMartinezMaizelShapiro2008" class="citation journal">Martinez HM, Maizel JV, Shapiro BA (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3727907">"RNA2D3D: a program for generating, viewing, and comparing 3-dimensional models of RNA"</a>. <i>J Biomol Struct Dyn</i>. <b>25</b> (6): 669–83. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1080%2F07391102.2008.10531240">10.1080/07391102.2008.10531240</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3727907">3727907</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18399701">18399701</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J+Biomol+Struct+Dyn&amp;rft.atitle=RNA2D3D%3A+a+program+for+generating%2C+viewing%2C+and+comparing+3-dimensional+models+of+RNA.&amp;rft.volume=25&amp;rft.issue=6&amp;rft.pages=669-83&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3727907&amp;rft_id=info%3Apmid%2F18399701&amp;rft_id=info%3Adoi%2F10.1080%2F07391102.2008.10531240&amp;rft.aulast=Martinez&amp;rft.aufirst=HM&amp;rft.au=Maizel%2C+JV&amp;rft.au=Shapiro%2C+BA&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3727907&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid20230624-221"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid20230624_221-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFReuterMathews2010" class="citation journal">Reuter JS, Mathews DH (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2984261">"RNAstructure: software for RNA secondary structure prediction and analysis"</a>. <i>BMC Bioinformatics</i>. <b>11</b> (1): 129. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-11-129">10.1186/1471-2105-11-129</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2984261">2984261</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20230624">20230624</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=RNAstructure%3A+software+for+RNA+secondary+structure+prediction+and+analysis.&amp;rft.volume=11&amp;rft.issue=1&amp;rft.pages=129&amp;rft.date=2010&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2984261&amp;rft_id=info%3Apmid%2F20230624&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-11-129&amp;rft.aulast=Reuter&amp;rft.aufirst=JS&amp;rft.au=Mathews%2C+DH&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2984261&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid12824344-222"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid12824344_222-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFYangJossinetLeontisChen2003" class="citation journal">Yang H, Jossinet F, Leontis N, Chen L, Westbrook J, Berman H, Westhof E (2003). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC168936">"Tools for the automatic identification and classification of RNA base pairs"</a>. <i>Nucleic Acids Res</i>. <b>31</b> (13): 3450–60. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkg529">10.1093/nar/gkg529</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC168936">168936</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/12824344">12824344</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.atitle=Tools+for+the+automatic+identification+and+classification+of+RNA+base+pairs.&amp;rft.volume=31&amp;rft.issue=13&amp;rft.pages=3450-60&amp;rft.date=2003&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC168936&amp;rft_id=info%3Apmid%2F12824344&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkg529&amp;rft.aulast=Yang&amp;rft.aufirst=H&amp;rft.au=Jossinet%2C+F&amp;rft.au=Leontis%2C+N&amp;rft.au=Chen%2C+L&amp;rft.au=Westbrook%2C+J&amp;rft.au=Berman%2C+H&amp;rft.au=Westhof%2C+E&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC168936&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid22826541-223"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid22826541_223-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFMenzelSeemannGorodkin2012" class="citation journal">Menzel P, Seemann SE, Gorodkin J (2012). "RILogo: visualizing RNA-RNA interactions". <i>Bioinformatics</i>. <b>28</b> (19): 2523–6. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts461">10.1093/bioinformatics/bts461</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22826541">22826541</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=RILogo%3A+visualizing+RNA-RNA+interactions.&amp;rft.volume=28&amp;rft.issue=19&amp;rft.pages=2523-6&amp;rft.date=2012&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbts461&amp;rft_id=info%3Apmid%2F22826541&amp;rft.aulast=Menzel&amp;rft.aufirst=P&amp;rft.au=Seemann%2C+SE&amp;rft.au=Gorodkin%2C+J&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid19398448-224"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19398448_224-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFDartyDenisePonty2009" class="citation journal">Darty K, Denise A, Ponty Y (2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2712331">"VARNA: Interactive drawing and editing of the RNA secondary structure"</a>. <i>Bioinformatics</i>. <b>25</b> (15): 1974–5. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp250">10.1093/bioinformatics/btp250</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2712331">2712331</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19398448">19398448</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=VARNA%3A+Interactive+drawing+and+editing+of+the+RNA+secondary+structure.&amp;rft.volume=25&amp;rft.issue=15&amp;rft.pages=1974-5&amp;rft.date=2009&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2712331&amp;rft_id=info%3Apmid%2F19398448&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtp250&amp;rft.aulast=Darty&amp;rft.aufirst=K&amp;rft.au=Denise%2C+A&amp;rft.au=Ponty%2C+Y&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2712331&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-pmid26099263-225"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid26099263_225-0">^</a></b></span> <span class="reference-text"><cite id="CITEREFKerpedjievHammerHofacker2015" class="citation journal">Kerpedjiev P, Hammer S, Hofacker IL (October 2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4595900">"Forna (force-directed RNA): Simple and effective online RNA secondary structure diagrams"</a>. <i>Bioinformatics</i>. <b>31</b> (20): 3377–9. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv372">10.1093/bioinformatics/btv372</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4595900">4595900</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/26099263">26099263</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Forna+%28force-directed+RNA%29%3A+Simple+and+effective+online+RNA+secondary+structure+diagrams&amp;rft.volume=31&amp;rft.issue=20&amp;rft.pages=3377-9&amp;rft.date=2015-10&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4595900&amp;rft_id=info%3Apmid%2F26099263&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtv372&amp;rft.aulast=Kerpedjiev&amp;rft.aufirst=P&amp;rft.au=Hammer%2C+S&amp;rft.au=Hofacker%2C+IL&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4595900&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
<li id="cite_note-226"><span class="mw-cite-backlink"><b><a href="#cite_ref-226">^</a></b></span> <span class="reference-text"><cite id="CITEREFWeinbergBreaker2011" class="citation journal">Weinberg, Zasha; Breaker, Ronald R (4 January 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3023696">"R2R - software to speed the depiction of aesthetic consensus RNA secondary structures"</a>. <i>BMC Bioinformatics</i>. <b>12</b> (1): 3. <a href="/wiki/Doi_(identifier)" class="mw-redirect" title="Doi (identifier)">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-12-3">10.1186/1471-2105-12-3</a>. <a href="/wiki/PMC_(identifier)" class="mw-redirect" title="PMC (identifier)">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3023696">3023696</a></span>. <a href="/wiki/PMID_(identifier)" class="mw-redirect" title="PMID (identifier)">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21205310">21205310</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=R2R+-+software+to+speed+the+depiction+of+aesthetic+consensus+RNA+secondary+structures&amp;rft.volume=12&amp;rft.issue=1&amp;rft.pages=3&amp;rft.date=2011-01-04&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3023696&amp;rft_id=info%3Apmid%2F21205310&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-12-3&amp;rft.aulast=Weinberg&amp;rft.aufirst=Zasha&amp;rft.au=Breaker%2C+Ronald+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3023696&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AList+of+RNA+structure+prediction+software" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r951705291"/></span>
</li>
</ol></div>
<!-- 
NewPP limit report
Parsed by mw1288
Cached time: 20200420071442
Cache expiry: 2592000
Dynamic content: false
Complications: [vary‐revision‐sha1]
CPU time usage: 1.936 seconds
Real time usage: 2.041 seconds
Preprocessor visited node count: 11977/1000000
Post‐expand include size: 571550/2097152 bytes
Template argument size: 3080/2097152 bytes
Highest expansion depth: 15/40
Expensive parser function count: 2/500
Unstrip recursion depth: 1/20
Unstrip post‐expand size: 971910/5000000 bytes
Number of Wikibase entities loaded: 0/400
Lua time usage: 1.108/10.000 seconds
Lua memory usage: 5.23 MB/50 MB
Lua Profile:
    Scribunto_LuaSandboxCallback::plain                              120 ms       10.5%
    ?                                                                120 ms       10.5%
    recursiveClone <mwInit.lua:41>                                   120 ms       10.5%
    Scribunto_LuaSandboxCallback::gsub                               100 ms        8.8%
    Scribunto_LuaSandboxCallback::callParserFunction                 100 ms        8.8%
    Scribunto_LuaSandboxCallback::find                               100 ms        8.8%
    <mw.lua:683>                                                      60 ms        5.3%
    Scribunto_LuaSandboxCallback::len                                 40 ms        3.5%
    Scribunto_LuaSandboxCallback::match                               40 ms        3.5%
    format                                                            40 ms        3.5%
    [others]                                                         300 ms       26.3%
-->
<!--
Transclusion expansion time report (%,ms,calls,template)
100.00% 1730.741      1 -total
 82.74% 1432.079      8 Template:Reflist
 67.24% 1163.810    209 Template:Cite_journal
  3.20%   55.441      1 Template:Short_description
  2.44%   42.240      1 Template:Pagetype
  2.43%   41.995      2 Template:Dead_link
  1.98%   34.232      2 Template:Fix
  1.73%   29.923      4 Template:Category_handler
  0.74%   12.890    187 Template:Yes
  0.56%    9.620      1 Template:Cite_arxiv
-->

<!-- Saved in parser cache with key enwiki:pcache:idhash:15308316-0!canonical!math=5 and timestamp 20200420071440 and revision id 952034884
 -->
</div><noscript><img src="//en.wikipedia.org/wiki/Special:CentralAutoLogin/start?type=1x1" alt="" title="" width="1" height="1" style="border: none; position: absolute;" /></noscript></div>
		<div class="printfooter">Retrieved from "<a dir="ltr" href="https://en.wikipedia.org/w/index.php?title=List_of_RNA_structure_prediction_software&amp;oldid=952034884">https://en.wikipedia.org/w/index.php?title=List_of_RNA_structure_prediction_software&amp;oldid=952034884</a>"</div>
		<div id="catlinks" class="catlinks" data-mw="interface"><div id="mw-normal-catlinks" class="mw-normal-catlinks"><a href="/wiki/Help:Category" title="Help:Category">Categories</a>: <ul><li><a href="/wiki/Category:Structural_bioinformatics_software" title="Category:Structural bioinformatics software">Structural bioinformatics software</a></li><li><a href="/wiki/Category:Lists_of_software" title="Category:Lists of software">Lists of software</a></li><li><a href="/wiki/Category:RNA" title="Category:RNA">RNA</a></li></ul></div><div id="mw-hidden-catlinks" class="mw-hidden-catlinks mw-hidden-cats-hidden">Hidden categories: <ul><li><a href="/wiki/Category:CS1_maint:_multiple_names:_authors_list" title="Category:CS1 maint: multiple names: authors list">CS1 maint: multiple names: authors list</a></li><li><a href="/wiki/Category:CS1:_long_volume_value" title="Category:CS1: long volume value">CS1: long volume value</a></li><li><a href="/wiki/Category:Articles_with_short_description" title="Category:Articles with short description">Articles with short description</a></li><li><a href="/wiki/Category:All_articles_with_dead_external_links" title="Category:All articles with dead external links">All articles with dead external links</a></li><li><a href="/wiki/Category:Articles_with_dead_external_links_from_March_2020" title="Category:Articles with dead external links from March 2020">Articles with dead external links from March 2020</a></li><li><a href="/wiki/Category:Articles_with_permanently_dead_external_links" title="Category:Articles with permanently dead external links">Articles with permanently dead external links</a></li><li><a href="/wiki/Category:Articles_with_dead_external_links_from_September_2019" title="Category:Articles with dead external links from September 2019">Articles with dead external links from September 2019</a></li></ul></div></div>
		<div class="visualClear"></div>
		
	</div>
</div>
<div id='mw-data-after-content'>
	<div class="read-more-container"></div>
</div>

<div id="mw-navigation">
	<h2>Navigation menu</h2>
	<div id="mw-head">
		
<div id="p-personal" role="navigation" class="" aria-labelledby="p-personal-label">
	<h3 id="p-personal-label">Personal tools</h3>
	<ul >
		<li id="pt-anonuserpage">Not logged in</li><li id="pt-anontalk"><a href="/wiki/Special:MyTalk" title="Discussion about edits from this IP address [n]" accesskey="n">Talk</a></li><li id="pt-anoncontribs"><a href="/wiki/Special:MyContributions" title="A list of edits made from this IP address [y]" accesskey="y">Contributions</a></li><li id="pt-createaccount"><a href="/w/index.php?title=Special:CreateAccount&amp;returnto=List+of+RNA+structure+prediction+software" title="You are encouraged to create an account and log in; however, it is not mandatory">Create account</a></li><li id="pt-login"><a href="/w/index.php?title=Special:UserLogin&amp;returnto=List+of+RNA+structure+prediction+software" title="You&#039;re encouraged to log in; however, it&#039;s not mandatory. [o]" accesskey="o">Log in</a></li>
	</ul>
</div>

		<div id="left-navigation">
			<div id="p-namespaces" role="navigation" class="vectorTabs " aria-labelledby="p-namespaces-label">
	<h3 id="p-namespaces-label">Namespaces</h3>
	<ul >
		<li id="ca-nstab-main" class="selected"><a href="/wiki/List_of_RNA_structure_prediction_software" title="View the content page [c]" accesskey="c">Article</a></li><li id="ca-talk"><a href="/wiki/Talk:List_of_RNA_structure_prediction_software" rel="discussion" title="Discuss improvements to the content page [t]" accesskey="t">Talk</a></li>
	</ul>
</div>

			<div id="p-variants" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-variants-label">
	<input type="checkbox" class="vectorMenuCheckbox" aria-labelledby="p-variants-label" />
	<h3 id="p-variants-label">
		<span>Variants</span>
	</h3>
	<ul class="menu" >
		
	</ul>
</div>

		</div>
		<div id="right-navigation">
			<div id="p-views" role="navigation" class="vectorTabs " aria-labelledby="p-views-label">
	<h3 id="p-views-label">Views</h3>
	<ul >
		<li id="ca-view" class="collapsible selected"><a href="/wiki/List_of_RNA_structure_prediction_software">Read</a></li><li id="ca-edit" class="collapsible"><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=edit" title="Edit this page [e]" accesskey="e">Edit</a></li><li id="ca-history" class="collapsible"><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=history" title="Past revisions of this page [h]" accesskey="h">View history</a></li>
	</ul>
</div>

			<div id="p-cactions" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-cactions-label">
	<input type="checkbox" class="vectorMenuCheckbox" aria-labelledby="p-cactions-label" />
	<h3 id="p-cactions-label">
		<span>More</span>
	</h3>
	<ul class="menu" >
		
	</ul>
</div>

			<div id="p-search" role="search">
	<h3 >
		<label for="searchInput">Search</label>
	</h3>
	<form action="/w/index.php" id="searchform">
		<div id="simpleSearch">
			<input type="search" name="search" placeholder="Search Wikipedia" title="Search Wikipedia [f]" accesskey="f" id="searchInput"/>
			<input type="hidden" value="Special:Search" name="title"/>
			<input type="submit" name="fulltext" value="Search" title="Search Wikipedia for this text" id="mw-searchButton" class="searchButton mw-fallbackSearchButton"/>
			<input type="submit" name="go" value="Go" title="Go to a page with this exact name if it exists" id="searchButton" class="searchButton"/>
		</div>
	</form>
</div>

		</div>
	</div>
	
<div id="mw-panel">
	<div id="p-logo" role="banner">
		<a  title="Visit the main page" class="mw-wiki-logo" href="/wiki/Main_Page"></a>
	</div>
	<div class="portal" role="navigation" id="p-navigation"  aria-labelledby="p-navigation-label">
	<h3  id="p-navigation-label">
		Navigation
	</h3>
	<div class="body">
		<ul><li id="n-mainpage-description"><a href="/wiki/Main_Page" title="Visit the main page [z]" accesskey="z">Main page</a></li><li id="n-contents"><a href="/wiki/Wikipedia:Contents" title="Guides to browsing Wikipedia">Contents</a></li><li id="n-featuredcontent"><a href="/wiki/Wikipedia:Featured_content" title="Featured content – the best of Wikipedia">Featured content</a></li><li id="n-currentevents"><a href="/wiki/Portal:Current_events" title="Find background information on current events">Current events</a></li><li id="n-randompage"><a href="/wiki/Special:Random" title="Load a random article [x]" accesskey="x">Random article</a></li><li id="n-sitesupport"><a href="https://donate.wikimedia.org/wiki/Special:FundraiserRedirector?utm_source=donate&amp;utm_medium=sidebar&amp;utm_campaign=C13_en.wikipedia.org&amp;uselang=en" title="Support us">Donate to Wikipedia</a></li><li id="n-shoplink"><a href="//shop.wikimedia.org" title="Visit the Wikipedia store">Wikipedia store</a></li></ul>
		
	</div>
</div>


	<div class="portal" role="navigation" id="p-interaction"  aria-labelledby="p-interaction-label">
	<h3  id="p-interaction-label">
		Interaction
	</h3>
	<div class="body">
		<ul><li id="n-help"><a href="/wiki/Help:Contents" title="Guidance on how to use and edit Wikipedia">Help</a></li><li id="n-aboutsite"><a href="/wiki/Wikipedia:About" title="Find out about Wikipedia">About Wikipedia</a></li><li id="n-portal"><a href="/wiki/Wikipedia:Community_portal" title="About the project, what you can do, where to find things">Community portal</a></li><li id="n-recentchanges"><a href="/wiki/Special:RecentChanges" title="A list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li><li id="n-contactpage"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us" title="How to contact Wikipedia">Contact page</a></li></ul>
		
	</div>
</div>

<div class="portal" role="navigation" id="p-tb"  aria-labelledby="p-tb-label">
	<h3  id="p-tb-label">
		Tools
	</h3>
	<div class="body">
		<ul><li id="t-whatlinkshere"><a href="/wiki/Special:WhatLinksHere/List_of_RNA_structure_prediction_software" title="List of all English Wikipedia pages containing links to this page [j]" accesskey="j">What links here</a></li><li id="t-recentchangeslinked"><a href="/wiki/Special:RecentChangesLinked/List_of_RNA_structure_prediction_software" rel="nofollow" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li><li id="t-upload"><a href="/wiki/Wikipedia:File_Upload_Wizard" title="Upload files [u]" accesskey="u">Upload file</a></li><li id="t-specialpages"><a href="/wiki/Special:SpecialPages" title="A list of all special pages [q]" accesskey="q">Special pages</a></li><li id="t-permalink"><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;oldid=952034884" title="Permanent link to this revision of the page">Permanent link</a></li><li id="t-info"><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;action=info" title="More information about this page">Page information</a></li><li id="t-wikibase"><a href="https://www.wikidata.org/wiki/Special:EntityPage/Q6595358" title="Link to connected data repository item [g]" accesskey="g">Wikidata item</a></li><li id="t-cite"><a href="/w/index.php?title=Special:CiteThisPage&amp;page=List_of_RNA_structure_prediction_software&amp;id=952034884&amp;wpFormIdentifier=titleform" title="Information on how to cite this page">Cite this page</a></li></ul>
		
	</div>
</div>

<div class="portal" role="navigation" id="p-coll-print_export"  aria-labelledby="p-coll-print_export-label">
	<h3  id="p-coll-print_export-label">
		Print/export
	</h3>
	<div class="body">
		<ul><li id="coll-create_a_book"><a href="/w/index.php?title=Special:Book&amp;bookcmd=book_creator&amp;referer=List+of+RNA+structure+prediction+software">Create a book</a></li><li id="coll-download-as-rl"><a href="/w/index.php?title=Special:ElectronPdf&amp;page=List+of+RNA+structure+prediction+software&amp;action=show-download-screen">Download as PDF</a></li><li id="t-print"><a href="/w/index.php?title=List_of_RNA_structure_prediction_software&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</a></li></ul>
		
	</div>
</div>

<div class="portal" role="navigation" id="p-lang"  aria-labelledby="p-lang-label">
	<h3  id="p-lang-label">
		Languages
	</h3>
	<div class="body">
		<ul></ul>
		<div class="after-portlet after-portlet-lang"><span class="wb-langlinks-add wb-langlinks-link"><a href="https://www.wikidata.org/wiki/Special:EntityPage/Q6595358#sitelinks-wikipedia" title="Add interlanguage links" class="wbc-editpage">Add links</a></span></div>
	</div>
</div>


</div>

</div>

<div id="footer" role="contentinfo" >
	<ul id="footer-info" class="">
		<li id="footer-info-lastmod"> This page was last edited on 20 April 2020, at 07:14<span class="anonymous-show">&#160;(UTC)</span>.</li>
		<li id="footer-info-copyright">Text is available under the <a rel="license" href="//en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License">Creative Commons Attribution-ShareAlike License</a><a rel="license" href="//creativecommons.org/licenses/by-sa/3.0/" style="display:none;"></a>;
additional terms may apply.  By using this site, you agree to the <a href="//foundation.wikimedia.org/wiki/Terms_of_Use">Terms of Use</a> and <a href="//foundation.wikimedia.org/wiki/Privacy_policy">Privacy Policy</a>. Wikipedia® is a registered trademark of the <a href="//www.wikimediafoundation.org/">Wikimedia Foundation, Inc.</a>, a non-profit organization.</li>
	</ul>
	<ul id="footer-places" class="">
		<li id="footer-places-privacy"><a href="https://foundation.wikimedia.org/wiki/Privacy_policy" class="extiw" title="wmf:Privacy policy">Privacy policy</a></li>
		<li id="footer-places-about"><a href="/wiki/Wikipedia:About" title="Wikipedia:About">About Wikipedia</a></li>
		<li id="footer-places-disclaimer"><a href="/wiki/Wikipedia:General_disclaimer" title="Wikipedia:General disclaimer">Disclaimers</a></li>
		<li id="footer-places-contact"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us">Contact Wikipedia</a></li>
		<li id="footer-places-developers"><a href="https://www.mediawiki.org/wiki/Special:MyLanguage/How_to_contribute">Developers</a></li>
		<li id="footer-places-statslink"><a href="https://stats.wikimedia.org/#/en.wikipedia.org">Statistics</a></li>
		<li id="footer-places-cookiestatement"><a href="https://foundation.wikimedia.org/wiki/Cookie_statement">Cookie statement</a></li>
		<li id="footer-places-mobileview"><a href="//en.m.wikipedia.org/w/index.php?title=List_of_RNA_structure_prediction_software&amp;mobileaction=toggle_view_mobile" class="noprint stopMobileRedirectToggle">Mobile view</a></li>
	</ul>
	<ul id="footer-icons" class="noprint">
		<li id="footer-copyrightico"><a href="https://wikimediafoundation.org/"><img src="/static/images/wikimedia-button.png" srcset="/static/images/wikimedia-button-1.5x.png 1.5x, /static/images/wikimedia-button-2x.png 2x" width="88" height="31" alt="Wikimedia Foundation"/></a></li>
		<li id="footer-poweredbyico"><a href="https://www.mediawiki.org/"><img src="/static/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" srcset="/static/images/poweredby_mediawiki_132x47.png 1.5x, /static/images/poweredby_mediawiki_176x62.png 2x" width="88" height="31"/></a></li>
	</ul>
	<div style="clear: both;"></div>
</div>


<script>(RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgPageParseReport":{"limitreport":{"cputime":"1.936","walltime":"2.041","ppvisitednodes":{"value":11977,"limit":1000000},"postexpandincludesize":{"value":571550,"limit":2097152},"templateargumentsize":{"value":3080,"limit":2097152},"expansiondepth":{"value":15,"limit":40},"expensivefunctioncount":{"value":2,"limit":500},"unstrip-depth":{"value":1,"limit":20},"unstrip-size":{"value":971910,"limit":5000000},"entityaccesscount":{"value":0,"limit":400},"timingprofile":["100.00% 1730.741      1 -total"," 82.74% 1432.079      8 Template:Reflist"," 67.24% 1163.810    209 Template:Cite_journal","  3.20%   55.441      1 Template:Short_description","  2.44%   42.240      1 Template:Pagetype","  2.43%   41.995      2 Template:Dead_link","  1.98%   34.232      2 Template:Fix","  1.73%   29.923      4 Template:Category_handler","  0.74%   12.890    187 Template:Yes","  0.56%    9.620      1 Template:Cite_arxiv"]},"scribunto":{"limitreport-timeusage":{"value":"1.108","limit":"10.000"},"limitreport-memusage":{"value":5484095,"limit":52428800},"limitreport-profile":[["Scribunto_LuaSandboxCallback::plain","120","10.5"],["?","120","10.5"],["recursiveClone \u003CmwInit.lua:41\u003E","120","10.5"],["Scribunto_LuaSandboxCallback::gsub","100","8.8"],["Scribunto_LuaSandboxCallback::callParserFunction","100","8.8"],["Scribunto_LuaSandboxCallback::find","100","8.8"],["\u003Cmw.lua:683\u003E","60","5.3"],["Scribunto_LuaSandboxCallback::len","40","3.5"],["Scribunto_LuaSandboxCallback::match","40","3.5"],["format","40","3.5"],["[others]","300","26.3"]]},"cachereport":{"origin":"mw1288","timestamp":"20200420071442","ttl":2592000,"transientcontent":false}}});});</script>
<script type="application/ld+json">{"@context":"https:\/\/schema.org","@type":"Article","name":"List of RNA structure prediction software","url":"https:\/\/en.wikipedia.org\/wiki\/List_of_RNA_structure_prediction_software","sameAs":"http:\/\/www.wikidata.org\/entity\/Q6595358","mainEntity":"http:\/\/www.wikidata.org\/entity\/Q6595358","author":{"@type":"Organization","name":"Contributors to Wikimedia projects"},"publisher":{"@type":"Organization","name":"Wikimedia Foundation, Inc.","logo":{"@type":"ImageObject","url":"https:\/\/www.wikimedia.org\/static\/images\/wmf-hor-googpub.png"}},"datePublished":"2008-01-19T10:53:47Z","dateModified":"2020-04-20T07:14:34Z","headline":"Wikimedia list article"}</script>
<script>(RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgBackendResponseTime":160,"wgHostname":"mw1397"});});</script></body></html>
